<!DOCTYPE html><html lang="en">
<head>
<meta charset="utf-8">
<meta name="format-detection" content="telephone=no">
<title>32: Molecular Evolution | Video Lectures | Introductory Biology | Biology | MIT OpenCourseWare</title>
<!-- Begin Automatic Metadata Insertion --><meta content="7-014-introductory-biology-spring-2005" name="WT.cg_n">
<meta content="32: Molecular Evolution" name="WT.cg_s">
<meta content="" name="Description">
<meta content="Walker, Graham" name="Author">
<meta content="Khodor, Julia" name="Author">
<meta content="Mischke, Michelle" name="Author">
<meta content="Chisholm, Penny" name="Author">
<meta content="microorganisms,geochemistry,geochemical agents,biosphere,bacterial genetics,carbon metabolism,energy metabolism,productivity,biogeochemical cycles,molecular evolution,population genetics,evolution,population growth,biology,biochemistry,genetics,molecular biology,recombinant DNA,cell cycle,cell signaling,cloning,stem cells,cancer,immunology,virology,genomics,molecular medicine,DNA,RNA,proteins,replication,transcription,mRNA,translation,ribosome,nervous system,amino acids,polypeptide chain,cell biology,neurobiology,gene regulation,protein structure,protein synthesis,gene structure,PCR,polymerase chain reaction,protein localization,endoplasmic reticulum,ecology,communities,Biology" name="keywords">
<meta content="32: Molecular Evolution | MIT OpenCourseWare" name="Search_Display">
<meta content="Biology" itemprop="about">
<!-- End Automatic Metadata Insertion --><link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/grid.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/base.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/menu.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/jquery.bubblepopup.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/courses.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/courses_new.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/jquery.jscrollpane.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/media_tabs.css">
<link href="http://ocw.mit.edu/xml/ocwcc.rdf" type="application/rdf+xml" rel="metadata">
<link rel="canonical" href="http://ocw.mit.edu/courses/biology/7-014-introductory-biology-spring-2005/video-lectures/32-molecular-evolution">
<link rel="apple-touch-icon" href="../../../common/images/apple-touch-icon.png">
<script type="text/javascript" src="../../../common/scripts/common-utils-offline.js"></script><script type="text/javascript" src="../../../common/scripts/ocw-offline.js"></script><script type="text/javascript" src="../../../common/scripts/jquery.bubblepopup.min.js"></script><script type="text/javascript" src="../../../common/scripts/jquery-ui.min.js"></script><script type="text/javascript" src="../../../common/scripts/jquery.jscrollpane.min.js"></script><script type="text/javascript" src="../../../common/scripts/bubble-popup-offline.js"></script><script type="text/javascript">
      $(document).ready(function() {
        $("#tabs").tabs();
        IpadScroller();
      });
    </script>
</head>
<body itemscope itemtype="http://schema.org/WebPage">
        
	

        <div id="top">
			<div id="grid">
				
				
					
<div id="portletwrapper-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572" class="portletWrapper kssattr-portlethash-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572">
<div class="portletStaticText portlet-static-site-header">
<div id="banner" role="banner" class="grid_6 alpha"><a href="http://ocw.mit.edu/"><img src="../../../common/images/ocw_mast.png" alt="MIT OpenCourseWare, Massachusetts Institute of Technology" class="logo"></a></div>
<form action="http://ocw.mit.edu/subscribe/index.htm?utm_source=header" method="get">
    <div id="subscribe" role="form toolbar" class="grid_6 omega">
    <div class="module">
    <table class="social"><tbody><tr>
<td class="text">Subscribe to the<br>
                OCW Newsletter</td>
                <td class="black"><input type="text" onclick="clearEmailBox()" onblur="fillEmailBox()" value="Enter Email" class="greytext emailField" id="email" name="fromemail"></td>
                <td class="black"><input type="image" src="../../../common/images/button_subscribe.png" alt="Subscribe" class="sub_button"></td>
                <td>
<a href="http://facebook.com/mitocw"><img src="../../../common/images/icon_fb.png" alt="Click to visit our Facebook page."></a><a href="http://twitter.com/mitocw"><img src="../../../common/images/icon_tw.png" alt="Click to visit our Twitter feed."></a>
</td>
            </tr></tbody></table>
</div>
    <p class="helplinks"><a href="http://ocw.mit.edu/help">Help</a>   |   <a href="../../../common/jsp/feedback.htm">Contact Us</a></p>
    </div>
    <div class="clear"> </div>
</form>
</div>

</div>





<div id="mega" role="navigation" class="grid_8 alpha">        
	<ul id="menu">
<li id="menu_home">
            <a href="http://ocw.mit.edu/"><img src="../../../common/images/top-nav_home.png" class="home_icon" alt="Click for site home page."></a><!-- Begin Home Item -->
        </li>
<!-- End Home Item -->        
        <li class="selected">
            <a href="#" class="drop">Courses</a><!-- Begin 5 columns Item -->
            <div class="dropdown_5columns-a mega-courses">                    
                <div class="col_1a">
                    <div class="row_1a">
                        <div class="quart">
                            <h2 class="nav">Find courses by:</h2>
                            <ul class="nav-bullet find_by">
<li><a href="http://ocw.mit.edu/courses/find-by-topic/">Topic</a></li>
                                <li><a href="http://ocw.mit.edu/courses/find-by-number/">MIT Course Number</a></li>
                                <li><a href="http://ocw.mit.edu/courses/find-by-department/">Department</a></li>
                            </ul>
</div>
                        <div class="quart">
                            <h2 class="nav">Collections</h2>
                            <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Lectures</a></li>
                                <li><a href="http://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
                                <li><a href="http://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
                                <li><a href="http://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
                                <li><a href="http://ocw.mit.edu/resources">Supplemental Resources</a></li>
                            </ul>
</div>
                        <div class="clear"> </div>
                    </div>
                    <div class="row_1b">
                        <h2 class="nav">Cross-Disciplinary Topic Lists</h2>
                        <div class="quart">
                        <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/energy-courses">Energy</a></li>
                            <li><a href="http://ocw.mit.edu/courses/entrepreneurship">Entrepreneurship</a></li>
                            <li><a href="http://ocw.mit.edu/courses/environment-courses">Environment</a></li>
                        </ul>
</div>    
                        <div class="quart">
                        <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/intro-programming">Introductory Programming</a></li>
                            <li><a href="http://ocw.mit.edu/courses/life-sciences">Life Sciences</a></li>
                            <li><a href="http://ocw.mit.edu/courses/transportation-courses">Transportation</a></li>
                        </ul>
</div>
                        <div class="clear"> </div>
                    </div>
                    <div class="clear"> </div>
                </div>
                <div class="col_1b">
                    <h2 class="nav">Translated Courses</h2>
                    <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/translated-courses/traditional-chinese">繁體字 / Traditional Chinese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/simplified-chinese">简体字 / Simplified Chinese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/spanish">Español / Spanish</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/portuguese">Português / Portuguese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/thai">ภาษาเขียน / Thai</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/persian">فارسی / Persian</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/turkish">Türkçe / Turkish</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/korean">(비디오)한국 / Korean</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses">More...</a></li>
                    </ul>
</div>
            </div>
        </li>
        <li>
            <a href="" class="drop">About</a>
            <div class="dropdown_1column-a">
                <div class="col_1">
                    <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/">About MIT OpenCourseWare</a></li>
                    </ul>
<ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/site-statistics/">Site Stats</a></li>
                        <li><a href="http://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
                        <li><a href="http://ocw.mit.edu/about/ocw-consortium/">OpenCourseWare Consortium</a></li>
                    </ul>
<ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/media-coverage/">Media Coverage</a></li>
                        <li><a href="http://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
                        <li><a href="http://ocw.mit.edu/about/media-coverage/press-releases/">Press Releases</a></li>
                    </ul>
<ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/about/next-decade/">OCW's Next Decade</a></li>
                    </ul>
</div>
            </div>  
        </li>    
        <li>
            <a href="" class="drop">Donate</a>        
            <div class="dropdown_1column-a">
                    <div class="col_1">
                        <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/donate/">Make a Donation</a></li>
                            <li><a href="http://ocw.mit.edu/donate/why-donate/">Why Donate?</a></li>
                            <li><a href="http://ocw.mit.edu/donate/ocw-course-champions-program/">Become a Course Champion</a></li>
                            <li><a href="http://ocw.mit.edu/donate/our-supporters/">Our Supporters</a></li>
                            <li><a href="http://ocw.mit.edu/donate/other-ways-to-contribute/">Other Ways to Contribute</a></li>
                            <li><a href="http://ocw.mit.edu/donate/shop-ocw">Shop OCW</a></li>
                        </ul>
<ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
                        </ul>
</div>
            </div>            
        </li>        
        <li>
            <a href="" class="drop">Featured Sites</a>        
            <div class="dropdown_1column-a">
                <div class="col_1">
                    <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/high-school/">Highlights for High School</a></li>
                    </ul>
<ul class="nav-bullet mega-div-top">
<li><a href="https://www.edx.org/school/mitx/allcourses">MITx</a></li>
                        <li><a href="http://teachingexcellence.mit.edu/">Teaching Excellence at MIT</a></li>
                    </ul>
</div>
            </div>            
        </li>
    </ul>
</div>
<div id="search" role="search" class="grid_4 omega">
    
    <form method="get" action="../../../common/search/AdvancedSearch.htm">
     	 <table class="search"><tbody><tr>
<td class="black"><input type="text" onblur="fillSearchBox()" onfocus="clearSearchBox()" maxlength="255" value="Search" name="q" class="greytext searchField" id="terms"></td> 			 
                    <td class="black"><input type="image" src="../../../common/images/button_search.png" name="btnG" class="sub_button"></td>			 
                    <td class="text2"><a href="../../../common/search/AdvancedSearch.htm">Advanced<br>Search</a></td>
                </tr></tbody></table>
</form>
</div>
<div class="clear"></div>
<!-- *end header* -->  

				
				
			</div>
<!-- top grid end -->
		</div>
<!-- top end -->
			
		<div id="center_media">
      	<div id="grid">
      		<div id="left">
        		<div id="breadcrumb_media">
                	<p>

    <a href="http://ocw.mit.edu/">Home</a>
    
        »
        
    
    
        
            <a href="http://ocw.mit.edu/courses">Courses</a>
            
                »
                
            
            
         
    
    
        
            <a href="http://ocw.mit.edu/courses/biology">Biology</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../../contents/index.htm">Introductory Biology</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../../contents/video-lectures/index.htm">Video Lectures</a>
            
                »
                
            
            
         
    
    
        
            
            
            32: Molecular Evolution
         
    
</p>

            	</div>
            	<div class="clear"></div>
        		<div id="media_title">
        		<h1 class="title" itemprop="name" property="dct:title">
        <span class="" id="parent-fieldname-title">
            32: Molecular Evolution
        </span>
    </h1>
        		</div>
           		<div class="clear"></div>
           		<div id="course_wrapper_media">
           			<div id="course_nav">
           				<script language="javascript" type="text/javascript">
function toggleMenu(objID) {
  if (!document.getElementById) return;
  var ob = document.getElementById(objID);
  ob.className = (ob.className == 'selected')?'': 'selected';
}
function toggleClass(id)
{
  var divtoggleClass= document.getElementById(id);
  divtoggleClass.className = (divtoggleClass.className == 'mO')?'mC': 'mO';
  return false;
}
function changeAlt(id)
{
  id.alt = (id.alt == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
  id.title = (id.title == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
}
</script><!--Left Nav Starts --><ul>
<li class="">
			   			<a href="../../../contents/index.htm">
		                  Course Home  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/syllabus/index.htm">
		                  Syllabus  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/calendar/index.htm">
		                  Calendar  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/readings/index.htm">
		                  Readings  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/recitations/index.htm">
		                  Recitations  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/assignments/index.htm">
		                  Assignments  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/exams/index.htm">
		                  Exams  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/study-materials/index.htm">
		                  Study Materials  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="selected">
			   			<a href="../../../contents/video-lectures/index.htm">
		                  Video Lectures  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/unit-questions/index.htm">
		                  Unit Questions  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    
		    
         	
	<!--second tal block close-->  
	
</ul>
<!--Left Nav Ends --><script type="text/javascript" id="openstudy-widget-loader" src="http://openstudy.com/javascripts/load.js?type=feed&amp;color=auto&amp;studygroup=MIT%20OCW%20Biology&amp;env=openstudy.com"></script>
</div>
           			<div id="course_inner_media">
      					 
        <div class="" id="parent-fieldname-text">
            
            
        </div>
    
      					 

     <div id="media-embed">
         <div class="attention_message" id="embed_1">
<p>Flash and JavaScript are required for this feature.</p>
<p>Download the video from <a href="http://deimos3.apple.com/WebObjects/Core.woa/Browse/mit.edu.1329584553.01329584560.1329911955?i=2014086142">iTunes U</a> or the <a href="http://www.archive.org/download/MIT7.014S05/ocw-7.014-32-02may05-220k_512kb.mp4">Internet Archive</a>.</p>
</div>
     </div>
    
     <script type="text/javascript">ocw_embed_chapter_media('embed_1', 'http://www.youtube.com/v/ONYokXoy04Q', 'youtube', '/courses/biology/7-014-introductory-biology-spring-2005/video-lectures/32-molecular-evolution', 'http://img.youtube.com/vi/ONYokXoy04Q/0.jpg',0,0, 'http://ocw.mit.edu/courses/biology/7-014-introductory-biology-spring-2005/video-lectures/32-molecular-evolution/7.014-2005-L32.srt')</script><div id="media_resource_next_prev_nav" style="margin-top: 1em;">
        <p>
        
            <a href="../../../contents/video-lectures/31-population-genetics-and-evolution/index.htm">
                <img src="../../../common/images/btn_previous_resource.png" style="margin: 0 30px 0 50px;" alt="Previous track" title="Previous track"></a>
     	
     	
        
            <a href="../../../contents/video-lectures/33-communities-i/index.htm"> 
                <img src="../../../common/images/btn_next_resource.png" alt="Next track" title="Next track"></a>
       
       </p>
     </div>
 


<script type="text/javascript">
		window.onload=function(){
		init();
		
		}
		var tabLinks = new Array();
		var contentDivs = new Array();
		function init() {
		  // Grab the tab links and content divs from the page
		  var tabListItems = document.getElementById('tabs').childNodes;
		  for ( var i = 0; i < tabListItems.length; i++ ) {
			if ( tabListItems[i].nodeName == "LI" ) {
			  var tabLink = getFirstChildWithTagName( tabListItems[i], 'A' );
			  var id = getHash( tabLink.getAttribute('href') );
			  tabLinks[id] = tabLink;
			  contentDivs[id] = document.getElementById( id );
			}
		  }
		  // Assign onclick events to the tab links, and
		  // highlight the first tab
		  var i = 0;
		  for ( var id in tabLinks ) {
			tabLinks[id].onclick = showTab;
			tabLinks[id].onfocus = function() { this.blur() };
			if ( i == 0 ) tabLinks[id].className = 'selected';
			i++;
		  }
		  // Hide all content divs except the first
		  var i = 0;
		  for ( var id in contentDivs ) {
			if ( i != 0 ) contentDivs[id].className = 'tabContent hide';
			i++;
		  }
		}
		function showTab() {
		  var selectedId = getHash( this.getAttribute('href') );
		  // Highlight the selected tab, and dim all others.
		  // Also show the selected content div, and hide all others.
		  for ( var id in contentDivs ) {
			if ( id == selectedId ) {
			  tabLinks[id].className = 'selected';
			  contentDivs[id].className = 'tabContent';
			} else {
			  tabLinks[id].className = '';
			  contentDivs[id].className = 'tabContent hide';
			}
		  }
		  // Stop the browser following the link
		  return false;
		}
		function getFirstChildWithTagName( element, tagName ) {
		  for ( var i = 0; i < element.childNodes.length; i++ ) {
			if ( element.childNodes[i].nodeName == tagName ) return element.childNodes[i];
		  }
		}
		function getHash( url ) {
		  var hashPos = url.lastIndexOf ( '#' );
		  return url.substring( hashPos + 1 );
		}
 </script><div id="media_tabs">
     
        <ul id="tabs">
<li class="first">
                <a href="#vid_about" class="selected">About this Video</a>
            </li>
            <li class="">
                <a href="#vid_index" class="">Playlist</a>
            </li>
            <li class="">
                <a href="#vid_playlist" class="">Transcript</a>
            </li>
            <li class="">
                <a href="#vid_related" class="">Download this Video</a>
            </li>
        </ul>
<div id="vid_about" itemprop="description" class="tabContent"><p><strong>Topics covered:</strong> Molecular Evolution<br><br><strong>Instructor: </strong> Prof. Martin Polz, Guest Lecturer</p></div>
        <div id="vid_index" itemprop="description" class="tabContent hide">
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/1-introduction/index.htm">
<img src="../../../contents/video-lectures/1-introduction/lec1.jpg" title="1: Introduction" alt="1: Introduction"><p>1: Introduction</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/2-biochemistry-i/index.htm">
<img src="../../../contents/video-lectures/2-biochemistry-i/lec2.jpg" title="2: Biochemistry I" alt="2: Biochemistry I"><p>2: Biochemistry I</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/3-biochemistry-ii/index.htm">
<img src="../../../contents/video-lectures/3-biochemistry-ii/lec3.jpg" title="3: Biochemistry II" alt="3: Biochemistry II"><p>3: Biochemistry II</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/4-biochemistry-iii/index.htm">
<img src="../../../contents/video-lectures/4-biochemistry-iii/lec4.jpg" title="4: Biochemistry III" alt="4: Biochemistry III"><p>4: Biochemistry III</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/5-biochemistry-iv/index.htm">
<img src="../../../contents/video-lectures/5-biochemistry-iv/lec5.jpg" title="5: Biochemistry IV" alt="5: Biochemistry IV"><p>5: Biochemistry IV</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/6-biochemistry-v/index.htm">
<img src="../../../contents/video-lectures/6-biochemistry-v/lec6.jpg" title="6: Biochemistry V" alt="6: Biochemistry V"><p>6: Biochemistry V</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/7-biochemistry-vi/index.htm">
<img src="../../../contents/video-lectures/7-biochemistry-vi/lec7.jpg" title="7: Biochemistry VI" alt="7: Biochemistry VI"><p>7: Biochemistry VI</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/8-biochemistry-vi-cont.-dna-as-genetic-material/index.htm">
<img src="../../../contents/video-lectures/8-biochemistry-vi-cont.-dna-as-genetic-material/lec8.jpg" title="8: Biochemistry VI (cont.) - DNA as Genetic Material" alt="8: Biochemistry VI (cont.) - DNA as Genetic Material"><p>8: Biochemistry VI (cont.) ...</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/9-molecular-biology-i/index.htm">
<img src="../../../contents/video-lectures/9-molecular-biology-i/lec9.jpg" title="9: Molecular Biology I" alt="9: Molecular Biology I"><p>9: Molecular Biology I</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/10-molecular-biology-ii-process-of-science/index.htm">
<img src="../../../contents/video-lectures/10-molecular-biology-ii-process-of-science/lec10.jpg" title="10: Molecular Biology II - Process of Science" alt="10: Molecular Biology II - Process of Science"><p>10: Molecular Biology II - ...</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/11-molecular-biology-iii/index.htm">
<img src="../../../contents/video-lectures/11-molecular-biology-iii/lec11.jpg" title="11: Molecular Biology III" alt="11: Molecular Biology III"><p>11: Molecular Biology III</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/12-molecular-biology-iv/index.htm">
<img src="../../../contents/video-lectures/12-molecular-biology-iv/lec12.jpg" title="12: Molecular Biology IV" alt="12: Molecular Biology IV"><p>12: Molecular Biology IV</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/mediaresource.2010-10-26.0571828698/index.htm">
<img src="../../../contents/video-lectures/mediaresource.2010-10-26.0571828698/lec13.jpg" title="13: 	  Molecular Biology IV (cont.) - Gene Regulation I" alt="13: 	  Molecular Biology IV (cont.) - Gene Regulation I"><p>13: 	  Molecular Biology IV...</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/14-gene-regulation-ii/index.htm">
<img src="../../../contents/video-lectures/14-gene-regulation-ii/lec14.jpg" title="14: Gene Regulation II" alt="14: Gene Regulation II"><p>14: Gene Regulation II</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/15-bacterial-genetics/index.htm">
<img src="../../../contents/video-lectures/15-bacterial-genetics/lec15.jpg" title="15: Bacterial Genetics" alt="15: Bacterial Genetics"><p>15: Bacterial Genetics</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/1-the-biosphere/index.htm">
<img src="../../../contents/video-lectures/1-the-biosphere/lec16.jpg" title="16: The Biosphere" alt="16: The Biosphere"><p>16: The Biosphere</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/17-carbon-and-energy-metabolism/index.htm">
<img src="../../../contents/video-lectures/17-carbon-and-energy-metabolism/lec17.jpg" title="17: Carbon and Energy Metabolism" alt="17: Carbon and Energy Metabolism"><p>17: Carbon and Energy Metab...</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/18-productivity-and-food-webs/index.htm">
<img src="../../../contents/video-lectures/18-productivity-and-food-webs/lec18.jpg" title="18: Productivity and Food Webs" alt="18: Productivity and Food Webs"><p>18: Productivity and Food Webs</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/19-regulation-of-productivity/index.htm">
<img src="../../../contents/video-lectures/19-regulation-of-productivity/lec19.jpg" title="19: Regulation of Productivity" alt="19: Regulation of Productivity"><p>19: Regulation of Productivity</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/20-limiting-factors-and-biogeochemical-cycleslimiting-factors-and-biogeochemical-cycles/index.htm">
<img src="../../../contents/video-lectures/20-limiting-factors-and-biogeochemical-cycleslimiting-factors-and-biogeochemical-cycles/lec20.jpg" title="20: Limiting Factors and Biogeochemical Cycles" alt="20: Limiting Factors and Biogeochemical Cycles"><p>20: Limiting Factors and Bi...</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/21-mendelian-genetics/index.htm">
<img src="../../../contents/video-lectures/21-mendelian-genetics/lec21.jpg" title="21: Mendelian Genetics" alt="21: Mendelian Genetics"><p>21: Mendelian Genetics</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/22-mitosis-and-meiosis/index.htm">
<img src="../../../contents/video-lectures/22-mitosis-and-meiosis/lec22.jpg" title="22: Mitosis and Meiosis" alt="22: Mitosis and Meiosis"><p>22: Mitosis and Meiosis</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/23-diploid-genetics/index.htm">
<img src="../../../contents/video-lectures/23-diploid-genetics/lec23.jpg" title="23: Diploid Genetics" alt="23: Diploid Genetics"><p>23: Diploid Genetics</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/24-recombinant-dna-i/index.htm">
<img src="../../../contents/video-lectures/24-recombinant-dna-i/lec24.jpg" title="24: Recombinant DNA I" alt="24: Recombinant DNA I"><p>24: Recombinant DNA I</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/25-recombinant-dna-ii/index.htm">
<img src="../../../contents/video-lectures/25-recombinant-dna-ii/lec25.jpg" title="25: Recombinant DNA II" alt="25: Recombinant DNA II"><p>25: Recombinant DNA II</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/26-recombinant-dna-iii/index.htm">
<img src="../../../contents/video-lectures/26-recombinant-dna-iii/lec26.jpg" title="26: Recombinant DNA III" alt="26: Recombinant DNA III"><p>26: Recombinant DNA III</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/27-recombinant-dna-iii-cont.-immunology-i/index.htm">
<img src="../../../contents/video-lectures/27-recombinant-dna-iii-cont.-immunology-i/lec27.jpg" title="27: Recombinant DNA III (cont.) - Immunology I" alt="27: Recombinant DNA III (cont.) - Immunology I"><p>27: Recombinant DNA III (co...</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/2-immunology-ii/index.htm">
<img src="../../../contents/video-lectures/2-immunology-ii/lec28.jpg" title="28: Immunology II" alt="28: Immunology II"><p>28: Immunology II</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/29-population-growth-i/index.htm">
<img src="../../../contents/video-lectures/29-population-growth-i/lec29.jpg" title="29: Population Growth I" alt="29: Population Growth I"><p>29: Population Growth I</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/30-population-growth-ii/index.htm">
<img src="../../../contents/video-lectures/30-population-growth-ii/lec30.jpg" title="30: Population Growth II" alt="30: Population Growth II"><p>30: Population Growth II</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/31-population-genetics-and-evolution/index.htm">
<img src="../../../contents/video-lectures/31-population-genetics-and-evolution/lec31.jpg" title="31: Population Genetics and Evolution" alt="31: Population Genetics and Evolution"><p>31: Population Genetics and...</p></a>
</div>
<div class="related-media-thumbnail-nolink">
<div class="now-playing-resource">Now Playing</div>
<img src="../../../contents/video-lectures/32-molecular-evolution/lec32.jpg" title="32: Molecular Evolution" alt="32: Molecular Evolution"><p>32: Molecular Evolution</p>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/33-communities-i/index.htm">
<img src="../../../contents/video-lectures/33-communities-i/lec33.jpg" title="33: Communities I" alt="33: Communities I"><p>33: Communities I</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/34-communities-ii/index.htm">
<img src="../../../contents/video-lectures/34-communities-ii/lec34.jpg" title="34: Communities II" alt="34: Communities II"><p>34: Communities II</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/35-communities-iii/index.htm">
<img src="../../../common/images/audio_file.gif" title="35: Communities III" alt="35: Communities III"><p>35: Communities III</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/video-lectures/36-ecological-applications/index.htm">
<img src="../../../common/images/audio_file.gif" title="36: Ecological Applications" alt="36: Ecological Applications"><p>36: Ecological Applications</p></a>
</div>
</div>
        <div id="vid_playlist" itemprop="description" class="tabContent hide">
<ul><li><a class="transcript-link" title="Open in a new window." alt="Open in a new window." style="text-decoration: none; font-size: 1.0em;" target="_blank" text-decoration: none font-size: href="../../../contents/video-lectures/7_0142005L32.pdf"> Download this transcript - PDF (English - US)</a></li></ul>
<p></p>
<p>So what I want to do today is recap a little bit what we talked about  last time, reiterate some of the important points, and then show you  how we can learn something about microorganisms in the environment by  talking about in-situ identification of microorganisms as well as  genomics. We'll first talk about genomics and general and then talk  about some applications of genomics to environmental microbiology  because I think there is some of the most exciting new developments are  in the area, actually. So last time we talked about molecular evolution  and ecology.</p>
<p>And just to recap, some of the main points were that  we can actually use genes or gene sequences for a couple of very  important questions that we want to explore. The first one was gene  sequences act as evolutionary chronometers. Now, what do I mean by that?  Basically what we said last time was that each gene, each sequence in  the genome accumulates mutations with a certain probability. So what we  mean is that all genes accumulate mutations over time. Now, these of  course are the mutations that do not kill the organisms, so not the  deleterious mutations, but these are mutations that are either slightly  deleterious, or don't matter, or are beneficial mutations.</p>
<p>OK, and  what this entails is that each gene accumulates mutation with a certain  probability over time. It basically means that two organisms that come  from species that are relatively closely related to each other have gene  sequences that will be much more similar to each other than genes from  an organism that comes from a species that's much more distantly  related. So, in practical terms what this means is your genes are much,  much more similar to those of a monkey than they are to a crocodile, for  example.</p>
<p>And we can take advantage of that by applying some  algorithms, some mathematical modeling essentially, to constrain these  relationships in those phylogenetic trees that we talked about last  time. And I also mentioned that the ribosomal RNA genes are particularly  important for that process. In principle, you could do it with any  protein coding machine or any kind of gene in the genome, but we use the  ribosomal RNA genes in particular because all organisms have them.  They're part of a handful of genes that are what we called universally  distributed last time. And what this allows us to do is then construct  phylogenetic relationships for all living organisms.</p>
<p>And I just  want to remind you of the tree of life that I showed you last time where  we can really explore the relationships amongst all living organisms.  And some of the important points there that we made were, for example,  that the tree of life supports the endosymbiont theory, that when you  actually look on the tree where the mitochondria and the chloroplasts  tree, they fall into the bacteria. Now, there is a question where  somebody asked in the online survey, can the mitochondria and  chloroplasts still live outside of the eukaryotic cell? And the answer  is no, they can't anymore because over evolutionary time the two  organisms have become so integrated that the mitochondria and  chloroplasts both lost their ability to live outside of the eukaryotic  host cell.</p>
<p>Another important point that we made last time that I  want to reiterate here is that gene sequences, when we go into the  environment, and obtain them directly from the environment act as a  proxy for microbial diversity in the environment. So, the number of  genes recovered directly from the environment is a measure of diversity.</p>
<p>And  we said that this actually plays a very, very important role in the  analysis of microbial communities, and I showed you the example here  where we went and took some ocean water and basically apply this  technique that outlined last time where we can actually amplify  ribosomal RNA genes from environmental samples, clone them, determine  the sequence, and then constructs phylogenetic trees. And what you see  here is a tree where we summarize the major groups that we found in the  sample have been only for two of those groups where we show the entire  set of sequences that we actually obtained because there were so many of  them out there.</p>
<p>And what we basically found was that over 1500  bacterial 16S ribosomal RNA gene sequences coexist in this environment.  And what we said also last time was that the analyses like these have  really taught us that microorganisms are the most diverse organisms on  the planet. So, most diversity is amongst the microorganisms, and one of  the big questions now is what are all those microorganisms doing in the  environment? And so, today what I want to do with you is basically  explore this question of how we can actually figure out what those  microorganisms are all doing in environmental samples? So we can say we  are exploring the function of microbes in the environment.</p>
<p>At  first, I want to cover how we can actually identify them in the  environment. And I want to show you one specific example, and then I  want to talk about genomics in general, and then basically end with an  application of genomics to environmental questions. So, let's first talk  about the in-situ identification of microorganisms. And the basic  problem that I alluded to already before is that most microbes are only  known [SOUND OFF/THEN ON] -- from 16S ribosomal RNA clone libraries.</p>
<p>And  we basically want to search and identify them in the environment. OK,  and I'll show you a specific example of that later on. Now last time, we  said that the ribosomal RNA sequences consist really, like all gene  sequences, in fact. We identified several stretches of nucleotides,  types of stretches, that can be found. We said the A type stretches and B  type stretches that are very important for construction of phylogenetic  relationships, because we can align them and look for changes in the  nucleotide sequences because they are the same length and only differ in  mutation and single nucleotide base pair changes. But then there's also  those C type stretches, if you remember, and those we said vary at much  faster rates because they are not functionally constrained in those  genes.</p>
<p>OK, so they can actually also accumulate length changes.  And, it's these C type stretches that we can use sort of as diagnostic  sequence stretches for microorganisms. So, what we can say is we  identify organisms by the C type stretches, C type sequence stretches.  And we call those signature sequences.</p>
<p>OK, and they allow the  differentiation of closely related organisms, -- because they vary at  very fast rates between organisms. And the way we do this is that we  construct so-called phylogenetic probes. I should probably write this  over here. Now what are those phylogenetic probes? They're basically  short pieces of DNA that have a fluorescent molecule attached to them.  -- DNA molecules that are roughly 20 nucleotides in length, and they  carry a florescent molecule.</p>
<p>Now what the short, single-stranded  stretches of DNA basically are is they are complementary to those C type  sequence stretches -- -- in the ribosomal RNA. And so basically what we  can do is we can collect microbial cells from the environment -- --  make them permeable -- -- and then basically mix them with those  phylogenetic probes. And these probes will then permeate into the cell  and bind to their complementary sequences.</p>
<p>Then we wash away the  unbound probe -- -- and we can view it in a microscope under UV light.  Let me show you an example of this. What you see here is basically a  light micrograph. So this is what you see basically when you collect  microbial cells from the environment under the microscope. Most bacteria  look the same, so you cannot actually differentiate them all by just  looking at them. But then these cells were fixed and permeabilized and  then basically mixed with two different phylogenetic probes that  identified two different types of organisms. One was labeled with a red  Fluor, the other one with a green Fluor. And what you see is that you  can now differentiate those two organisms.</p>
<p>Now, why is this  especially interesting? Well here's just a specific example where people  were looking for bacteria capable of nitrogen oxidation. These are  bacteria that are very important in, for example, sewage treatment. And  it was known that there were two different types out there, one that  oxidizes ammonia to nitrite, -- and that a second one that oxidizes  nitrite to nitrate.</p>
<p>And by doing this type of analysis what people  basically learned is that those two organisms live in very, very close  proximity at all times. So the organisms that oxidized ammonia to  nitrite are really attached, and oftentimes even surround by the  organisms that take the nitrite to nitrate. So, where you have is a very  close cooperation between two different types of microorganisms, and  the transfer of one of the substrates that's a product of the metabolism  of one of the organisms to another one: so extremely efficient process  that really is very important to take into consideration when you want  to understand processes like sewer treatment, but also nitrogen  biogeochemistry and the environment. Any questions? OK, so for the  remainder of the lecture I want to talk about genomics, -- and then in  particular also its application to questions of environmental  microbiology and environmental science.</p>
<p>So first, what I want to  do is give you a little bit of the definition of genomics, and then  cover how it is actually possible that we can sequence entire genomes,  and I want to give you some highlights of what we have found by  comparing different genomes to each other. And then I want to talk about  this field about environmental genomics where we can use genomic  techniques to actually learn something about the function of different  uncultured microorganisms in the environment. So first, our definition,  it's basically to interpret or to sequence, -- interpret, and compare  whole genomes.</p>
<p>And as you will see the comparison part actually  plays an increasingly important role because we have now actually genome  sequences available from almost all, or from at least some of the major  groups of life. So this, again, is a different kind of representation  of the tree of life. You have bacteria, archaea, and eukarya again. And  as you can see, we have a lot of representatives. In fact, this doesn't  even come close to the diversity that we have now sequenced as well over  a hundred bacterial genome sequence now, several archeael genomes, and  increasingly also in eukaryotic genomes. Now, genomes, so how is this  done? How can we actually sequence genomes? Well, on the face of it we  use very large facilities where you have sequencing machines present.</p>
<p>There's  one very important one at MIT, actually at the Broad Institute, and  here you see all those really industrial scale production lines  actually. But the basic problem is that genomes are large. E. coli, for  example, has roughly 4.4 million base pairs, and the human genome is  even much, much larger. It has about 3 billion base pairs.</p>
<p>OK, so  genomes are very, very large. But a single sequencing reaction-- --  gives you only roughly 500-1,000 nucleotides or base pairs. So how is it  that we can actually sequence entire genomes? I'm going to walk you  through this, and there is some variation on the theme, but this is  still a major approach that's still used in some of the sequencing  facilities. Now, you start out by extracting genomic DNA from organisms,  and then you use restriction enzymes to cut the DNA into relatively  large pieces of DNA, so about 160 kilobase pairs long. On average, this  is shown here.</p>
<p>Kilo means a thousand, so 160,000 base pairs long.  These pieces are then cloned into specific cloning vectors that are  called BAC vectors. So therefore, cloning large pieces of DNA, and BAC  stands for Bacterial Artificial Chromosome. And what they basically are,  are plasmids, very special plasmids that can carry large pieces of  genome, or large genome fragments. So, by cloning into those BAC  vectors, what you do is you basically divide up the genome, and then the  step number three is mostly done for eukaryotic genomes because they  are so much larger.</p>
<p>You can actually map and analyze the  fragments, and map them onto genome maps where you know the location of  different restriction fragments and different genes, actually. For  bacteria, this step is mostly skipped, actually. What you do with each  one of those BACs, is you cut them further up into 1 kilobase per  fragment, so much smaller fragments. And these are called, and these are  cloned then into normal plasmid vectors. And so you generate what are  called shotgun clones.</p>
<p>So, these are then cloned into E. coli, you  go through the same type of steps that we discussed before already with  environmental clone libraries. And you can actually determine the  sequence of each one of those pieces of DNA. And what you will then get,  is small fragments of overlapping DNA sequences. That it shown here.  You'll find overlaps, basically, which piece together the whole genome.  And so, first to assemble, you piece together these genome fragments  that are present in the BACs, and then finally you piece together the  entire genome propose large sequence pieces, and you get a so-called  draft genome sequence. The next step in this analysis, then, is that you  do so-called genome annotation is.</p>
<p>And the first very important  step is that you translate the gene sequences into amino acids. So, the  nucleotide sequences into amino acids particularly in prokaryotes. This  step can be done right away -- -- and what this allows you to do, is you  can look for what we call open reading frames, or ORFs. And what you  look for is a start codon and a stop codon that basically branches or  frames a stretch of amino acids encoded by the nucleotides. So you look  for ORFs. And these are your putative genes.</p>
<p>The next step that  you can do, then, is you can go to databases and now you compare your  ORFs to information that is present in the databases. So basically, you  inquire the database and ask, is a gene sequence that is similar to the  one that I have statistically significantly similar present that allows  me to say something about the function of this particular gene? So  function, can then be identified by comparison with databases. Any  questions? OK, so that allows you, then, to basically say something  about the different genes that you have found in the genome, but to give  you an impression of how new this field really is and how little we  still know about the diversity of genes and organisms, on average when  we sequence a new bacterial genome we find about 30% of the genes, or a  third of the genes have no known functional analog of the databases.</p>
<p>OK, so there's a lot to learn about the diversity of life and about the functional diversity of life.</p>
<p>In  eukaryotes, there are some little twists, as you all know. And  basically, that is that genes of course consist of introns and exons,  right? And so it's basically relatively difficult to directly identify  those open reading frames. And what you have to do is that you have to  actually oftentimes, so let's write this down. And what people  oftentimes do, then, is that they search for matching sequences in  so-called cDNA libraries.</p>
<p>Now what are cDNA libraries? Let me just  show you this on the next slide. Skip this. Basically what you can do  is you can isolate messenger RNA from cells and that translate the  messenger RNA by a process called reverse transcription that the viral  enzyme that translates RNA into DNA, so you can translate it into DNA  fragments. And you can then clone those DNA fragments into plasmids,  sequence those, and then basically see what are the pieces that are  actually, what are the introns in the genes? What are the pieces that  are excised when the messenger RNA is actually created from the genome?  And so, let me just cover now a few of the major insights that people  have come up with.</p>
<p>Of course, it's a very growing field and a lot  of excitement is coming out. And I first want to talk about bacteria and  archaea -- -- and then say a few words also about eukaryotes or  eukaryote. First of all, what we learned about, bacteria and archaea, is  that their genomes are very compact. Whenever they have pieces of DNA  that are not frequently used, they're actually lost from the genome.</p>
<p>OK,  so they lose genes, I should say, relatively easily, and we can see  this that the genome size is correlated to metabolic diversity. So, for  example, we have Mycoplasma genetalium and Streptomyces -- coelicor are  two very different bacteria. The first one is an obligate intracellular  parasite. OK, so, which means it's actually bathed in a nutrient  solution in the eukaryotic cells that it invades.</p>
<p>It doesn't have  to make amino acids. It gets it just from the host cell. And it turns  out it has a very small genome, so only 0.58-based mega-base pairs, so  580,000 base pairs, and only 517 genes. And interestingly, actually  people are now using this organism to try and ask, well, what's the  minimum number of genes that organism can actually will live with? And  so, they are deleting in a stepwise fashion the different genes in this  organism, and it turns out that you need about two to 300 genes minimum  in order to make the things survive. On the other hand, streptomyces is a  soil bacterium -- -- has a very complex lifestyle, can degrade a lot of  environmental substrates, and it has a very big genome, one of the  biggest bacterial genomes.</p>
<p>And so, those two organisms basically  span pretty much the range of bacterial genome sizes. And so, it's  thought that it has about 7,846 genes. Now, we also have a very large  genetic diversity -- -- between species. And typically what you find is  that roughly 15 to 30% of genes are unique to a specific species. And  that's really because bacteria and archaea have the capability to affect  a lot of chemical reactions that eukaryotes, for example, cannot.</p>
<p>There's  about 20 million known organic substances, organic chemicals, and  almost all of them are biodegradable by bacteria. Even the minutest  compounds if it were not biodegradable bacteria, would build up in the  environment, OK? So, if it just where a cofactor that some organism  produces because we have such a long period of time of evolution on this  planet and evolutionary history, you probably would be able to dig it  up in your backyard. One of the other very important and interesting  insights that has come out with comparing genomes for microorganisms is  that lateral gene transfer is a very important process amongst  microorganisms.</p>
<p>Now what do I mean by lateral gene transfer? It  basically means that we find evidence among bacterial genomes that they  have actually taken genes from completely unrelated organisms. And I  just want to show you one example here from that of thermotoga maritima  -- -- which lives in hot springs.</p>
<p>This is a very interesting  bacterium that lives in hot water of around 80?C and thrives only in  those kinds of environments. And they coexist there with many archaea.  And when people sequenced the genome of thermotoga maritima what they  found was that about 25% of the genes have their closest relatives in  archaeal genomes. So roughly 25% of genes in thermotoga are of archaeal  origin. And how can we actually figure something like that out? Well,  the most important technique is, again, phylogenetic tree construction.</p>
<p>And  so when you have, for example, gene A, well let me draw this, actually,  on a new board. So you're comparing, say, three organisms, organism A,  B, and C and you compare gene one with gene two. And you notice that  most genes adhere to this pattern, but that every now and then there's a  gene that gives you this type of pattern. What you can then conclude is  that this gene, C, has not coevolved with the other genes in the genome  of these organisms but was actually transferred into it from another  source.</p>
<p>And I don't have time to go actually into the mechanisms.  If you're interested, I teach a graduate class that undergraduates  actually take in our department, environmental microbiology, where we  discussed a lot of the mechanisms. It's basically a lot of viruses can  affect gene transfer but also plasmids and transposons.</p>
<p>But for  bacteria, again, you should remember that often new function is actually  oftentimes arises by lateral gene transfer. And one of the interesting  things is that lateral gene transfer is actually very important in the  evolution of pathogenic bacteria. So, the so-called virulence genes,  which are the genes that basically affect pathogenesis. Do you have a  question? Among pathogenic bacteria, often arise by lateral gene  transfer.</p>
<p>OK. Any questions? OK, now for eukarya, I just want to  make the point that their genomes are generally orders of magnitudes  larger -- OK, and that the exons, so the stretches that really encode  the protein that make up the organism, the exons are only typically a  few percent of the genome. That's particularly in higher eukaryotes.  Yeasts, for example, have a much more compact genome also. We, for  example, are full of DNA that people still have a very hard time  figuring out what that actually does. But it seems that the majority of  the genome, so-called repeated sequences -- -- many of which seems to be  ancient retroviruses that have inserted themselves into the genome and  have since then lost actually their function.</p>
<p>OK, so the remaining  time I want to just give you an example of how we can now use these  techniques that I outlined before to learn something about  microorganisms in the environment. It's called environmental. Basically,  the way this all started was by going into the environment and  extracting nuclear gases and treating them exactly the same way as if  you had a single genome. But, again, remember, we have a very large  mixture of microorganisms present in the environment. And where this is  mostly done was in the ocean, actually. And what people did, was they  constructed those BAC clones directly from the environment and then  looked amongst those BAC clones for specific 16S ribosomal RNA genes.  Remember, this is the marker that we have for microorganisms in the  environment.</p>
<p>We know the diversity of microorganisms through those  types of genes, and we have a lot of the data available. And so, in  order to link a specific function of such an organism that we only know  from the 16S ribosomal RNA genes. So, to ask the question of what much  of this organism might be carrying out in the environment, it's very  useful to sequence BAC clones that have 16S ribosomal RNA genes on them,  and determine what kinds of protein coding genes are on there that  might reveal some of the function of the organism in the environment.  And one example that I want to show you is that of the proteorhodopsin.  So basically, the initial task was to sequence BAC clones containing  ribosomal RNA genes, and look for other genes that might reveal some of  the function.</p>
<p>So, you don't want to look for all the genes that  encode proteins that are important to the cell cycle and things like  that, but really sort of metabolic genes that might tell you something  about the type of metabolism that this organism carries out in the  environment. And so, what the first example that turned out to be  really, really important is that people found rhodopsin genes on one of  those BAC fragments, and it turns out this rhodopsin catalyzes or these  rhodopsin genes produce a protein that inserts itself into the bacterial  membrane, and it's a photoreceptor that when it's hit by light, it  actually becomes a proton pump.</p>
<p>So, it expels protons from the  cell interior to the outside, and you already know that this is  important in energy generation in all living cells. So proton gradient  across membranes basically give the cells sort of a battery status that  can be exploited by ATPase molecules or ATPase proteins that equalize  the proton gradient and affect ATP synthesis in doing so. Now, why is  this so important? Well, it turned out that this type of protein is  present in almost all microbial cells that were previously thought to be  heterotrophs alone in the ocean in the parts of the ocean that receive  enough life.</p>
<p>And what this means is that our estimates of the  global carbon budget of the ocean were basically wrong because most  microorganisms in the ocean have this. So most prokaryotes in the ocean  have a light-driven proton pump which is called proteorhodopsin.</p>
<p>And  it basically allows them to gain energy from sunlight. And there's an  increasing number of such examples now where we are learning to  interpret environmental communities, and the function of environmental  microbial communities through those genomic approaches. And it reveals  basically an enormous diversity of organisms out there. And what we also  are learning to do now is to assemble entire genomes from those samples  by applying genomic techniques. And this is an example here where you  see, this was published last year, where people went out and basically  were able to piece together from pieces of genes obtained from the  environment, entire genomes or fragments of entire genomes.</p>
<p>And  that's shown here. Those are contiguous sequences. OK, so if you have  any questions let me know by e-mail, or if you're interested in pursuing  this further I also teach another class in civil and environmental  engineering.</p>
</div>
        <div id="vid_related" itemprop="description" class="tabContent hide">
<h2 class="subhead">Free Downloads</h2>
<h3 class="subsubhead">Video</h3>
<ul>
<li>iTunes U (<a href="http://deimos3.apple.com/WebObjects/Core.woa/Browse/mit.edu.1329584553.01329584560.1329911955?i=2014086142">MP4 - 118MB</a>)</li>
<li>Internet Archive (<a href="http://www.archive.org/download/MIT7.014S05/ocw-7.014-32-02may05-220k_512kb.mp4">MP4 - 207MB</a>)</li>
</ul>
<br><h3 class="subsubhead">Subtitle</h3>
<ul><li>English - US (<a href="../../../contents/video-lectures/32-molecular-evolution/7.014-2005-L32.srt">SRT</a>)</li></ul>
<br>
</div>
    
   </div>  


      					 
        <div class="" id="parent-fieldname-bottom_html_area">
            
            
        </div>
    
                    </div>
<!--Course_inner_chip tag close -->
           		</div>
<!--Course_wrapper tag close --> 
            </div>
<!--left tag close -->
            <div id="right">
                <!--Begin Right Portion -->
                    <div>
    
<div id="portletwrapper-6f63772e7269676874746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a646f6e617465" class="portletWrapper kssattr-portlethash-6f63772e7269676874746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a646f6e617465">
<div class="portletStaticText portlet-static-donate"><p class="zero"><a href="http://ocw.mit.edu/donate"><img src="../../../common/images/button_donate-now.png" alt="Donate Now." class="donate"></a></p></div>

</div>




</div>

                	<div>
    



</div>


        <div class="" id="parent-fieldname-rsi_top_html_area">
            
            
        </div>
    

<!-- RSI google ad space-->


<div id="google_ads">    
    <script type="text/javascript" src="http://partner.googleadservices.com/gampad/google_service.js"></script><script type="text/javascript">GS_googleAddAdSenseService("ca-pub-6588555046597237");GS_googleEnableAllServices();</script><script type="text/javascript">GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_A_DL");GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_B_DL");GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_C_DL");</script><script type="text/javascript">GA_googleFetchAds();</script><script language="javascript" type="text/javascript">
GA_googleAddAttr("TYPE","HOUSE");
GA_googleAddAttr("DEPARTMENT","7");
GA_googleAddAttr("CRS_BEG2","01");
GA_googleAddAttr("CRS_END","4");
GA_googleAddAttr("SESSION","S");
GA_googleAddAttr("YEAR","05");
</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_A_DL");</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_B_DL");</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_C_DL");</script>
</div>

<!-- End RSI ads--> 

<div>
    



</div>

            </div>
<!--Right div close -->
            <div class="clear"></div> 
        </div>
<!--grid tag close --> 
      </div>
		
		<div id="bottom">
			<div id="grid">
				
<div id="portletwrapper-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572" class="portletWrapper kssattr-portlethash-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572">
<div class="portletStaticText portlet-static-site-footer"><div id="bottom">
<div id="grid">
<!-- *begin footer* -->
<div id="footer" role="navigation sitemap">
<div id="foot-c1" class="grid_2 alpha">
<h4 class="footer">Courses</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/courses/find-by-topic/">Find by Topic</a></li>
    <li><a href="http://ocw.mit.edu/courses/find-by-number/">Find by Course Number</a></li>
    <li><a href="http://ocw.mit.edu/courses/find-by-department/">Find by Department</a></li>
    <li><a href="http://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/translated-courses/">Translated Courses</a></li>
    <li><a href="http://ocw.mit.edu/resources/">Supplemental Resources</a></li>
</ul>
</div>
<div id="foot-c2" class="grid_2">
<h4 class="footer">About</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/about/">About OpenCourseWare</a></li>
    <li><a href="http://ocw.mit.edu/about/site-statistics/">Site Stats</a></li>
    <li><a href="http://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
    <li><a href="http://ocw.mit.edu/about/ocw-consortium/">OCW Consortium</a></li>
    <li><a href="http://ocw.mit.edu/about/media-coverage/">Media Coverage</a></li>
    <li><a href="http://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
    <li><a href="http://ocw.mit.edu/about/media-coverage/press-releases/">Press Releases</a></li>
    <li><a href="http://ocw.mit.edu/about/next-decade/">OCW's Next Decade</a></li>
</ul>
</div>
<div id="foot-c3" class="grid_2">
<h4 class="footer">Donate</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/donate">Make a Donation</a></li>
    <li><a href="http://ocw.mit.edu/donate/why-donate/">Why Donate?</a></li>
    <li><a href="http://ocw.mit.edu/donate/ocw-course-champions-program/">Become a Course Champion</a></li>
    <li><a href="http://ocw.mit.edu/donate/our-supporters/">Our Supporters</a></li>
    <li><a href="http://ocw.mit.edu/donate/other-ways-to-contribute/">Other Ways to Contribute</a></li>
    <li><a href="http://ocw.mit.edu/donate/shop-ocw/">Shop OCW</a></li>
    <li><a href="http://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
</ul>
</div>
<div id="foot-c4" class="grid_2">
<h4 class="footer">Featured Sites</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/high-school/">Highlights for High School</a></li>
    <li><a href="https://www.edx.org/school/mitx/allcourses">MITx</a></li>
    <li><a href="http://teachingexcellence.mit.edu/">Teaching Excellence at MIT</a></li>
</ul>
<h4 class="footer" style="margin-top: 14px;">Tools</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/help/">Help &amp; FAQs</a></li>
    <li><a href="../../../common/jsp/feedback.htm">Contact Us</a></li>
    <li><a href="../../../common/search/AdvancedSearch.htm">Advanced Search</a></li>
    <li><a href="http://ocw.mit.edu/help/site-map/">Site Map</a></li>
    <li><a href="../../../common/terms/index.htm">Privacy &amp; Terms of Use</a></li>
</ul>
</div>
<div id="foot-c5" class="grid_4 omega" itemprop="publisher" itemscope="" itemtype="http://schema.org/CollegeOrUniversity">
<h4 class="footer">About <span itemprop="name">MIT OpenCourseWare</span>
</h4>
<p class="accent" itemprop="description">MIT OpenCourseWare makes the materials used in the teaching of almost all of MIT's subjects available on the Web, free of charge. With more than 2,000 courses available, OCW is delivering on the promise of open sharing of knowledge.</p>
</div>
<div id="foot-copy" class="grid_8 alpha">
<a href="http://web.mit.edu"><img src="../../../common/images/logo_mit.png" alt="Massachusetts Institute of Technology logo and name." style="width: 195px; height: 44px;"></a>                 <a href="http://www.ocwconsortium.org/"><img src="../../../common/images/logo_ocwc.gif" alt="OpenCourseWare Consortium logo." style="width: 80px; height: 44px;"></a>                 <a rel="license" itemprop="useRightsUrl" href="http://creativecommons.org/licenses/by-nc-sa/3.0/us/deed.en_US"><img src="../../../common/images/cc_by-nc-sa.png" alt="Creative Commons logo with terms BY-NC-SA." style="width: 126px; height: 44px;"></a>
<p class="copyright">© 2001–2013<br>
Massachusetts Institute of Technology</p>
<p style="font-size: 0.9em; margin-bottom: 15px;">Your use of the MIT OpenCourseWare site and materials is subject to our <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/us/deed.en_US" rel="license">Creative Commons License</a> and other <a href="../../../common/terms/index.htm" rel="cc:morePermissions">terms of use</a>.</p>
</div>
<div id="foot-support" class="grid_4 omega"> </div>
</div>
</div>
</div></div>

</div>





                
			</div> <!-- bottom grid end -->
		</div>
<!-- bottom end -->
		
		
   </body>
</html>
