<!DOCTYPE html><html lang="en">
<head>
<meta charset="utf-8">
<meta name="format-detection" content="telephone=no">
<title>13: Recombinant DNA 1 | Audio Lectures | Introductory Biology | Biology | MIT OpenCourseWare</title>
<!-- Begin Automatic Metadata Insertion --><meta content="7-013-introductory-biology-spring-2006" name="WT.cg_n">
<meta content="13: Recombinant DNA 1" name="WT.cg_s">
<meta content="" name="Description">
<meta content="Sive, Hazel" name="Author">
<meta content="Jacks, Tyler" name="Author">
<meta content="biology,biochemistry,genetics,molecular biology,recombinant DNA,cell cycle,cell signaling,cloning,stem cells,cancer,immunology,virology,genomics,molecular medicine,DNA,RNA,proteins,replication,transcription,mRNA,translation,ribosome,nervous system,amino acids,polypeptide chain,cell biology,neurobiology,gene regulation,protein structure,protein synthesis,gene structure,PCR,polymerase chain reaction,protein localization,endoplasmic reticulum,human biology,inherited diseases,developmental biology,evolution,human genetics,human diseases,infectious agents,infectious diseases,Biology" name="keywords">
<meta content="13: Recombinant DNA 1 | MIT OpenCourseWare" name="Search_Display">
<meta content="Biology" itemprop="about">
<!-- End Automatic Metadata Insertion --><link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/grid.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/base.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/menu.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/jquery.bubblepopup.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/courses.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/courses_new.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/jquery.jscrollpane.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/media_tabs.css">
<link href="http://ocw.mit.edu/xml/ocwcc.rdf" type="application/rdf+xml" rel="metadata">
<link rel="canonical" href="http://ocw.mit.edu/courses/biology/7-013-introductory-biology-spring-2006/audio-lectures/13-recombinant-dna-1">
<link rel="apple-touch-icon" href="../../../common/images/apple-touch-icon.png">
<script type="text/javascript" src="../../../common/scripts/common-utils-offline.js"></script><script type="text/javascript" src="../../../common/scripts/ocw-offline.js"></script><script type="text/javascript" src="../../../common/scripts/jquery.bubblepopup.min.js"></script><script type="text/javascript" src="../../../common/scripts/jquery-ui.min.js"></script><script type="text/javascript" src="../../../common/scripts/jquery.jscrollpane.min.js"></script><script type="text/javascript" src="../../../common/scripts/bubble-popup-offline.js"></script><script type="text/javascript">
      $(document).ready(function() {
        $("#tabs").tabs();
        IpadScroller();
      });
    </script>
</head>
<body itemscope itemtype="http://schema.org/WebPage">
        
	

        <div id="top">
			<div id="grid">
				
				
					
<div id="portletwrapper-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572" class="portletWrapper kssattr-portlethash-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572">
<div class="portletStaticText portlet-static-site-header">
<div id="banner" role="banner" class="grid_6 alpha"><a href="http://ocw.mit.edu/"><img src="../../../common/images/ocw_mast.png" alt="MIT OpenCourseWare, Massachusetts Institute of Technology" class="logo"></a></div>
<form action="http://ocw.mit.edu/subscribe/index.htm?utm_source=header" method="get">
    <div id="subscribe" role="form toolbar" class="grid_6 omega">
    <div class="module">
    <table class="social"><tbody><tr>
<td class="text">Subscribe to the<br>
                OCW Newsletter</td>
                <td class="black"><input type="text" onclick="clearEmailBox()" onblur="fillEmailBox()" value="Enter Email" class="greytext emailField" id="email" name="fromemail"></td>
                <td class="black"><input type="image" src="../../../common/images/button_subscribe.png" alt="Subscribe" class="sub_button"></td>
                <td>
<a href="http://facebook.com/mitocw"><img src="../../../common/images/icon_fb.png" alt="Click to visit our Facebook page."></a><a href="http://twitter.com/mitocw"><img src="../../../common/images/icon_tw.png" alt="Click to visit our Twitter feed."></a>
</td>
            </tr></tbody></table>
</div>
    <p class="helplinks"><a href="http://ocw.mit.edu/help">Help</a>   |   <a href="../../../common/jsp/feedback.htm">Contact Us</a></p>
    </div>
    <div class="clear"> </div>
</form>
</div>

</div>





<div id="mega" role="navigation" class="grid_8 alpha">        
	<ul id="menu">
<li id="menu_home">
            <a href="http://ocw.mit.edu/"><img src="../../../common/images/top-nav_home.png" class="home_icon" alt="Click for site home page."></a><!-- Begin Home Item -->
        </li>
<!-- End Home Item -->        
        <li class="selected">
            <a href="#" class="drop">Courses</a><!-- Begin 5 columns Item -->
            <div class="dropdown_5columns-a mega-courses">                    
                <div class="col_1a">
                    <div class="row_1a">
                        <div class="quart">
                            <h2 class="nav">Find courses by:</h2>
                            <ul class="nav-bullet find_by">
<li><a href="http://ocw.mit.edu/courses/find-by-topic/">Topic</a></li>
                                <li><a href="http://ocw.mit.edu/courses/find-by-number/">MIT Course Number</a></li>
                                <li><a href="http://ocw.mit.edu/courses/find-by-department/">Department</a></li>
                            </ul>
</div>
                        <div class="quart">
                            <h2 class="nav">Collections</h2>
                            <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Lectures</a></li>
                                <li><a href="http://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
                                <li><a href="http://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
                                <li><a href="http://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
                                <li><a href="http://ocw.mit.edu/resources">Supplemental Resources</a></li>
                            </ul>
</div>
                        <div class="clear"> </div>
                    </div>
                    <div class="row_1b">
                        <h2 class="nav">Cross-Disciplinary Topic Lists</h2>
                        <div class="quart">
                        <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/energy-courses">Energy</a></li>
                            <li><a href="http://ocw.mit.edu/courses/entrepreneurship">Entrepreneurship</a></li>
                            <li><a href="http://ocw.mit.edu/courses/environment-courses">Environment</a></li>
                        </ul>
</div>    
                        <div class="quart">
                        <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/intro-programming">Introductory Programming</a></li>
                            <li><a href="http://ocw.mit.edu/courses/life-sciences">Life Sciences</a></li>
                            <li><a href="http://ocw.mit.edu/courses/transportation-courses">Transportation</a></li>
                        </ul>
</div>
                        <div class="clear"> </div>
                    </div>
                    <div class="clear"> </div>
                </div>
                <div class="col_1b">
                    <h2 class="nav">Translated Courses</h2>
                    <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/translated-courses/traditional-chinese">繁體字 / Traditional Chinese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/simplified-chinese">简体字 / Simplified Chinese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/spanish">Español / Spanish</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/portuguese">Português / Portuguese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/thai">ภาษาเขียน / Thai</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/persian">فارسی / Persian</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/turkish">Türkçe / Turkish</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/korean">(비디오)한국 / Korean</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses">More...</a></li>
                    </ul>
</div>
            </div>
        </li>
        <li>
            <a href="" class="drop">About</a>
            <div class="dropdown_1column-a">
                <div class="col_1">
                    <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/">About MIT OpenCourseWare</a></li>
                    </ul>
<ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/site-statistics/">Site Stats</a></li>
                        <li><a href="http://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
                        <li><a href="http://ocw.mit.edu/about/ocw-consortium/">OpenCourseWare Consortium</a></li>
                    </ul>
<ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/media-coverage/">Media Coverage</a></li>
                        <li><a href="http://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
                        <li><a href="http://ocw.mit.edu/about/media-coverage/press-releases/">Press Releases</a></li>
                    </ul>
<ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/about/next-decade/">OCW's Next Decade</a></li>
                    </ul>
</div>
            </div>  
        </li>    
        <li>
            <a href="" class="drop">Donate</a>        
            <div class="dropdown_1column-a">
                    <div class="col_1">
                        <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/donate/">Make a Donation</a></li>
                            <li><a href="http://ocw.mit.edu/donate/why-donate/">Why Donate?</a></li>
                            <li><a href="http://ocw.mit.edu/donate/ocw-course-champions-program/">Become a Course Champion</a></li>
                            <li><a href="http://ocw.mit.edu/donate/our-supporters/">Our Supporters</a></li>
                            <li><a href="http://ocw.mit.edu/donate/other-ways-to-contribute/">Other Ways to Contribute</a></li>
                            <li><a href="http://ocw.mit.edu/donate/shop-ocw">Shop OCW</a></li>
                        </ul>
<ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
                        </ul>
</div>
            </div>            
        </li>        
        <li>
            <a href="" class="drop">Featured Sites</a>        
            <div class="dropdown_1column-a">
                <div class="col_1">
                    <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/high-school/">Highlights for High School</a></li>
                    </ul>
<ul class="nav-bullet mega-div-top">
<li><a href="https://www.edx.org/school/mitx/allcourses">MITx</a></li>
                        <li><a href="http://teachingexcellence.mit.edu/">Teaching Excellence at MIT</a></li>
                    </ul>
</div>
            </div>            
        </li>
    </ul>
</div>
<div id="search" role="search" class="grid_4 omega">
    
    <form method="get" action="../../../common/search/AdvancedSearch.htm">
     	 <table class="search"><tbody><tr>
<td class="black"><input type="text" onblur="fillSearchBox()" onfocus="clearSearchBox()" maxlength="255" value="Search" name="q" class="greytext searchField" id="terms"></td> 			 
                    <td class="black"><input type="image" src="../../../common/images/button_search.png" name="btnG" class="sub_button"></td>			 
                    <td class="text2"><a href="../../../common/search/AdvancedSearch.htm">Advanced<br>Search</a></td>
                </tr></tbody></table>
</form>
</div>
<div class="clear"></div>
<!-- *end header* -->  

				
				
			</div>
<!-- top grid end -->
		</div>
<!-- top end -->
			
		<div id="center_media">
      	<div id="grid">
      		<div id="left">
        		<div id="breadcrumb_media">
                	<p>

    <a href="http://ocw.mit.edu/">Home</a>
    
        »
        
    
    
        
            <a href="http://ocw.mit.edu/courses">Courses</a>
            
                »
                
            
            
         
    
    
        
            <a href="http://ocw.mit.edu/courses/biology">Biology</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../../contents/index.htm">Introductory Biology</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../../contents/audio-lectures/index.htm">Audio Lectures</a>
            
                »
                
            
            
         
    
    
        
            
            
            13: Recombinant DNA 1
         
    
</p>

            	</div>
            	<div class="clear"></div>
        		<div id="media_title">
        		<h1 class="title" itemprop="name" property="dct:title">
        <span class="" id="parent-fieldname-title">
            13: Recombinant DNA 1
        </span>
    </h1>
        		</div>
           		<div class="clear"></div>
           		<div id="course_wrapper_media">
           			<div id="course_nav">
           				<script language="javascript" type="text/javascript">
function toggleMenu(objID) {
  if (!document.getElementById) return;
  var ob = document.getElementById(objID);
  ob.className = (ob.className == 'selected')?'': 'selected';
}
function toggleClass(id)
{
  var divtoggleClass= document.getElementById(id);
  divtoggleClass.className = (divtoggleClass.className == 'mO')?'mC': 'mO';
  return false;
}
function changeAlt(id)
{
  id.alt = (id.alt == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
  id.title = (id.title == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
}
</script><!--Left Nav Starts --><ul>
<li class="">
			   			<a href="../../../contents/index.htm">
		                  Course Home  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/syllabus/index.htm">
		                  Syllabus  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/calendar/index.htm">
		                  Calendar  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/readings/index.htm">
		                  Readings  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/recitations/index.htm">
		                  Recitations  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/assignments/index.htm">
		                  Assignments  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/exams/index.htm">
		                  Exams  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/study-materials/index.htm">
		                  Study Materials  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="selected">
			   			<a href="../../../contents/audio-lectures/index.htm">
		                  Audio Lectures  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/video-lectures/index.htm">
		                  Video Lectures  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    
		    
         	
	<!--second tal block close-->  
	
</ul>
<!--Left Nav Ends --><script type="text/javascript" id="openstudy-widget-loader" src="http://openstudy.com/javascripts/load.js?type=feed&amp;color=auto&amp;studygroup=MIT%20OCW%20Biology&amp;env=openstudy.com"></script>
</div>
           			<div id="course_inner_media">
      					 
        <div class="" id="parent-fieldname-text">
            
            
        </div>
    
      					 

     <div id="media-embed">
         <div class="attention_message" id="embed_1">
<p>Flash and JavaScript are required for this feature.</p>
<p>Download the track from <a href="http://deimos3.apple.com/WebObjects/Core.woa/Browse/mit.edu-dz.2820703204?i=1664446236">iTunes U</a> or the <a href="http://www.archive.org/download/MIT7.013S05/Lecture08.mp3">Internet Archive</a>.</p>
</div>
     </div>
    
     <script type="text/javascript">ocw_embed_chapter_media('embed_1', 'http://www.archive.org/download/MIT7.013S05/Lecture08.mp3', 'sound', '/courses/biology/7-013-introductory-biology-spring-2006/audio-lectures/13-recombinant-dna-1', 'http://ocw.mit.edu/images/audio_only_slate.gif',0,0, 'http://ocw.mit.edu/courses/biology/7-013-introductory-biology-spring-2006/audio-lectures/13-recombinant-dna-1/7.013-2006-L13.srt')</script><div id="media_resource_next_prev_nav" style="margin-top: 1em;">
        <p>
        
            <a href="../../../contents/audio-lectures/12-molecular-biology-3/index.htm">
                <img src="../../../common/images/btn_previous_resource.png" style="margin: 0 30px 0 50px;" alt="Previous track" title="Previous track"></a>
     	
     	
        
            <a href="../../../contents/audio-lectures/14-recombinant-dna-2/index.htm"> 
                <img src="../../../common/images/btn_next_resource.png" alt="Next track" title="Next track"></a>
       
       </p>
     </div>
 


<script type="text/javascript">
		window.onload=function(){
		init();
		
		}
		var tabLinks = new Array();
		var contentDivs = new Array();
		function init() {
		  // Grab the tab links and content divs from the page
		  var tabListItems = document.getElementById('tabs').childNodes;
		  for ( var i = 0; i < tabListItems.length; i++ ) {
			if ( tabListItems[i].nodeName == "LI" ) {
			  var tabLink = getFirstChildWithTagName( tabListItems[i], 'A' );
			  var id = getHash( tabLink.getAttribute('href') );
			  tabLinks[id] = tabLink;
			  contentDivs[id] = document.getElementById( id );
			}
		  }
		  // Assign onclick events to the tab links, and
		  // highlight the first tab
		  var i = 0;
		  for ( var id in tabLinks ) {
			tabLinks[id].onclick = showTab;
			tabLinks[id].onfocus = function() { this.blur() };
			if ( i == 0 ) tabLinks[id].className = 'selected';
			i++;
		  }
		  // Hide all content divs except the first
		  var i = 0;
		  for ( var id in contentDivs ) {
			if ( i != 0 ) contentDivs[id].className = 'tabContent hide';
			i++;
		  }
		}
		function showTab() {
		  var selectedId = getHash( this.getAttribute('href') );
		  // Highlight the selected tab, and dim all others.
		  // Also show the selected content div, and hide all others.
		  for ( var id in contentDivs ) {
			if ( id == selectedId ) {
			  tabLinks[id].className = 'selected';
			  contentDivs[id].className = 'tabContent';
			} else {
			  tabLinks[id].className = '';
			  contentDivs[id].className = 'tabContent hide';
			}
		  }
		  // Stop the browser following the link
		  return false;
		}
		function getFirstChildWithTagName( element, tagName ) {
		  for ( var i = 0; i < element.childNodes.length; i++ ) {
			if ( element.childNodes[i].nodeName == tagName ) return element.childNodes[i];
		  }
		}
		function getHash( url ) {
		  var hashPos = url.lastIndexOf ( '#' );
		  return url.substring( hashPos + 1 );
		}
 </script><div id="media_tabs">
     
        <ul id="tabs">
<li class="first">
                <a href="#vid_about" class="selected">About this Track</a>
            </li>
            <li class="">
                <a href="#vid_index" class="">Playlist</a>
            </li>
            <li class="">
                <a href="#vid_playlist" class="">Transcript</a>
            </li>
            <li class="">
                <a href="#vid_related" class="">Download this Track</a>
            </li>
        </ul>
<div id="vid_about" itemprop="description" class="tabContent"><p><strong>Lecturer:</strong> Prof. Tyler Jacks</p></div>
        <div id="vid_index" itemprop="description" class="tabContent hide">
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/3-biochemistry-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="3: Biochemistry 1" alt="3: Biochemistry 1"><p>3: Biochemistry 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/5-biochemistry-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="5: Biochemistry 3" alt="5: Biochemistry 3"><p>5: Biochemistry 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/7-genetics-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="7: Genetics 2" alt="7: Genetics 2"><p>7: Genetics 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/8-genetics-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="8: Genetics 3" alt="8: Genetics 3"><p>8: Genetics 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/9-genetics-4/index.htm">
<img src="../../../common/images/audio_file.gif" title="9: Genetics 4" alt="9: Genetics 4"><p>9: Genetics 4</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/11-molecular-biology-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="11: Molecular Biology 2" alt="11: Molecular Biology 2"><p>11: Molecular Biology 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/12-molecular-biology-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="12: Molecular Biology 3" alt="12: Molecular Biology 3"><p>12: Molecular Biology 3</p></a>
</div>
<div class="related-media-thumbnail-nolink">
<div class="now-playing-resource">Now Playing</div>
<img src="../../../common/images/audio_file.gif" title="13: Recombinant DNA 1" alt="13: Recombinant DNA 1"><p>13: Recombinant DNA 1</p>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/14-recombinant-dna-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="14: Recombinant DNA 2" alt="14: Recombinant DNA 2"><p>14: Recombinant DNA 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/15-recombinant-dna-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="15: Recombinant DNA 3" alt="15: Recombinant DNA 3"><p>15: Recombinant DNA 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/16-cell-biology-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="16: Cell Biology 1" alt="16: Cell Biology 1"><p>16: Cell Biology 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/17-cell-biology-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="17: Cell Biology 2" alt="17: Cell Biology 2"><p>17: Cell Biology 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/19-steps-in-development/index.htm">
<img src="../../../common/images/audio_file.gif" title="19: Steps in Development" alt="19: Steps in Development"><p>19: Steps in Development</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/20-fertilization-fertility/index.htm">
<img src="../../../common/images/audio_file.gif" title="20: Fertilization/Fertility" alt="20: Fertilization/Fertility"><p>20: Fertilization/Fertility</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/21-cell-type-and-position/index.htm">
<img src="../../../common/images/audio_file.gif" title="21: Cell Type and Position" alt="21: Cell Type and Position"><p>21: Cell Type and Position</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/22-morphogenesis/index.htm">
<img src="../../../common/images/audio_file.gif" title="22: Morphogenesis" alt="22: Morphogenesis"><p>22: Morphogenesis</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/23-stem-cells/index.htm">
<img src="../../../common/images/audio_file.gif" title="23: Stem Cells" alt="23: Stem Cells"><p>23: Stem Cells</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/24-cloning/index.htm">
<img src="../../../common/images/audio_file.gif" title="24: Cloning" alt="24: Cloning"><p>24: Cloning</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/25-nervous-system-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="25: Nervous System 1" alt="25: Nervous System 1"><p>25: Nervous System 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/26-nervous-system-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="26: Nervous System 2" alt="26: Nervous System 2"><p>26: Nervous System 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/27-nervous-system-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="27: Nervous System 3" alt="27: Nervous System 3"><p>27: Nervous System 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/28-cancer-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="28: Cancer 1" alt="28: Cancer 1"><p>28: Cancer 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/29-cancer-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="29: Cancer 2" alt="29: Cancer 2"><p>29: Cancer 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/30-cancer-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="30: Cancer 3" alt="30: Cancer 3"><p>30: Cancer 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/31-immunology-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="31: Immunology 1" alt="31: Immunology 1"><p>31: Immunology 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/32-immunology-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="32: Immunology 2" alt="32: Immunology 2"><p>32: Immunology 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/34-virology-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="34: Virology 2" alt="34: Virology 2"><p>34: Virology 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/35-prion-disease/index.htm">
<img src="../../../common/images/audio_file.gif" title="35: Prion Disease" alt="35: Prion Disease"><p>35: Prion Disease</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/36-future-of-biology/index.htm">
<img src="../../../common/images/audio_file.gif" title="36: Future of Biology" alt="36: Future of Biology"><p>36: Future of Biology</p></a>
</div>
</div>
        <div id="vid_playlist" itemprop="description" class="tabContent hide">
<ul><li><a class="transcript-link" title="Open in a new window." alt="Open in a new window." style="text-decoration: none; font-size: 1.0em;" target="_blank" text-decoration: none font-size: href="../../../contents/audio-lectures/7_0132006L13.pdf"> Download this transcript - PDF (English - US)</a></li></ul>
<p></p>
<p> </p>
<p>We're going to talk today about recombinant DNA. This is a new  section of the class. We'll spend three lectures talking about  recombinant DNA, just sort of a practical how-to guide with respect to  manipulating DNA now that you've learned a bit about it and how we use  it in research and in other aspects of medicine and agriculture and so  on.</p> <p>Now, recombinant DNA has been around for a long time.</p> <p>And, frankly, it's got a bit of a bad rap. When people in the general  public hear the term recombinant DNA they sort of conjure up these  ideas of mad scientists going around, you know, cloning bizarre  organisms or creating dangerous forms of life.</p> <p>Although that's possible, it's not what we typically do with this technology.</p> <p>In fact, it has some rather benign and other quite important  applications. So, Claudette, I'm not going to get this board, am I? OK.</p> <p>Basically what we're going to be talking about is genetic engineering.</p> <p>That ought to make the budding engineers in the crowd happy. We're  going to manipulate the DNA of organisms and actually transfer DNA from  one organism to another for various purposes. There are three general  tenants that you need to pay attention to here.</p> <p>The basic goal is to isolate, from a complex genome, a piece of DNA of interest.</p> <p>And amplify that piece of DNA, which could be a specific gene that  you're interested in, or it might be some other sequence within that  DNA.</p> <p>It doesn't have to necessarily be a coding gene that you're  interested in. The purposes of this are to, for example, produce  therapeutic proteins or compounds.</p> <p>In the context of disease, for example, where individuals have a  genetic disease in which they cannot make a particular enzyme, you might  be able to treat that disease by producing the relevant enzyme via this  technology in another organism and then supply it to that individual.</p> <p>The organism that you put it in could be a bacterium that produces it in large quantities.</p> <p>And you purify the enzyme and then perhaps inject it into the individual. It doesn't have to be bacteria.</p> <p>You can actually use recombinant DNA technology, genetic engineering  to produce a human protein in, for example, some farm animal. And there  are actually companies that will introduce human genes into sheep or  cows such that you can purify the recombinant protein from the milk of  that organism.</p> <p>And perhaps it might be more folded more properly or modified more  properly than it would be if in bacteria. And it might be more easily  purified from the milk than it would be from the bacterial cells.</p> <p>We can also engineer organisms.</p> <p>So this is to make different life forms that have different purposes,  different values to us. And I've given you the example in past lectures  about genetically modified foods.</p> <p>And your book talks about this as well.</p> <p>You can make crops that are resistant to pesticides. You can make  crops that are resistant to herbicides. You can make crops that stay  fresher longer, taste better, have better shape for packing, all sorts  of things by manipulating the genome of those organisms such that they  now have new and useful properties.</p> <p>We'll also teach you about how to make transgenic animals.</p> <p>I've given you one example of this already where you might want to  make a therapeutic protein in the milk of a transgenic cow, but there  are lots of purposes for making transgenic animals.</p> <p>One important one, especially for me, is to make disease models. So  once we understand a human genetic disease we can use that information  to create animal species, most typically mice that carry those very same  mutations.</p> <p>And then study the disease process in the context of a laboratory  mouse in ways that are difficult or impossible to do in humans.</p> <p>And we'll teach you how to do that. It all flows from the same  general principles of manipulating DNA and transferring it from one  organism to another.</p> <p>The value of doing this is several-fold.</p> <p>Understanding our or other DNA sequences.</p> <p>So without this technology, we actually couldn't have done the genome  project. We need to purify up a lot of our DNA in order to sequence it  to determine its sequence.</p> <p>And likewise of other organisms. So this was instrumental in bringing  us to the point we are today of understanding our genetic information  to the nucleotide level. As I mentioned, this technology allows us to  understand disease-causing mutations.</p> <p>Again, we've taught you about that mutations can occur, disabled  enzymes or other proteins cause disease, but that doesn't tell you why.</p> <p>And using this technology to sort of dissect what's happening to that  individual gene by manipulating it in different ways, putting it in  different contexts we can begin to understand what the specific  consequences of that mutation are, which ultimately allows us to deal  with the disease more effectively.</p> <p>And another value of just manipulating DNA, which we'll come to in a  future lecture, is what you hear about in the press as DNA  fingerprinting.</p> <p>It's a very practical use of this technology in which you want to  distinguish two individuals based on the specific changes in their DNA.  And we can do that because we can amplify up DNA sequences from very,  very small quantities and figure out the specific nucleotide sequences  of those DNA fragments and be able to tell then two people apart.</p> <p>And this is useful in paternity cases, it's useful in criminal cases, it's useful for archeological purposes and otherwise.</p> <p>So there are really a great number of values to this set of  technologies. So where do they come from? Well, we've been able to do  this sort of thing since the late 1960s, early 1970s. And this is  referred to, in our field, as the Recombinant DNA Revolution</p> <p>And there were three things that happened in this time period that made this technology possible.</p> <p>Firstly was the ability to separate genomic DNA, specifically into fragments.</p> <p>So genomic DNA, as you know, is huge. Our chromosomes are long, long  linear pieces of DNA. Even bacterial chromosomes are a few million  nucleotides in length.</p> <p>That's much too much to deal with for many of these applications. And  so it was necessary to basically chop the genomic DNA up into  manageable sized pieces and then be able to deal with those pieces  individually.</p> <p>The second thing that was necessary --</p> <p>-- was the ability to transfer the isolated pieces --</p> <p>-- into another organism.</p> <p>And the organism of choice then and largely now is bacteria. So it  was observations made in the 1960s that bacteria would actually take up  from their environment pieces of DNA and sometimes incorporate them  within their cells.</p> <p>That observation led to the notion that you could do that on purpose,  not just at random, and that would be then using the bacteria basically  as a cargo ship for the introduced DNA.</p> <p>And, finally, the ability to amplify the fragments of interest to large quantities.</p> <p>So if I take one piece of DNA from this guy, it's a single molecule, that's not enough. I cannot use that for much.</p> <p>I cannot use it to sequence his DNA, for example. I cannot use it to  study specific properties of that gene. Instead I need to amplify it up  to very large quantities.</p> <p>And, therefore, it was necessary to figure out ways basically to turn  the bacteria into little DNA factories that would churn out large  quantities of DNA of interest.</p> <p>And, again, we'll review how that was possible. And all of these  things came together more or less the same time of this timeframe, and  then the field took off. Now we could manipulate DNA very precisely and  begin to understand DNA at a very specific level.</p> <p>So I'm going to give you some examples of what we do.</p> <p>I'm actually going to give you perhaps the first and only practical  demonstration of this. We're going to do an experiment before you, a  dangerous thing to do in front of a live audience. We're actually going  to transfer a gene from one organism to another.</p> <p>We're going to start it today and we'll finish it in subsequent  lectures. The gene of interest is a toxic gene. It's a toxic gene that's  present in an organism known as S. pyogenes.</p> <p>You might know of this organism, might have heard of this organism because it's the flesh eating bacterium.</p> <p>Anybody heard of the flesh eating bacterium? This is true. If you get  infected with S. pyogenes and you're slightly immunocompromised, but  even if you have a bad strain of it even if you're very healthy, the  bacterium will go about eating your flesh.</p> <p>It actually can be lethal it's so devastating.</p> <p>The organism, S. pyogenes is a bacterium that has a genome, of  course. That genome has about a thousand genes. And the sequence of this  genome is known.</p> <p>So by standard DNA sequencing technologies, we'll actually review  those for you in a couple of lectures, specific DNA sequences of this  organism is known. OK? So we know exactly which genes there are there.</p> <p>And, just for the sake of discussion, let's consider a couple of  genes. Gene A, gene B and gene T. And it's this gene that we're  interested in. It's the gene that encodes a very critical toxin that the  bacterium makes that kills your cells.</p> <p>And so when you're badly infected by this bacterium, it's dumping out  this toxin and it's killing the cells in the surrounding. And that's  why it's flesh eating. OK?</p> <p>Now, you also heard in the previous couple of lectures ago that these genomes have other important properties.</p> <p>And just to remind you they have origins of replication, ORIs. All  genomes, for the purposes of replication, all pieces of DNA that are  going to be replicated need at least one origin of replication.</p> <p>So this genome has one. And this will come up in another context  shortly, but what we're really interested in is this gene T here, the  toxin gene. We want to clone it.</p> <p>We want to move it from this organism to another organism.</p> <p>And specifically the other organism of choice, and this is often the  case, is E. coli. E. coli is a common gut bacterium present in all of  you in very, very large quantities. It has been modified for laboratory  purposes largely weakened so that it cannot easily make its way back  into you and cause possible harm.</p> <p>It's used in laboratories in another context all the time.</p> <p>I'm depicting here an E. coli cell. This is always confusing because I  depicted here with a very similar circle the genome of S. pyogenes.</p> <p>This is a bacterial cell. This is the membrane of the bacterium. And,  of course, E. coli has its own genome which has roughly the same size  and roughly the same numbers of genes. What we're going to do is  transfer one more gene from this organism into that organism.</p> <p>Really, the purpose of today's lecture is how do you do that? But  before we consider how you do it, let's ask the question why you would  want to do it.</p> <p>Anybody have any ideas? Why would you want to transfer this potentially deadly toxin producing gene to another organism? Yeah?</p> <p>It could be that in a different context it's actually beneficial in one organism.</p> <p>You may be able to make a better E. coli for some purpose if you  transferred this gene. My favorite idea, actually, is global terror. </p> <p>We can laugh about this but, you know, there are terrorists around who are thinking these very thoughts.</p> <p>Sad but true, people are making modified organisms with dangerous  genes to make them more ìpathogenicî for deliberate release into  populations.</p> <p>So I'm not teaching you how to do this for malicious purposes but just to know that there are people who do that sort of thing.</p> <p>Related to this point over here, we want to understand these genes  for the purposes of biological research. We want to understand as much  as we can about disease-related properties of genes and so on.</p> <p>And sometimes you cannot do the experiments you want to do in the organism in which the disease occurs.</p> <p>And it's easier to do it, in a sense, in isolation in another organism. And then another example might be to produce a vaccine.</p> <p>If I could produce this protein that's so toxic in large quantities, I  might be able to make a version of it which is not toxic, very similar  but not toxic. And then I could inject it into people, they would raise  an antibody response against it such that if they were ever challenged  with the real dangerous guy they would be immune.</p> <p>So that would be a very useful thing to do.</p> <p>And, again, if we were worried about the terrorists we might do  exactly that. So this is the goal of our experiment. And we're going to  start the experiment today. So what I've done is acquired from one of my  colleagues in the Biology Department two tubes in which we have placed  these bacteria.</p> <p>S. pyogenes is in this one. E. coli is in this one. There are  bacterial cells in here that grow in suspension. They grow inside the  liquid medium.</p> <p>If we want to isolate the DNA we first have to isolate the bacteria.</p> <p>So we spin these tubes in a centrifuge. The bacteria then pellet to  the bottom of the tube. So the bacteria are now collected at the bottom  of these tubes. We then decant off the liquid. I meant to bring a  beaker, but I don't think they'll mind if I use this one.</p> <p>So it's always a little bit dangerous to do this, and we'll have to  take care of this later because it is a bit dangerous. It's always a  little bit dangerous to do this because you don't want to bump yourself  like I just did.</p> <p>It actually is starting to hurt a little bit.</p> <p>So we'll get rid of the liquid here. And then we'll add various  chemical solutions to isolate the DNA from this bacterium. Wow. It's  really starting to go. I'm just kidding.</p> <p>Gets them every time.</p> <p>There really was no bacterium in there of any kind. I know. OK. But  we're still going to go through this example. So we want to transfer  this gene into this organism.</p> <p>What are we going to do? Well, the first thing we need to worry about  is the fact that this gene, the T gene here, is contained on a very  large piece of DNA.</p> <p>As I said, it's got a thousand other genes. It's about 4 million base  pairs long. And I'm only interested in this one, so I need a way to  isolate the T gene away from the other genes present in this genome.</p> <p>So for this purpose I need a tool. And the tool that I need is a restriction enzyme</p> <p>Restriction enzymes are nicely described in your book. They are  enzymes present in bacterial cells which are designed to cut, to cleave  DNA sequences.</p> <p>And they do so in a site-specific fashion. They are sequence-specific --</p> <p>-- DNA cutting enzymes. We call them endonucleases, but you don't have to worry about that term. They are DNA cutting enzymes.</p> <p>Some of them produce, as you'll see in a moment, what we call sticky  ends, based on exactly how they cut the DNA. And others produce what we  call blunt ends because they cut the DNA in a slightly different way.</p> <p>There are hundreds of restriction enzymes that have been purified from various bacteria.</p> <p>There are now companies that will sell you the purified restriction  endonuclease. There are whole businesses that are geared around selling  this stuff to molecular biologists around the world. So you can order it  up like you would order up a chemical from a chemical supply house.</p> <p>As I said, these are site-specific, sequence-specific DNA cutting enzymes.</p> <p>So they recognize particular sequences in the DNA. One such enzyme,  which was one of the first discovered, an enzyme called EcoR1, which  comes from E.</p> <p>coli, that's why it's EcoR1, R1 is probably restriction enzyme number  one, recognizes a particular sequence reading from the 5 prime end of  the DNA molecule towards the 3 prime end. It recognizes the sequence  G-A-A-T-T-C.</p> <p>And then the other side of that, of course, is a 3 prime end.</p> <p>This is the polarity of DNA that you've been shown in many cases  before. Now what is the reverse strand? What does the opposite, the  complimentary strand of this sequence look like? Can anybody tell me?  You want to yell it out in unison.</p> <p>Here is a G and --</p> <p>That was very good. Notice anything about this? Yes. It reads the same ways forwards as backwards.</p> <p>G-A-A-T-T-C. G-A-A-T-T-C. What do we call that? It's a palindrome.  Your book uses the word MOM, which is a fairly boring palindrome.</p> <p>It is reads the same way forwards as backwards. I always like "A man,  a plan, a canal, Panama." That's a palindrome, too. It reads the same  way forwards as backwards. Anyway, many, not all, but many restriction  enzymes recognize palindromic sequences.</p> <p>They read the same way forwards as backwards.</p> <p>OK? And because of that they cleave the DNA on both strands at the  same position. EcoR1, this enzyme that recognizes this sequence will  cleave the DNA between this phosphodiester bond, between the  phosphodiester bond between the G and the A.</p> <p>And likewise on this strand. And the consequence of that is to  produce a break in the DNA, so you now have 5 prime G, 3 prime C-T-A-A.</p> <p>Breaks occur here and the strands get separated. They get pulled apart.</p> <p>The hydrogen bonds that are holding these base pairs together are not  strong enough to hold the two molecules together, at least most of the  time, so most of the time they pull apart. So you end up with a fragment  that has an end that looks like this and another fragment that has an  end that looks like this, A-A-T-C, here's the 3 prime end, G, and here's  the 5 prime end.</p> <p>OK? So you snip it at either side and pull it apart.</p> <p>Oops. I did something wrong. I missed a T, yup, on both of them.  Thank you. I don't think I've ever drawn those right. In all the years  I've taught this course, I always make it. Not the same mistake  necessarily but a mistake.</p> <p>OK, so they make a break. And that then cleaves the DNA into two pieces.</p> <p>Now, importantly, at low enough temperatures these ends can find each  other. And we call them sticky because they are complimentary to one  another.</p> <p>They can actually reform those base pairs. So you would get a  molecule which has 5 prime G, three prime C-T-T-A-A. And then base  paired to this T-T-A-A would be an A-A-T-T sequence, which was  covalently bound on this side to the C.</p> <p>No.</p> <p>Yeah, covalently bound on this side to the C and non-covalently bound  to this G. OK? So there are non-covalent bonds here and here and there  are base pairs in the middle that are holding these strands together.</p> <p>We call this annealing.</p> <p>And, again, at low enough temperatures you can get sticky ends of  restriction enzyme cleaved sites to come together this way. Now,  importantly another enzyme which we could use, discovered more or less  the same time, is an enzyme called DNA ligase, which you would have  heard about in the replication lectures.</p> <p>And DNA ligase will tie to non-covalently linked pieces of DNA together by a covalent bond.</p> <p>So DNA ligase will reintroduce this phosphodiester bond and this  phosphodiester bond to produce now a completely covalently-bound  sequence.</p> <p>OK? So we can cut the DNA at sites we want to. If we change the  temperature we can get these DNA molecules to come together in a  non-covalent fashion. And then, if we add this enzyme DNA ligase, they  will become covalently bond once again.</p> <p>OK?</p> <p>Key enzymes in our tool kit. As I said, there are other enzymes that  will cut DNA in a blunt fashion. They will not produce these sticky  ends. I won't show you a new example of that today, and it's not  terribly important that you know about them, but you can imagine that an  enzyme that cuts in the middle between the central positions of the  recognition sequence will make a blunt end.</p> <p>They won't be sticky.</p> <p>And those aren't as useful to us because the sticky ends actually  help us get two pieces of DNA together in order to promote the cloning  step at the end. OK. So the S. pyogenes genome that we need to isolate  our T gene from has a very large sequence.</p> <p>It's 4.1 times ten to the sixth base pairs long all the way around the circle. And I was wondering. --</p> <p>Well, let me just remind you that EcoR1 recognizes a 6 base pair  recognition site. So how many EcoR1 sites are there in this genome?  What's the frequency of EcoR1 sites along a piece of DNA?</p> <p>Well, it has to be this particular sequence, G-A-A-T-C-C, reading in that direction.</p> <p>The frequency of any given nucleotide at any given site is one in  four because there are four nucleotides. The frequency of six of a given  sequence in a row is one over four to the sixth, which is one in 4,096.</p> <p>So every 4,000 base pairs or so, at random, you will find an EcoR1 site.</p> <p>OK? They won't be equally distributed. There will be some parts of  the genome where there are many by chance, other parts where there are a  few, but on average there will be about one every 4,000 base pairs.</p> <p>And since the genome is 4 million base pairs long that means that  there's going to be about a thousand EcoR1 sites scattered around this  genome. OK?</p> <p>And they might be close together in some places, further apart in other places, and so on around the gene.</p> <p>Now, the T gene is in a particular place within the genome. I'm going  to draw it blue. Here's the T gene. And when deciding what restriction  endonuclease, what restriction enzyme to use to cleave the T gene in an  intact fashion, I'll consult the sequence, which I know, I'll choose an  enzyme which I know won't cleave this gene, won't cut this gene.</p> <p>There are no EcoR1 sites within this gene.</p> <p>OK? So there might be one on this side and there might be one on this  side, but I've chosen EcoR1 because I know there are no G-A-A-T-T-C  sequences within the T gene. OK? So to give you a visual depiction of  what we're talking about, more visual aids, I brought with me a very,  very dangerous piece of rope, a flesh eating piece of rope.</p> <p>They never fall for it twice, do they? In which I've depicted the S.</p> <p>pyogenes genome as a piece of covalently bound rope here. And the  EcoR1 sites are shown in yellow. OK? That's really what they are.  They're just little tags basically that the specific enzymes recognize.</p> <p>They're not there for the purposes of the enzyme recognizing them.  They're just there. And the enzyme can recognize them. OK? And here I've  colored in blue the T gene.</p> <p>And you'll see that there are no EcoR1 sites within the T gene.</p> <p>So now if I take this purified piece of DNA that I recovered from S.  pyogenes and I put in solution with the restriction enzyme EcoR1 in the  right sort of buffer and so on, the restriction enzyme will do what it's  able to do.</p> <p>It will go along and find those sites and cut them. OK? Everywhere it  sees one it will cut one. You know what I really should do? I should  figure out how to put the rope back together. That would be a trick.</p> <p>I don't know how to do that.</p> <p>I should work on that. That would be great. I should do that. All  right. So we've cleaved our DNA molecule into fragments. OK? And the S.  pyogenes genome that we drew up on the board will likewise be separated  into fragments.</p> <p>It will be separated into about a thousand fragments because there  are about a thousand sites. OK? I now want to separate those fragments  from one another.</p> <p>In order to clone the T gene, I need to separate the T gene away from  all the other genes and all the other DNA sequences in this genome.</p> <p>So how do I do it? Well, I take advantage of the fact that the pieces  of DNA that I've liberated are different lengths. You can see that  there are some short ones, there are some long ones, and it's actually  quite easy to separate DNA based on its size.</p> <p>The way you do that is to introduce the DNA solution that contains  the fragments of all sizes into what we call an agarose gel. It's like a  slab of Jell-O. At one end, as shown here, here's the slab of Jell-O in  which little wells have been cut out, little indentations have been  placed, we introduce the DNA solution, usually with a colored dye to let  us know what we're doing, and then we put buffer on top of the gel and  at the two wells at the end of the gel, and then we apply an electric  field.</p> <p>This is called gel electrophorisis.</p> <p>We apply a positive charge on one end, a negative charge on the other  end. Which way will the DNA go? Which way will the DNA go? Is DNA  positively charges or negatively charged? It's negatively charged  because all those phosphates which carry a negative charge.</p> <p>So in neutral pH buffer it will move towards the positive pole. And so the DNA will actually separate.</p> <p>And that's what you're seeing here. These are gels of different concentrations of the stuff that makes them stiff.</p> <p>So at a low percentage you see this fragmentation of fragments. The  larger fragments don't go very far, the smaller fragments go further  into the gel, and at different concentrations of this stuff you'll get  different separations in different positions.</p> <p>And so with this separation technique I can isolate this fragment, which is of a particular size, away from other fragments.</p> <p>So if you imagine this is the gel here, I put a negative charge here,  a positive charge here, the different DNA fragments are going to  separate along the gel with the long ones at the top and the short ones  at the bottom, and the middle sized ones will separate in the middle and  the T gene will separate right there.</p> <p>And, if I have a lot of DNA to start with, I'm going to have a lot of  molecules of the T gene in this position on the gel. And if I shine a  UV light, which was what was done here, I can literally see that  position.</p> <p>And since I know how big it is, because I know the sequence of this  organism, I can know that it's exactly 1,512 base pairs long.</p> <p>And, therefore, it should run right there in this particular position  on the gel. So I can purify it. Now, usually when I purify it, I purify  a bunch of other things that are of similar size. So I don't usually  get it all only this molecule, but I do a pretty good job of purifying  it.</p> <p>And now I need to amplify it. I've got some of that DNA, but I need to have more of the DNA.</p> <p>I need to have much, much, many, many more copies in order to be able to do the manipulations I want to do downstream.</p> <p>So the next thing I need to do is get this piece of DNA, and the  other ones that I'm less interested in, into bacteria and get the  bacteria to make more and more of it. So what I do in principle is to  introduce the DNA into different bacteria.</p> <p>There you go.</p> <p>And now I need those bacteria to make more of that stuff in order to  give me what I need. But there are certain problems. There are certain  problems associated with what I've just told to you. The problems are  that I cannot tell which of these folks got the right gene.</p> <p>That's one problem. I need to figure out a way to know who got the T gene and who didn't.</p> <p>I also need to know who got any piece of DNA versus who got nothing.  The rest of you didn't get any DNA. You are of no interest to me.</p> <p>If you were bacteria, I wish you would die. It's actually true.  You'll see in a second. The rest of these guys I'm somewhat interested  in, and I'm really interested in him because he's got the TG.</p> <p>I've picked on you a lot, haven't I?</p> <p>So those are two of the problems. And the third problem is that I  need these guys to divide, which they're not terribly willing to do. I  need them to divide, but bacteria will divide.</p> <p>But the bacteria, if these guys were bacteria, wouldn't know what to  do with that little fragment of DNA that I gave them. They wouldn't know  how to replicate it, how to make more if it because I've just given  them a naked piece of DNA and that's not terribly useful.</p> <p>So we need to overcome that.</p> <p>And I'm going to tell you how. The process that we just described,  the introduction of DNA into the bacterium is called transformation,  bacterial transformation. Again, here's a bacterial cell with its own  genome.</p> <p>We treat the bacterium with chemicals that sort of loosen up the  membrane a little bit or we electroshock the bacteria which blows holes  in the membrane.</p> <p>And then when there are holes in the membrane, these little fragments  of DNA which I'm representing here, maybe I'll just show it as a single  linear piece of DNA, can float in inside the cytoplasm of the bacteria.</p> <p>They literally just float in there. OK?</p> <p>Now, the bacterium, if they knew what to do with this piece of DNA,  are remarkably good biofactories because bacteria will divide at optimal  conditions three times every hour, three divisions every hour, which  means 72 divisions every day.</p> <p>And if they could duplicate this DNA every time they divided, if they  could and they were able to divide 72 times in a day, if there were one  copy of that piece of DNA per cell at the beginning of the process,  just one cell that had that piece of DNA in it then at the end of the  day there could be two to the seventy-second molecules of DNA.</p> <p>OK? 72 divisions. Two to the seventy-second which is roughly  equivalent to ten to the twentieth copies. Now, each of you has ten to  the thirteenth cells in you.</p> <p>So you've got ten to the thirteenth or so copies of any particular piece of DNA.</p> <p>This is ten to the seventh times more than that, so that's 10 million  persons' worth of a given piece of DNA. It's really quite remarkable.  Now, Claudette always makes me point out of the fact that you never  reach this theoretical limit because you would need an MIT's worth of  bacterial solution to do it.</p> <p>So in practice, when we do this sort of experiment, we get about ten to the tenth to ten to the thirteenth bacteria at the end.</p> <p>And, if each bacteria were able to copy this DNA faithfully, we would  have about ten to the tenth or ten to the thirteenth copies of the DNA.</p> <p>Still quite impressive for not a lot of work. The problem I've  already alluded to is that the bacteria don't know how to deal with this  introduced piece of DNA. This piece of DNA, for it to be brought up to  ten to the tenth or ten to the thirteenth copies needs to be replicated.</p> <p>The DNA needs to be replicated.</p> <p>What do we know about replication? What did you learn about  replication? What is the key thing that a piece of DNA needs in order  for it to be replicated? An origin of replication, right? And these  fragments most likely are not carry an origin of replication, so they're  not going to be replicated.</p> <p>So for this purpose we need an origin of replication.</p> <p>And you'll see how we accomplish that. We also mentioned the fact  that most of the bacteria, when we do this transformation, this is  actually a relatively inefficient process. So most bacteria get no DNA.</p> <p>And they are, of course, not interesting to us. That's all of you guys in the back there. So we need to get rid of all of you.</p> <p>And then we need to find which bacterium has the clone of interest.  Which bacterium has the piece of DNA of interest? This guy compared to  the rest of them.</p> <p>So the solutions to all of those problems come in the form of what we  call vectors. The piece of DNA that we introduce at the beginning is  not an isolated fragment of DNA, in fact.</p> <p>Instead, we introduce the piece of DNA in the context of another DNA molecule called a vector.</p> <p>Vectors, for the most part, are derived from naturally occurring  small DNA molecules that are carried in many bacteria outside of the  main bacterial genome.</p> <p>This is the main bacterial genome. Many bacteria carry within their  cytoplasm small copies of DNA, also circular, called plasmids.</p> <p>And these are short. They might be on average 5,000 base pairs long  compared to a couple of million base pairs long. So they're small  circular pieces of DNA.</p> <p>They get transferred from bacterium to bacterium in the wild for the  purposes of genetic exchange between bacteria, also for transmitting  drug resistance or pathogenic properties from one bacteria to another.</p> <p>So they do get exchanged from bacteria to bacteria. They were  discovered a long time ago and it was recognized that they would be very  useful as vehicle, vectors for cloned DNA.</p> <p>They have three key features, which I will illustrate in this diagram.</p> <p>One is the origin of replication. So they do replicate inside  bacterial cells. They have their own origin of replication. So you can  see where we're going here. We're going to put our piece of DNA, along  with this, in order to get the replication of our DNA as well.</p> <p>They have restriction sites.</p> <p>Of course, they're going to have restriction sites throughout their  DNA sequence. But they're designed now on purpose to have particular  restriction sites in a particular region of the piece of DNA which we  use for cloning purposes.</p> <p>So there might be EcoR1 site here and nowhere else in this plasmid, another enzyme represented once, and so on and so forth.</p> <p>Because we're going to introduce our DNA into here and we don't want to disrupt the sequence of the rest of the plasmid.</p> <p>And the final thing they have of value is a drug resistance gene,  sometimes multiple drug resistance genes. An example of a drug  resistance gene is the amp resistance gene, and this confers resistance  to a common antibiotic called ampicillin, very similar to penicillin.</p> <p>If you treat most bacteria with penicillin they will die, with ampicillin, sorry, they will die.</p> <p>If you treat a bacterium that's carrying a plasmid that has an  ampicillin resistance gene the bacterium will live, OK, because it  inactivates the ampicillin. It makes it non-toxic to the bacteria. And  now you can see how we're going to distinguish bacteria that get DNA  from bacteria that don't get any DNA.</p> <p>The bacteria that get DNA are going to get a drug resistance gene.</p> <p>So they're going to live in the presence of ampicillin. If you don't  get any DNA and you get treated with ampicillin you're going to die. OK.  So, quickly.</p> <p>I've taken my S. pyogenes genome and I cut it up into EcoR1 sites.</p> <p>I've cut it with EcoR1, so I've cut it into EcoR1 fragments. And now  I'm representing them as double-stranded with their sticky ends showing.  Here's the end of an EcoR1 site. Here's the end of an EcoR1 site.</p> <p>I'm just going to show you two for the sake of simplicity. Here's  another fragment liberated by EcoR1 from the S. pyogenes genome.</p> <p>Maybe these are two of similar size and I purified them together when I used my agarose gel technique.</p> <p>So these are two pieces of DNA. And this one contains our friend the T  gene. We've liberated these, they have their sticky ends, and then we  mix them with a plasmid that likewise has been cleaved with EcoR1.</p> <p>So we have lots and lots of cut plasmid that has the complimentary sticky ends.</p> <p>OK? This plasmid is that thing. I'm showing it as double-stranded now  so I can emphasize the sticky ends. When I lower the temperature, at  some frequency this DNA molecule with find its way here, an anneal at  this end over here, at this end over here, and this piece of DNA will  find its way to a different plasmid an anneal over here and over here.</p> <p>If I wait a while for that annealing process to take place and then I  add DNA ligase, the enzyme that seals the nicks that form when those  things come together, I will get fully closed double-stranded DNA  molecules which are the plasmid plus a new piece of DNA.</p> <p>This one will give some chunk of the S.</p> <p>pyogenes genome I don't care about. This one will give a plasmid that  has a piece of the S. pyogenes genome that carries the T gene. This one  I do care about. OK? So, in the final minute.</p> <p>I now take these plasmids that I've generated by this technique,  these recombinant plasmids, and I introduce them into bacteria in  culture using this transformation technique.</p> <p>And then I plate them onto a Petri dish. And the Petri dish,  importantly, has agar for the bacterial colonies to grow and it contains  ampicillin.</p> <p>It contains the drug ampicillin. Now, as I said, the transformation process is inefficient.</p> <p>Most bacteria that I treat in the transformation process will get no  DNA. They'll land on this plate. And what will happen to them? They'll  die. The plasmids that did get DNA, sorry, the bacterium that did pick  up a plasmid will have picked up an ampicillin resistance gene.</p> <p>They will grow.</p> <p>And they grow on little colonies, into little colonies after about a  day which have about a million cells each. And they're separated from  one another. So this colony might have that plasmid in it.</p> <p>Every cell in this colony has that plasmid in it. And this colony might have that plasmid in it. OK?</p> <p>I'm interested in this one because I want to now purify large quantities of that piece of DNA.</p> <p>I'm actually not interested in this one. And next lecture we'll talk  about how you specifically purify this, isolate and identify this colony  compared to the rest.</p>
</div>
        <div id="vid_related" itemprop="description" class="tabContent hide">
<h2 class="subhead">Free Downloads</h2>
<h3 class="subsubhead">Audio</h3>
<ul>
<li>iTunes U (<a href="http://deimos3.apple.com/WebObjects/Core.woa/Browse/mit.edu-dz.2820703204?i=1664446236">MP3 - 11MB</a>)</li>
<li>Internet Archive (<a href="http://www.archive.org/download/MIT7.013S05/Lecture08.mp3">MP3 - 11MB</a>)</li>
</ul>
<br><h3 class="subsubhead">Subtitle</h3>
<ul><li>English - US (<a href="../../../contents/audio-lectures/13-recombinant-dna-1/7.013-2006-L13.srt">SRT</a>)</li></ul>
<br>
</div>
    
   </div>  


      					 
        <div class="" id="parent-fieldname-bottom_html_area">
            
            
        </div>
    
                    </div>
<!--Course_inner_chip tag close -->
           		</div>
<!--Course_wrapper tag close --> 
            </div>
<!--left tag close -->
            <div id="right">
                <!--Begin Right Portion -->
                    <div>
    
<div id="portletwrapper-6f63772e7269676874746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a646f6e617465" class="portletWrapper kssattr-portlethash-6f63772e7269676874746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a646f6e617465">
<div class="portletStaticText portlet-static-donate"><p class="zero"><a href="http://ocw.mit.edu/donate"><img src="../../../common/images/button_donate-now.png" alt="Donate Now." class="donate"></a></p></div>

</div>




</div>

                	<div>
    



</div>


        <div class="" id="parent-fieldname-rsi_top_html_area">
            
            
        </div>
    

<!-- RSI google ad space-->


<div id="google_ads">    
    <script type="text/javascript" src="http://partner.googleadservices.com/gampad/google_service.js"></script><script type="text/javascript">GS_googleAddAdSenseService("ca-pub-6588555046597237");GS_googleEnableAllServices();</script><script type="text/javascript">GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_A_DL");GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_B_DL");GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_C_DL");</script><script type="text/javascript">GA_googleFetchAds();</script><script language="javascript" type="text/javascript">
GA_googleAddAttr("TYPE","HOUSE");
GA_googleAddAttr("DEPARTMENT","7");
GA_googleAddAttr("CRS_BEG2","01");
GA_googleAddAttr("CRS_END","3");
GA_googleAddAttr("SESSION","S");
GA_googleAddAttr("YEAR","06");
</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_A_DL");</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_B_DL");</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_C_DL");</script>
</div>

<!-- End RSI ads--> 

<div>
    



</div>

            </div>
<!--Right div close -->
            <div class="clear"></div> 
        </div>
<!--grid tag close --> 
      </div>
		
		<div id="bottom">
			<div id="grid">
				
<div id="portletwrapper-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572" class="portletWrapper kssattr-portlethash-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572">
<div class="portletStaticText portlet-static-site-footer"><div id="bottom">
<div id="grid">
<!-- *begin footer* -->
<div id="footer" role="navigation sitemap">
<div id="foot-c1" class="grid_2 alpha">
<h4 class="footer">Courses</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/courses/find-by-topic/">Find by Topic</a></li>
    <li><a href="http://ocw.mit.edu/courses/find-by-number/">Find by Course Number</a></li>
    <li><a href="http://ocw.mit.edu/courses/find-by-department/">Find by Department</a></li>
    <li><a href="http://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/translated-courses/">Translated Courses</a></li>
    <li><a href="http://ocw.mit.edu/resources/">Supplemental Resources</a></li>
</ul>
</div>
<div id="foot-c2" class="grid_2">
<h4 class="footer">About</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/about/">About OpenCourseWare</a></li>
    <li><a href="http://ocw.mit.edu/about/site-statistics/">Site Stats</a></li>
    <li><a href="http://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
    <li><a href="http://ocw.mit.edu/about/ocw-consortium/">OCW Consortium</a></li>
    <li><a href="http://ocw.mit.edu/about/media-coverage/">Media Coverage</a></li>
    <li><a href="http://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
    <li><a href="http://ocw.mit.edu/about/media-coverage/press-releases/">Press Releases</a></li>
    <li><a href="http://ocw.mit.edu/about/next-decade/">OCW's Next Decade</a></li>
</ul>
</div>
<div id="foot-c3" class="grid_2">
<h4 class="footer">Donate</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/donate">Make a Donation</a></li>
    <li><a href="http://ocw.mit.edu/donate/why-donate/">Why Donate?</a></li>
    <li><a href="http://ocw.mit.edu/donate/ocw-course-champions-program/">Become a Course Champion</a></li>
    <li><a href="http://ocw.mit.edu/donate/our-supporters/">Our Supporters</a></li>
    <li><a href="http://ocw.mit.edu/donate/other-ways-to-contribute/">Other Ways to Contribute</a></li>
    <li><a href="http://ocw.mit.edu/donate/shop-ocw/">Shop OCW</a></li>
    <li><a href="http://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
</ul>
</div>
<div id="foot-c4" class="grid_2">
<h4 class="footer">Featured Sites</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/high-school/">Highlights for High School</a></li>
    <li><a href="https://www.edx.org/school/mitx/allcourses">MITx</a></li>
    <li><a href="http://teachingexcellence.mit.edu/">Teaching Excellence at MIT</a></li>
</ul>
<h4 class="footer" style="margin-top: 14px;">Tools</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/help/">Help &amp; FAQs</a></li>
    <li><a href="../../../common/jsp/feedback.htm">Contact Us</a></li>
    <li><a href="../../../common/search/AdvancedSearch.htm">Advanced Search</a></li>
    <li><a href="http://ocw.mit.edu/help/site-map/">Site Map</a></li>
    <li><a href="../../../common/terms/index.htm">Privacy &amp; Terms of Use</a></li>
</ul>
</div>
<div id="foot-c5" class="grid_4 omega" itemprop="publisher" itemscope="" itemtype="http://schema.org/CollegeOrUniversity">
<h4 class="footer">About <span itemprop="name">MIT OpenCourseWare</span>
</h4>
<p class="accent" itemprop="description">MIT OpenCourseWare makes the materials used in the teaching of almost all of MIT's subjects available on the Web, free of charge. With more than 2,000 courses available, OCW is delivering on the promise of open sharing of knowledge.</p>
</div>
<div id="foot-copy" class="grid_8 alpha">
<a href="http://web.mit.edu"><img src="../../../common/images/logo_mit.png" alt="Massachusetts Institute of Technology logo and name." style="width: 195px; height: 44px;"></a>                 <a href="http://www.ocwconsortium.org/"><img src="../../../common/images/logo_ocwc.gif" alt="OpenCourseWare Consortium logo." style="width: 80px; height: 44px;"></a>                 <a rel="license" itemprop="useRightsUrl" href="http://creativecommons.org/licenses/by-nc-sa/3.0/us/deed.en_US"><img src="../../../common/images/cc_by-nc-sa.png" alt="Creative Commons logo with terms BY-NC-SA." style="width: 126px; height: 44px;"></a>
<p class="copyright">© 2001–2013<br>
Massachusetts Institute of Technology</p>
<p style="font-size: 0.9em; margin-bottom: 15px;">Your use of the MIT OpenCourseWare site and materials is subject to our <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/us/deed.en_US" rel="license">Creative Commons License</a> and other <a href="../../../common/terms/index.htm" rel="cc:morePermissions">terms of use</a>.</p>
</div>
<div id="foot-support" class="grid_4 omega"> </div>
</div>
</div>
</div></div>

</div>





                
			</div> <!-- bottom grid end -->
		</div>
<!-- bottom end -->
		
		
   </body>
</html>
