<!DOCTYPE html><html lang="en">
<head>
<meta charset="utf-8">
<meta name="format-detection" content="telephone=no">
<title>15: Recombinant DNA 3 | Audio Lectures | Introductory Biology | Biology | MIT OpenCourseWare</title>
<!-- Begin Automatic Metadata Insertion --><meta content="7-013-introductory-biology-spring-2006" name="WT.cg_n">
<meta content="15: Recombinant DNA 3" name="WT.cg_s">
<meta content="" name="Description">
<meta content="Sive, Hazel" name="Author">
<meta content="Jacks, Tyler" name="Author">
<meta content="biology,biochemistry,genetics,molecular biology,recombinant DNA,cell cycle,cell signaling,cloning,stem cells,cancer,immunology,virology,genomics,molecular medicine,DNA,RNA,proteins,replication,transcription,mRNA,translation,ribosome,nervous system,amino acids,polypeptide chain,cell biology,neurobiology,gene regulation,protein structure,protein synthesis,gene structure,PCR,polymerase chain reaction,protein localization,endoplasmic reticulum,human biology,inherited diseases,developmental biology,evolution,human genetics,human diseases,infectious agents,infectious diseases,Biology" name="keywords">
<meta content="15: Recombinant DNA 3 | MIT OpenCourseWare" name="Search_Display">
<meta content="Biology" itemprop="about">
<!-- End Automatic Metadata Insertion --><link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/grid.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/base.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/menu.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/jquery.bubblepopup.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/courses.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/courses_new.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/jquery.jscrollpane.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/media_tabs.css">
<link href="http://ocw.mit.edu/xml/ocwcc.rdf" type="application/rdf+xml" rel="metadata">
<link rel="canonical" href="http://ocw.mit.edu/courses/biology/7-013-introductory-biology-spring-2006/audio-lectures/15-recombinant-dna-3">
<link rel="apple-touch-icon" href="../../../common/images/apple-touch-icon.png">
<script type="text/javascript" src="../../../common/scripts/common-utils-offline.js"></script><script type="text/javascript" src="../../../common/scripts/ocw-offline.js"></script><script type="text/javascript" src="../../../common/scripts/jquery.bubblepopup.min.js"></script><script type="text/javascript" src="../../../common/scripts/jquery-ui.min.js"></script><script type="text/javascript" src="../../../common/scripts/jquery.jscrollpane.min.js"></script><script type="text/javascript" src="../../../common/scripts/bubble-popup-offline.js"></script><script type="text/javascript">
      $(document).ready(function() {
        $("#tabs").tabs();
        IpadScroller();
      });
    </script>
</head>
<body itemscope itemtype="http://schema.org/WebPage">
        
	

        <div id="top">
			<div id="grid">
				
				
					
<div id="portletwrapper-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572" class="portletWrapper kssattr-portlethash-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572">
<div class="portletStaticText portlet-static-site-header">
<div id="banner" role="banner" class="grid_6 alpha"><a href="http://ocw.mit.edu/"><img src="../../../common/images/ocw_mast.png" alt="MIT OpenCourseWare, Massachusetts Institute of Technology" class="logo"></a></div>
<form action="http://ocw.mit.edu/subscribe/index.htm?utm_source=header" method="get">
    <div id="subscribe" role="form toolbar" class="grid_6 omega">
    <div class="module">
    <table class="social"><tbody><tr>
<td class="text">Subscribe to the<br>
                OCW Newsletter</td>
                <td class="black"><input type="text" onclick="clearEmailBox()" onblur="fillEmailBox()" value="Enter Email" class="greytext emailField" id="email" name="fromemail"></td>
                <td class="black"><input type="image" src="../../../common/images/button_subscribe.png" alt="Subscribe" class="sub_button"></td>
                <td>
<a href="http://facebook.com/mitocw"><img src="../../../common/images/icon_fb.png" alt="Click to visit our Facebook page."></a><a href="http://twitter.com/mitocw"><img src="../../../common/images/icon_tw.png" alt="Click to visit our Twitter feed."></a>
</td>
            </tr></tbody></table>
</div>
    <p class="helplinks"><a href="http://ocw.mit.edu/help">Help</a>   |   <a href="../../../common/jsp/feedback.htm">Contact Us</a></p>
    </div>
    <div class="clear"> </div>
</form>
</div>

</div>





<div id="mega" role="navigation" class="grid_8 alpha">        
	<ul id="menu">
<li id="menu_home">
            <a href="http://ocw.mit.edu/"><img src="../../../common/images/top-nav_home.png" class="home_icon" alt="Click for site home page."></a><!-- Begin Home Item -->
        </li>
<!-- End Home Item -->        
        <li class="selected">
            <a href="#" class="drop">Courses</a><!-- Begin 5 columns Item -->
            <div class="dropdown_5columns-a mega-courses">                    
                <div class="col_1a">
                    <div class="row_1a">
                        <div class="quart">
                            <h2 class="nav">Find courses by:</h2>
                            <ul class="nav-bullet find_by">
<li><a href="http://ocw.mit.edu/courses/find-by-topic/">Topic</a></li>
                                <li><a href="http://ocw.mit.edu/courses/find-by-number/">MIT Course Number</a></li>
                                <li><a href="http://ocw.mit.edu/courses/find-by-department/">Department</a></li>
                            </ul>
</div>
                        <div class="quart">
                            <h2 class="nav">Collections</h2>
                            <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Lectures</a></li>
                                <li><a href="http://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
                                <li><a href="http://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
                                <li><a href="http://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
                                <li><a href="http://ocw.mit.edu/resources">Supplemental Resources</a></li>
                            </ul>
</div>
                        <div class="clear"> </div>
                    </div>
                    <div class="row_1b">
                        <h2 class="nav">Cross-Disciplinary Topic Lists</h2>
                        <div class="quart">
                        <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/energy-courses">Energy</a></li>
                            <li><a href="http://ocw.mit.edu/courses/entrepreneurship">Entrepreneurship</a></li>
                            <li><a href="http://ocw.mit.edu/courses/environment-courses">Environment</a></li>
                        </ul>
</div>    
                        <div class="quart">
                        <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/intro-programming">Introductory Programming</a></li>
                            <li><a href="http://ocw.mit.edu/courses/life-sciences">Life Sciences</a></li>
                            <li><a href="http://ocw.mit.edu/courses/transportation-courses">Transportation</a></li>
                        </ul>
</div>
                        <div class="clear"> </div>
                    </div>
                    <div class="clear"> </div>
                </div>
                <div class="col_1b">
                    <h2 class="nav">Translated Courses</h2>
                    <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/translated-courses/traditional-chinese">繁體字 / Traditional Chinese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/simplified-chinese">简体字 / Simplified Chinese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/spanish">Español / Spanish</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/portuguese">Português / Portuguese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/thai">ภาษาเขียน / Thai</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/persian">فارسی / Persian</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/turkish">Türkçe / Turkish</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/korean">(비디오)한국 / Korean</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses">More...</a></li>
                    </ul>
</div>
            </div>
        </li>
        <li>
            <a href="" class="drop">About</a>
            <div class="dropdown_1column-a">
                <div class="col_1">
                    <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/">About MIT OpenCourseWare</a></li>
                    </ul>
<ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/site-statistics/">Site Stats</a></li>
                        <li><a href="http://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
                        <li><a href="http://ocw.mit.edu/about/ocw-consortium/">OpenCourseWare Consortium</a></li>
                    </ul>
<ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/media-coverage/">Media Coverage</a></li>
                        <li><a href="http://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
                        <li><a href="http://ocw.mit.edu/about/media-coverage/press-releases/">Press Releases</a></li>
                    </ul>
<ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/about/next-decade/">OCW's Next Decade</a></li>
                    </ul>
</div>
            </div>  
        </li>    
        <li>
            <a href="" class="drop">Donate</a>        
            <div class="dropdown_1column-a">
                    <div class="col_1">
                        <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/donate/">Make a Donation</a></li>
                            <li><a href="http://ocw.mit.edu/donate/why-donate/">Why Donate?</a></li>
                            <li><a href="http://ocw.mit.edu/donate/ocw-course-champions-program/">Become a Course Champion</a></li>
                            <li><a href="http://ocw.mit.edu/donate/our-supporters/">Our Supporters</a></li>
                            <li><a href="http://ocw.mit.edu/donate/other-ways-to-contribute/">Other Ways to Contribute</a></li>
                            <li><a href="http://ocw.mit.edu/donate/shop-ocw">Shop OCW</a></li>
                        </ul>
<ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
                        </ul>
</div>
            </div>            
        </li>        
        <li>
            <a href="" class="drop">Featured Sites</a>        
            <div class="dropdown_1column-a">
                <div class="col_1">
                    <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/high-school/">Highlights for High School</a></li>
                    </ul>
<ul class="nav-bullet mega-div-top">
<li><a href="https://www.edx.org/school/mitx/allcourses">MITx</a></li>
                        <li><a href="http://teachingexcellence.mit.edu/">Teaching Excellence at MIT</a></li>
                    </ul>
</div>
            </div>            
        </li>
    </ul>
</div>
<div id="search" role="search" class="grid_4 omega">
    
    <form method="get" action="../../../common/search/AdvancedSearch.htm">
     	 <table class="search"><tbody><tr>
<td class="black"><input type="text" onblur="fillSearchBox()" onfocus="clearSearchBox()" maxlength="255" value="Search" name="q" class="greytext searchField" id="terms"></td> 			 
                    <td class="black"><input type="image" src="../../../common/images/button_search.png" name="btnG" class="sub_button"></td>			 
                    <td class="text2"><a href="../../../common/search/AdvancedSearch.htm">Advanced<br>Search</a></td>
                </tr></tbody></table>
</form>
</div>
<div class="clear"></div>
<!-- *end header* -->  

				
				
			</div>
<!-- top grid end -->
		</div>
<!-- top end -->
			
		<div id="center_media">
      	<div id="grid">
      		<div id="left">
        		<div id="breadcrumb_media">
                	<p>

    <a href="http://ocw.mit.edu/">Home</a>
    
        »
        
    
    
        
            <a href="http://ocw.mit.edu/courses">Courses</a>
            
                »
                
            
            
         
    
    
        
            <a href="http://ocw.mit.edu/courses/biology">Biology</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../../contents/index.htm">Introductory Biology</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../../contents/audio-lectures/index.htm">Audio Lectures</a>
            
                »
                
            
            
         
    
    
        
            
            
            15: Recombinant DNA 3
         
    
</p>

            	</div>
            	<div class="clear"></div>
        		<div id="media_title">
        		<h1 class="title" itemprop="name" property="dct:title">
        <span class="" id="parent-fieldname-title">
            15: Recombinant DNA 3
        </span>
    </h1>
        		</div>
           		<div class="clear"></div>
           		<div id="course_wrapper_media">
           			<div id="course_nav">
           				<script language="javascript" type="text/javascript">
function toggleMenu(objID) {
  if (!document.getElementById) return;
  var ob = document.getElementById(objID);
  ob.className = (ob.className == 'selected')?'': 'selected';
}
function toggleClass(id)
{
  var divtoggleClass= document.getElementById(id);
  divtoggleClass.className = (divtoggleClass.className == 'mO')?'mC': 'mO';
  return false;
}
function changeAlt(id)
{
  id.alt = (id.alt == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
  id.title = (id.title == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
}
</script><!--Left Nav Starts --><ul>
<li class="">
			   			<a href="../../../contents/index.htm">
		                  Course Home  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/syllabus/index.htm">
		                  Syllabus  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/calendar/index.htm">
		                  Calendar  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/readings/index.htm">
		                  Readings  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/recitations/index.htm">
		                  Recitations  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/assignments/index.htm">
		                  Assignments  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/exams/index.htm">
		                  Exams  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/study-materials/index.htm">
		                  Study Materials  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="selected">
			   			<a href="../../../contents/audio-lectures/index.htm">
		                  Audio Lectures  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/video-lectures/index.htm">
		                  Video Lectures  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    
		    
         	
	<!--second tal block close-->  
	
</ul>
<!--Left Nav Ends --><script type="text/javascript" id="openstudy-widget-loader" src="http://openstudy.com/javascripts/load.js?type=feed&amp;color=auto&amp;studygroup=MIT%20OCW%20Biology&amp;env=openstudy.com"></script>
</div>
           			<div id="course_inner_media">
      					 
        <div class="" id="parent-fieldname-text">
            
            
        </div>
    
      					 

     <div id="media-embed">
         <div class="attention_message" id="embed_1">
<p>Flash and JavaScript are required for this feature.</p>
<p>Download the track from <a href="http://deimos3.apple.com/WebObjects/Core.woa/Browse/mit.edu-dz.2820703216?i=2085243275">iTunes U</a> or the <a href="http://www.archive.org/download/MIT7.013S05/Lecture10.mp3">Internet Archive</a>.</p>
</div>
     </div>
    
     <script type="text/javascript">ocw_embed_chapter_media('embed_1', 'http://www.archive.org/download/MIT7.013S05/Lecture10.mp3', 'sound', '/courses/biology/7-013-introductory-biology-spring-2006/audio-lectures/15-recombinant-dna-3', 'http://ocw.mit.edu/images/audio_only_slate.gif',0,0, 'http://ocw.mit.edu/courses/biology/7-013-introductory-biology-spring-2006/audio-lectures/15-recombinant-dna-3/7.013-2006-L15.srt')</script><div id="media_resource_next_prev_nav" style="margin-top: 1em;">
        <p>
        
            <a href="../../../contents/audio-lectures/14-recombinant-dna-2/index.htm">
                <img src="../../../common/images/btn_previous_resource.png" style="margin: 0 30px 0 50px;" alt="Previous track" title="Previous track"></a>
     	
     	
        
            <a href="../../../contents/audio-lectures/16-cell-biology-1/index.htm"> 
                <img src="../../../common/images/btn_next_resource.png" alt="Next track" title="Next track"></a>
       
       </p>
     </div>
 


<script type="text/javascript">
		window.onload=function(){
		init();
		
		}
		var tabLinks = new Array();
		var contentDivs = new Array();
		function init() {
		  // Grab the tab links and content divs from the page
		  var tabListItems = document.getElementById('tabs').childNodes;
		  for ( var i = 0; i < tabListItems.length; i++ ) {
			if ( tabListItems[i].nodeName == "LI" ) {
			  var tabLink = getFirstChildWithTagName( tabListItems[i], 'A' );
			  var id = getHash( tabLink.getAttribute('href') );
			  tabLinks[id] = tabLink;
			  contentDivs[id] = document.getElementById( id );
			}
		  }
		  // Assign onclick events to the tab links, and
		  // highlight the first tab
		  var i = 0;
		  for ( var id in tabLinks ) {
			tabLinks[id].onclick = showTab;
			tabLinks[id].onfocus = function() { this.blur() };
			if ( i == 0 ) tabLinks[id].className = 'selected';
			i++;
		  }
		  // Hide all content divs except the first
		  var i = 0;
		  for ( var id in contentDivs ) {
			if ( i != 0 ) contentDivs[id].className = 'tabContent hide';
			i++;
		  }
		}
		function showTab() {
		  var selectedId = getHash( this.getAttribute('href') );
		  // Highlight the selected tab, and dim all others.
		  // Also show the selected content div, and hide all others.
		  for ( var id in contentDivs ) {
			if ( id == selectedId ) {
			  tabLinks[id].className = 'selected';
			  contentDivs[id].className = 'tabContent';
			} else {
			  tabLinks[id].className = '';
			  contentDivs[id].className = 'tabContent hide';
			}
		  }
		  // Stop the browser following the link
		  return false;
		}
		function getFirstChildWithTagName( element, tagName ) {
		  for ( var i = 0; i < element.childNodes.length; i++ ) {
			if ( element.childNodes[i].nodeName == tagName ) return element.childNodes[i];
		  }
		}
		function getHash( url ) {
		  var hashPos = url.lastIndexOf ( '#' );
		  return url.substring( hashPos + 1 );
		}
 </script><div id="media_tabs">
     
        <ul id="tabs">
<li class="first">
                <a href="#vid_about" class="selected">About this Track</a>
            </li>
            <li class="">
                <a href="#vid_index" class="">Playlist</a>
            </li>
            <li class="">
                <a href="#vid_playlist" class="">Transcript</a>
            </li>
            <li class="">
                <a href="#vid_related" class="">Download this Track</a>
            </li>
        </ul>
<div id="vid_about" itemprop="description" class="tabContent"><p><strong>Lecturer:</strong> Prof. Hazel Sive</p></div>
        <div id="vid_index" itemprop="description" class="tabContent hide">
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/3-biochemistry-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="3: Biochemistry 1" alt="3: Biochemistry 1"><p>3: Biochemistry 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/5-biochemistry-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="5: Biochemistry 3" alt="5: Biochemistry 3"><p>5: Biochemistry 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/7-genetics-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="7: Genetics 2" alt="7: Genetics 2"><p>7: Genetics 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/8-genetics-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="8: Genetics 3" alt="8: Genetics 3"><p>8: Genetics 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/9-genetics-4/index.htm">
<img src="../../../common/images/audio_file.gif" title="9: Genetics 4" alt="9: Genetics 4"><p>9: Genetics 4</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/11-molecular-biology-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="11: Molecular Biology 2" alt="11: Molecular Biology 2"><p>11: Molecular Biology 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/12-molecular-biology-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="12: Molecular Biology 3" alt="12: Molecular Biology 3"><p>12: Molecular Biology 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/13-recombinant-dna-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="13: Recombinant DNA 1" alt="13: Recombinant DNA 1"><p>13: Recombinant DNA 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/14-recombinant-dna-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="14: Recombinant DNA 2" alt="14: Recombinant DNA 2"><p>14: Recombinant DNA 2</p></a>
</div>
<div class="related-media-thumbnail-nolink">
<div class="now-playing-resource">Now Playing</div>
<img src="../../../common/images/audio_file.gif" title="15: Recombinant DNA 3" alt="15: Recombinant DNA 3"><p>15: Recombinant DNA 3</p>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/16-cell-biology-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="16: Cell Biology 1" alt="16: Cell Biology 1"><p>16: Cell Biology 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/17-cell-biology-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="17: Cell Biology 2" alt="17: Cell Biology 2"><p>17: Cell Biology 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/19-steps-in-development/index.htm">
<img src="../../../common/images/audio_file.gif" title="19: Steps in Development" alt="19: Steps in Development"><p>19: Steps in Development</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/20-fertilization-fertility/index.htm">
<img src="../../../common/images/audio_file.gif" title="20: Fertilization/Fertility" alt="20: Fertilization/Fertility"><p>20: Fertilization/Fertility</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/21-cell-type-and-position/index.htm">
<img src="../../../common/images/audio_file.gif" title="21: Cell Type and Position" alt="21: Cell Type and Position"><p>21: Cell Type and Position</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/22-morphogenesis/index.htm">
<img src="../../../common/images/audio_file.gif" title="22: Morphogenesis" alt="22: Morphogenesis"><p>22: Morphogenesis</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/23-stem-cells/index.htm">
<img src="../../../common/images/audio_file.gif" title="23: Stem Cells" alt="23: Stem Cells"><p>23: Stem Cells</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/24-cloning/index.htm">
<img src="../../../common/images/audio_file.gif" title="24: Cloning" alt="24: Cloning"><p>24: Cloning</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/25-nervous-system-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="25: Nervous System 1" alt="25: Nervous System 1"><p>25: Nervous System 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/26-nervous-system-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="26: Nervous System 2" alt="26: Nervous System 2"><p>26: Nervous System 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/27-nervous-system-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="27: Nervous System 3" alt="27: Nervous System 3"><p>27: Nervous System 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/28-cancer-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="28: Cancer 1" alt="28: Cancer 1"><p>28: Cancer 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/29-cancer-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="29: Cancer 2" alt="29: Cancer 2"><p>29: Cancer 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/30-cancer-3/index.htm">
<img src="../../../common/images/audio_file.gif" title="30: Cancer 3" alt="30: Cancer 3"><p>30: Cancer 3</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/31-immunology-1/index.htm">
<img src="../../../common/images/audio_file.gif" title="31: Immunology 1" alt="31: Immunology 1"><p>31: Immunology 1</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/32-immunology-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="32: Immunology 2" alt="32: Immunology 2"><p>32: Immunology 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/34-virology-2/index.htm">
<img src="../../../common/images/audio_file.gif" title="34: Virology 2" alt="34: Virology 2"><p>34: Virology 2</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/35-prion-disease/index.htm">
<img src="../../../common/images/audio_file.gif" title="35: Prion Disease" alt="35: Prion Disease"><p>35: Prion Disease</p></a>
</div>
<div class="related-media-thumbnail">
<a href="../../../contents/audio-lectures/36-future-of-biology/index.htm">
<img src="../../../common/images/audio_file.gif" title="36: Future of Biology" alt="36: Future of Biology"><p>36: Future of Biology</p></a>
</div>
</div>
        <div id="vid_playlist" itemprop="description" class="tabContent hide">
<ul><li><a class="transcript-link" title="Open in a new window." alt="Open in a new window." style="text-decoration: none; font-size: 1.0em;" target="_blank" text-decoration: none font-size: href="../../../contents/audio-lectures/7_0132006L15.pdf"> Download this transcript - PDF (English - US)</a></li></ul>
<p></p>
<p> </p>
<p>Professor Jacks is out of town so I am going to tell you about  Recombinant DNA 3, then he's going to come back and tell you about Cell  Biology, and then you will have finished the foundations part of the  course.</p> <p>And we'll move onto things that build on the foundation, the  Formation Module and the part of the Systems Module, which I'll be  teaching you for the next few weeks, but today is Recombinant DNA 3.</p> <p>And, as you've been hearing for the last couple of lectures, this is  one of the How-To Modules that we've put in the course. How to make use  of the information that you have been learning in Molecular Biology and  in Biochemistry and in Genetics to use these disciplines or these pieces  of information to do something useful.</p> <p>And recombinant DNA is really an extraordinary set of technologies that just keeps getting more and more extraordinary.</p> <p>And the way one can manipulate biological systems now is really very  exciting. And it continues to be exciting. When I was a beginning  graduate student we were able to clone the first pieces of DNA.</p> <p>And now we can really do a lot more than just clone DNA.</p> <p>So I want to tell you about some of the things that are really  essential to understand about this technology, and then take you through  some of the forefronts of where recombinant DNA technology is now.</p> <p>We're going to cover three things in this lecture.</p> <p>DNA sequencing, using genetic polymorphisms for various genotyping  analyses, and then I'm going to try to touch on, and we'll have to see  how we do here, making animals that are so-called transgenic.</p> <p>So transgenic technology.</p> <p>And I'm going to use PowerPoint pretty much for most of the lecture,  so you have most of the relevant stuff in front of you. I'm going to  frame this in terms of a human disease, familial hypercholesterolemia.</p> <p>So you may remember way back when in biochemistry we talked about cholesterol.</p> <p>Anyone remember what class of macromolecules cholesterol belongs to?  Lipids. Thank you. Lipids. OK. I'm not even going to give a frog for  that. And we have this sense of cholesterol being a really bad kind of  molecule but, in fact, cholesterol is an essential lipid.</p> <p>It's extremely important. Without cholesterol you'd die and you need it for many things.</p> <p>Not only for building membranes in your cells but also, if you think  way back, you may remember me telling you that cholesterol was part of  or had a chemical structure that was very similar to the steroid hormone  family.</p> <p>And steroid hormones, and we'll discuss this more in the future, are  very important molecules that tell one part of the body what to do, that  regulate what different parts of the body are doing. So cholesterol is  part of this whole signaling system.</p> <p>And really it's not actually understood all of what cholesterol does, but it's very important.</p> <p>However, too much of it is not good. And it's probably not good  because, but it's not actually clear. I'll tell you what happens if you  have too much cholesterol, but actually why it happens is not that  clear.</p> <p>So let me talk about this slide up here, and then we'll talk about what too much cholesterol does for you.</p> <p>So familial hypercholesterolemia is an inherited disease, and it's  caused by mutations in a gene called the LDL receptor, that encodes for  something called the LDL receptor.</p> <p>Now, LDL stands for low density lipoprotein. And you had this in a  previous lecture because I'd been mentioned these to you. Low density  lipoproteins. And these bind to various lipids, including cholesterol,  and are taken up into the cell.</p> <p>And some of them are OK, you probably need some LDLs, but too much LDL is bad.</p> <p>And if you have too much LDL receptor, the thing that actually binds  to the LDLs, you get too much LDL taken up into the cell. So this LDL  receptor, you'll talk more about this in cell biology, this LDL  receptor, and you've already had some of this, the LDL receptor is a  protein that binds to these LDLs, takes them into the cell, and then  your cell gets full of LDLs.</p> <p>OK? And as a consequence of this, your cholesterol levels go way up.</p> <p>Now, you can be heterozygote or homozygote for familiar  hypercholesterolemia, for the LDL receptor gene. OK? For the familiar  hypercholesterolemia gene. Try to say that one quickly. All right.</p> <p>So if you're heterozygote, you have an increased risk of heart disease.</p> <p>In particular for this thing called atherosclerosis I'll talk more  about in a moment. If you are homozygote, so you have two copies of a  mutated LDL receptor gene, you get severe heart symptoms and you die  early.</p> <p>OK?</p> <p>What is atherosclerosis? Atherosclerosis is a disease that occurs  because you get these buildups of stuff in the blood vessels. And the  stuff is fat and it's proteins, and it basically makes a big lump that  eventually occludes or blocks the blood vessel.</p> <p>And so atherosclerosis is bad because impedes blood flow.</p> <p>And if you impede blood flow, eventually your heart will seize up and  you will have a heart attack, and that can have, obviously, very severe  consequences.</p> <p>So atherosclerosis occurs because you have high levels of LDL. And  it's really, the actual etiology of atherosclerosis is not really clear.</p> <p>Part it may be that there's just too much fat around and that starts  actually getting deposited out of solution, but it's much more  complicated than that.</p> <p>And there seems to be a very complicated chain of events by which you  get these atherosclerosis plaques sitting on the lining of blood  vessels and impeding blood flow. OK. So there is a lot of interest  medically in atherosclerosis, particularly in countries such as ours  where food is plentiful and people tend to have too much.</p> <p>And obesity is a problem anyway because that is part of the set of  risk factors for atherosclerosis. So here are the risk factors. High  levels of LDL, high blood pressure, diabetes, cigarette smoke and so on.</p> <p>And familial hypercholesterolemia is contributory to high levels of LDL and atherosclerosis. OK.</p> <p>So one of the things I want to do is to keep thinking about this disorder and walk you through how you figure out who's got FH.</p> <p>OK. What you can do is to get blood cells from people at-risk, and  you can actually examine the LDL receptor gene in the blood cells of  people who are at-risk for familial hypercholesterolemia.</p> <p>And what I tell you about is how you can actually sequence the gene,  the FH gene, see if you can find the mutation and see whether or not you  can then identify people who are at-risk for the disorder.</p> <p>So the first thing I want to tell you about today is DNA sequencing.  DNA sequencing. What is DNA sequencing? Does someone care to give me a  definition or think about what I might mean by DNA sequencing?</p> <p>In particular, what part of the DNA are we sequencing? Thank you, Jamie.</p> <p>You want to say it louder? The bases. Yes. So in DNA sequencing, and  maybe I even wrote this, what is this, what you want to do is to  determine the base sequence of the DNA. OK? You want to determine the  sequence of AGCT along a DNA fragment.</p> <p>This technique is powerful beyond almost anything else.</p> <p>It's an extraordinary technique. The ability to sequence DNA is  extraordinary. And it's extraordinary because you can get out of it  information that is absolutely essential for understanding life.</p> <p>What you can get from DNA sequencing is an understanding of the  coding capacity of a gene. So, just like you did in your exam, we gave  you a string of DNA and you conceptually translated it into the protein.</p> <p>Well, you can do that in real life by looking through the genome, the  human genome and finding stretches of DNA and conceptually turning them  into RNA and into protein and saying, OK, is this is a gene? Does it  code for something? And what does it code for? So you can figure out the  coding capacity of a gene.</p> <p>Part of that is actually identifying is a gene a gene?</p> <p>So we've sequenced the entire human genome. And I've told you  previously that only about 5% of the genome is actually genes and the  rest is other stuff.</p> <p>So one of the things you want to do with DNA sequencing is to  identify genes. And that's actually very difficult to do it turns out.  But that's one of the things you can do with DNA sequencing. I'll talk  more about identifying genes that are associated with disease, that are  causative of disease.</p> <p>And particularly alleles that are associated with disease such as in the case of familial hypercholesterolemia.</p> <p>One can figure out evolutionary relationships between organisms. So  you've probably heard for years about how similar we are to chimpanzees  or how similar we are to dogs or to dolphins or whatever.</p> <p>But, actually, we didn't really know. Now we can sequence a human  genome, we can sequence a chimp genome, a dog genome, a dolphin genome,  and we can actually look and see how similar we are.</p> <p>And we can try to figure out, in evolutionary time, what's changed  between the dolphin and ourselves and what makes a dolphin a dolphin and  ourselves ourselves.</p> <p>It's a very tough question, but DNA sequencing is essential for  trying to answer that kind of question. And then one can ask about the  genome is other ways. Can one find the promoters of all the different  genes? Remember promoters that make genes be transcribed? The  centromeres, the middle of chromosomes.</p> <p>Various other elements in the genome that are essential for its function.</p> <p>So I'm going to spend quite some time talking about DNA sequencing  and tell you that DNA sequencing, most of the DNA sequencing we do uses a  trick. And it's a terrific trick. It really is. So this DNA sequencing,  I'll write it because I don't think I have this on one of your  PowerPoints.</p> <p>The method of DNA sequencing I'm going to tell you about was devised by a scientist called Fred Sanger.</p> <p>So I'll tell you about it. It's called dideoxy, it's also called chain termination, and it's also called Sanger sequencing.</p> <p>Professor Sanger is a British scientist who received two Nobel  Prizes. The first was for figuring out how proteins, how to sequence  proteins, and the second was for figuring out how to sequence DNA.</p> <p>When I was a student, I heard Professor Sanger talk. And he gave a  lecture which was really memorable. It was packed, a packed auditorium.  And he spoke the entire time like this. I don't think he looked up once.</p> <p>He gave the entire lecture like this, and he was barely audible.</p> <p>But at the end of the lecture he got a standing ovation from  everybody because really what he's done, figuring out how to sequence  proteins and how to sequence DNA was really an extraordinary  accomplishment.</p> <p>So that's the method I'll tell you about. And it uses a cool trick.  So you know now that the sugar in DNA has a 3 prime hydroxyl group, and  that hydroxyl group is the group unto which the phosphate gets added.</p> <p>Right? And without that hydroxyl group you could not add on the next nucleotide, right? It's a question.</p> <p>Think about it. OK? I don't mean it to be rhetorical. I want you to  really be thinking, OK, about this, because otherwise you won't  understand the method. So here's the 3 prime hydroxyl on regular  deoxyribose.</p> <p>OK?</p> <p>In the Sanger or dideoxy method one uses in the reaction mix, and  I'll go through this with you in a moment, a sugar or nucleotide that's a  dideoxy nucleotide. In other words, on both the 2 prime and the 3 prime  of the sugar, of the ribose there is no hydroxyl group.</p> <p>There are just those hydrogens.</p> <p>Now, a dideoxy nucleotide such as this one can get incorporated into  DNA just fine because this phosphate, the triphosphate here can react  with a regular nucleotide that's got a 3 prime hydroxyl.</p> <p>However, once it's been incorporated you cannot elongate the chain  anymore because there is no reactive hydroxyl group. OK. So based on  this principle let me explain.</p> <p>I've got one of your handouts here.</p> <p>OK. So here we go. Revision, your template, your primer, here's your  template strand, always goes 3 prime to 5 prime. Here's your 5 prime to 3  prime primer. If you add nucleotides, deoxynucleotide triphosphates and  DNA polymerase, you will polymerize the whole fragment.</p> <p>If you add, however, to the mix of dNTPs and DNA polymerase a  low-level of dideoxy nucleotide triphosphates, every time you add on a  nucleotide the polymerase can either use a regular nucleotide  triphosphate, in which case the chain can elongate subsequently, or it  can use a dideoxy nucleotide triphosphate.</p> <p>If it uses one of the dideoxy NTPs the chain will terminate. It cannot be elongated any further.</p> <p>So you get something like this. And the trick here is really this  low-level of ddNTPs. OK? So if you have your template and your primer  and you do a reaction with your dNTPs at a reasonable level and you  spike the reaction with a low-level of dideoxy NTPs, you get a whole  bunch of different length chains polymerized.</p> <p>Because there is some probability, at every position, that you're  either going to get a ddNTP incorporated, in which case the chain  terminates, or you're going to get a regular nucleotide incorporated in  which case the chain can continue for a bit.</p> <p>OK? So that is paramount to dideoxy sequencing. So let's continue now  by looking at a specific polymer and following through exactly what  happens.</p> <p>So here I've given you a template and a primer. And we're going to do the same reaction that we just did conceptually.</p> <p>We're going to do it again conceptually except with letters. We're  going to mix together. And we're going to do, and I see a mistake up  here already, but that's OK. You'll bear with me. What I've done here is  to put in some dideoxy ATP.</p> <p>And I meant to say here I've got dATP at high levels.</p> <p>And I've got all the other nucleotides here, too, at high levels. OK?  That's my error and I will correct it. You should correct it now in  your handout. So where it says dATP high, that should actually say dNTPs  high, not just dATP.</p> <p>OK? All right. So let's look and see what happens to this reaction.  And I've noted here that this dideoxy ATP can be radioactive or  florescent.</p> <p>Or actually it doesn't have to work that way but let's just leave it that way for now.</p> <p>OK. That actually is not necessarily true. So let's just focus on the  ddATP plus the high dNTPs, and let's see what happens. OK. So one thing  that can happen is that, here's your primer in red and here's the  polymerized DNA in blue, you get a bit of DNA polymerase.</p> <p>Now here's an A.</p> <p>See? It goes GAGTAA. And I've given you a reaction where the first  two As use regular dATP. And so the chain will continue after that. All  right? So here we go, GAGTA. And then the next A that's put in is a  dideoxy A.</p> <p>And that's the end of that polymerization reaction, and the fragments  you're going to get out of it is this little red and blue composite  there.</p> <p>You can do the same thing where you say actually in some molecules  you get polymerization past the second A, and you keep going until you  get to the next A.</p> <p>And at that point, by chance, you get a dideoxy ATP added to some  molecules. That is the end of polymerization for those molecules. The  chain terminates. For some molecules, however, you'll put in a regular  dATP and the chain will continue.</p> <p>But it will terminate, excuse me, at the next A that's put in because you put a dideoxy A in.</p> <p>So in different molecules you're going to land up with a spectrum of  elongated products of different length. All right? And what's crucial  here is that the length of the molecules that chain terminate, because  they incorporated dideoxy nucleotide, correspond to the position of that  particular nucleotide along the chain.</p> <p>So you're only going to get a molecule chain terminating with A when there was a T on the template strand.</p> <p>OK? And so you can map the positions of the T on the template or the A  on the elongated strand by the length of the elongated products that  come out of this reaction. I'm going to assume you're with me here.</p> <p>OK.</p> <p>So the point is the polymerized fragments terminate where dideoxy A  incorporates. Now, you've got to do four reactions to determine the  sequence of something. OK. And I've noted here. And the length of the  terminated fragment indicates the position of A.</p> <p>You may need to go and work with this a bit. OK? It's a very clever  method but it may not be something that's immediately apparent, so go  and work with it if you need to.</p> <p>So the length of the terminated fragments indicates the positions of A  in the elongated strand, or if you want in T of the template strand.</p> <p>In order to get the positions of all the different nucleotides along  that DNA fragment you have to do four separate reactions. One that  includes dideoxy ATP, one that includes dideoxy CTP, one dideoxy GTP and  one dideoxy TTP.</p> <p>And you do those separately so that you can monitor the positions of  each of those four nucleotides by the position of chain terminating as  you're going along. OK. So assuming that you guys are with me here at  this point, are you? No.</p> <p>That's an honest answer. Raise your hands if you're with me. OK. If  you're not with me, don't worry about. You have to go work with it. It's  not intuitive. It's very clever. I mean there's a reason this guy got  the Nobel Prize for this.</p> <p>OK?</p> <p>It's a really clever method. OK. So the deal is this. So now what you  get out of this is a whole mix of fragments of different lengths that  have terminated at positions of particular nucleotides, depending on how  you've spiked the reaction.</p> <p>And you've got to separate them from one another somehow to figure  out what those positions are. And you can do this in a couple of ways.</p> <p>You can use gel electrophoresis, which was discussed with you  previously, where you separate the DNA on the basis of size where the  DNA migrates in a gel in an electric field and long fragments stay near  the top of the gel and short fragments go to the bottom of the gel  because they migrate quickly.</p> <p>And what you can do on a gel, and you've somehow labeled, don't worry  about this right now, but somehow you're able to detect each of the  fragments that has come out of your mix.</p> <p>OK? So remember you're doing the sequencing reaction on millions and millions or billions of molecules.</p> <p>And so you've got this kind of stochastic mix of molecules of  different lengths. And you want to separate this mix of molecules of  different lengths. OK. So what you can end up with, once you've  separated all these different molecules, is in your dideoxy A reaction  mix a series of one, two, three, four, five different sized fragments.</p> <p>In your ddG mix, you got out of that also a series of five different sized fragments.</p> <p>And notice that they're different in size from the ones in the ddA  lane, the ones in the ddC lane and the ones in the ddT lane. And the  reason they're different in size is because their size indicates the  position of where a particular nucleotide is in the DNA fragment or  particular bases in the DNA fragment.</p> <p>And then the trick is you could look at this gel and you could read off the sequence.</p> <p>So the shortest fragments that you're going to get are the ones that  are nearest the beginning of that molecule you made, nearest the 5 prime  end. So the bottom one is G, here's the band in the ddG lane.</p> <p>Then up above it there is this band indicating a fragment in the ddA lane.</p> <p>Above it there's one in the G lane again. Above it there's one in the  T lane. So the sequence goes G-A-G-T, and then you can keep reading  A-A-C-G-G-T-A-T-G-C-A.</p> <p>OK? Literally like that on a gel. OK? So you can do that on a gel.  It's really fantastic. And this is what old sequencing gels look like.</p> <p>And, actually, I used to run them. I used to spend hours and hours running these gels.</p> <p>They're very, very thin. They're about a millimeter thick acrylamide  so that you can resolve the fragments that are one nucleotide different  in size. Think about that. OK? Each of these fragments, indicated by a  band, is one nucleotide different in size.</p> <p>Otherwise, you couldn't get the one nucleotide resolution.</p> <p>So you do that by running very, very thin gels so that you can  resolve the fragments well, and then you read off the bottom. OK? I've  thrown out all my old sequencing gels.</p> <p>And the reason that I have is that there is new technology where you  don't use this kind of display anymore. This is a display where your  fragments were labeled with radioactivity and you exposed them to x-ray  film and you read the sequence after exposure.</p> <p>Nowadays this is done by machine.</p> <p>And the dideoxy nucleotides are labeled fluorescently. OK? So they're  not labeled with radioactivity. They're literally labeled with labels  that fluoresce with different colors when you put UV light on them.</p> <p>And you do your dideoxy reaction and you run a gel. Again, it's a gel.</p> <p>It's actually a very thin tube of a gel mostly, but your run your  gel. And, again, it's the same idea. You resolve fragments at single  base resolution, single nucleotide resolution, and they keep, the gel  keeps running and running.</p> <p>And single fragments actually run off the bottom of the gel. And as  they're passing down the gel they are detected by a laser. A laser  excites the fluorochrome.</p> <p>And the detector, there is a detector which will detect whether or not it's yellow, orange, blue or green.</p> <p>OK? And that will tell you which base is being, has been incorporated  at that position. So you get things that come out. It's kind of small  but you can go back and look, where instead of getting a gel with those  bands that I showed you, you get these peaks and valleys that are  different colors.</p> <p>And that's what current DNA sequencing readout looks like.</p> <p>And, in fact, there are machines. What did I do? Lots of primers.  Well, it depends. Many copies of the same primer, right. Yes. Dr. Gardel  is pointing out that there are many copies of the same primer in a  reaction mix.</p> <p>Certainly there are. There are billions of molecules in the reaction mix, and so there are billions of primers.</p> <p>OK, so you have to have a primer for each molecule. OK. And each band, you should realize, is not a single molecule.</p> <p>It's a composite of many, many molecules, many thousands of molecules  that have all chain terminated at the same position. So what I want to  point out here is that this is what today's readout looks like.</p> <p>And, in fact, nowadays you just get a printout from the company or from the machine that tells you a DNA sequence.</p> <p>And it's this improvement in technology, but that basically uses this  chain termination method, that has allowed one to sequence, rapidly  enough to sequence the human genome and to sequence multiple human  genomes in multiple animals.</p> <p>OK. So let's see. Actually, I have a movie. I guess we can take the  time to watch this movie. Let's see if it will work. All right. So  primer template. Four reactions, each with lots of molecules, each with  their primer.</p> <p>DNA polymerase, dNTPs, dATP, dGTP, dCTP, dTTP, dCTP, excuse me.</p> <p>OK. They're your four reactions. OK. I think is a less dorky movie  than some. OK. So here we go. Here's your primer and your template, and  here's polymerization. And, ah, there we go, chain termination, dideoxy  nucleotide incorporation, and you cannot get elongation.</p> <p>The poor G is thwarted in its desire to elongate. OK?</p> <p>So you land up with this mix, just like I showed you, and you land up  with a set of four reactions, each with molecules of different lengths  in them.</p> <p>And here's your gel, and you load them on your gel, and they migrate  through your electric field. And there you have your things, you have  your fragments. This is a piece of x-ray film you put on top.</p> <p>There are your little bands, your radioactive bands, and here we go.</p> <p>GT, you can read it. OK. Enough. Enough. OK. You can go and look at  this yourself. This is an old gel apparatus that one used to do DNA  sequencing on.</p> <p>This was the first generation of machine that you could do the  fluorescent sequencing on. This is a room full of sequencing machines of  the kind that was used to sequence the human genome.</p> <p>In fact, many rooms of machines going all day and all night sequencing and sequencing and sequencing.</p> <p>We have a lot of nucleotides. And it takes a long time to sequence.  Although, in retrospect it's not such a long time. And now all the  sequencing machines that sequence the human genome are sitting around  looking for other work because they all exist.</p> <p>And so that is why we are sequencing things like dolphins and dogs  and multiple strains of dogs, multiple breeds, excuse me, of dogs  because we have all these sequencing machines sitting around.</p> <p>OK.</p> <p>Honestly, I think that's true, not that it's not useful. All right.  So I'm going to move on here. This is Professor Jack's joke that I  decided to use also. OK. This is something about DNA sequencing and the  implications of being able to use DNA sequencing for genotyping.</p> <p>So I'm going to use that. You can go and read that on your thing. I'm  going to move on right to talking about familial hypercholesterolemia  and the notion of a disease allele.</p> <p>So here's part of the normal FH gene, the LDL receptor gene, and here it is.</p> <p>And there is a T here in red. And here is the mutant gene sequence  and there is an A. So if you're wild type you have a T at this position  that's arrowed and if you're a mutant you have an A. And if you do your  conceptual protein translation here you get your amino acid, part of the  amino acid chain.</p> <p>Obviously it's not at the beginning.</p> <p>And obviously this is DNA and this is protein, so we've removed the  RNA here, the RNA step. And you can see here is the amino acid of your  wild type, the sequence of your wild type gene. And in your LDL receptor  mutant there is a stop codon at this position that terminates the LDL  receptor.</p> <p>And so the receptor gene is mutant and does not function as it should.</p> <p>OK. All right. So let me move onto the next thing I want to talk  about, which is this question of polymorphisms. What is a polymorphism?</p> <p>Anyone.</p> <p>All right. I'll tell you what a polymorphism is. A polymorphism is defined as some kind of variation in DNA sequence.</p> <p>And it's defined as a variation in DNA sequence at a particular position.</p> <p>So our DNA, all of us have very similar DNA.</p> <p>If we were to sequence me and we were to sequence you and we were to  sequence you, we would find that our DNA was greater than 99% identical.  If we lined up our three times ten to the ninth base pairs in a very  long line, we would find it was very similar.</p> <p>There was about 1% difference in sequence between each of us.</p> <p>And most of that, some of that corresponds to disease gene alleles.  We all are supposed to carry about a thousand bad genes, or a thousand  genes that if homozygous would give us something bad, and sometimes do.</p> <p>And some of those correspond to changes in differences in DNA sequence that are not directly in genes.</p> <p>All of these differences between different individuals are called polymorphisms, DNA sequence variation.</p> <p>And you can use these to help figure out whether or not someone has a  particular disease allele, and also you can use it to figure out where  the DNA from a sample comes from me or from you or from Dr.</p> <p>Gardel.</p> <p>OK? And I'll talk about this, using polymorphisms to map genotype.  I'm going to talk about a particular kind of polymorphism, and these are  called SNPs which is pronounced ìsnipî. This stands for single  nucleotide polymorphisms.</p> <p>So I've said again that human genomes are 99% identical, but there  are throughout the genome changes, differences between regions.</p> <p>Single nucleotide polymorphisms are variations in one region. Here's a sample sequence I made up.</p> <p>Here's a G in one individual and an A in another individual. And if  you take the population, you find very often that there just is a choice  of two, sometimes more, but often just a choice of two nucleotides in  one position.</p> <p>Most of the genomes are identical, but you find these little regions  where in many individuals of a population there are these variations.</p> <p>In fact, these variations have to be present in more than 1% of the population for this thing to be called a SNP.</p> <p>This is a definition that humans have given but it's a useful  definition as a genetic tool. So if there is a polymorphism present in  about 1% of the population, whereby I might have an A here, excuse me,  and Dr.</p> <p>Gardel has a G at that position, that would be a SNP, and we would be polymorphic for that SNP.</p> <p>In fact, my two chromosomes, OK, that are homologous chromosomes might on one copy carry an A and on the other copy carry a G.</p> <p>Now, these different bases are present at different frequencies. So,  for example, it might be very common to have a G at this position in the  sequence and it might be very rare to have an A at that position.</p> <p>All right?</p> <p>And that's useful because you can use the frequency of these  different nucleotides, these different bases to help you use the SNP to  genotype. And I want to point out that usually SNPs occur outside coding  regions because 95%, actually more than that, 99% of the genome is not  coding per se.</p> <p>95% is not genes, but then if you remove all the introns and promoters and so on, 99% does not code for any protein.</p> <p>OK. So usually these SNPs are present outside coding regions. So here's to explore this a bit more.</p> <p>You can find lots of these SNPs. There are about three million SNPs  in the human genome, and a very large percentage of those SNPs has been  identified by DNA sequencing. So you can get the idea. You have to  sequence DNA from lots and lots of individuals to identify these SNPs,  but people have done it.</p> <p>And we know now more than a million SNPs in the human genome that are  located all over different chromosomes, and we know where they're  located on different chromosomes.</p> <p>And so you can use these SNPs to make kind of a map, I'll tell you in  a moment. So here are some possible genotypes. I've given you a choice  of two for each of these. OK? So, for example, for this red SNP here you  can be AA, AC or CC on the two homologous chromosomes.</p> <p>All right. So let's keep going with this thread. So because you have  these SNPs all over your genome and you know where they are, you can use  them to make a map of your entire genome. That doesn't depend on the  genes.</p> <p>It just depends on the sequence. And knowing these SNPs is a lot  easier to work with than having to sequence the entire genome of  somebody every time you want some information.</p> <p>So you can use these SNPs to identify each person.</p> <p>So I have a SNP map of all these hundreds of thousands of SNPs, or up  to a million. The usual maps presently used are about 300,000 SNPs per  genome. I have a map of 300,000 SNPs where there are different,  actually, I don't, but I could, where there are different alleles at  different frequencies, different bases present at different frequencies  at specific positions.</p> <p>And we could pick any one of you and make a SNP map for you.</p> <p>And it would look really different from mine, not because the SNPs  themselves are different, they'd be the same SNPs, but the actual bases  and the combination of bases between all these different SNPs would be  different between different individuals.</p> <p>And this SNP-type map is the basis for DNA fingerprinting that is  used in forensics and to figure out disease alleles. I'll talk more  about this in a second. I want to point out that there are other kinds  of polymorphisms that are used in genotyping, restriction fragment  length polymorphisms and things called simple repeat polymorphisms.</p> <p>And you can look in your book for these restriction fragment length polymorphisms, but let's talk more about SNPs.</p> <p>So SNP genotyping, here's a whole list, but the ones I'm going to focus on are disease gene mapping and forensics.</p> <p>Also, you use SNP genotyping for paternity suits. OK? So if someone  comes and, you know, if someone says it's my kid and the other one says  it's my kid, you can figure out very easily whose it is by looking at  these various SNPs and figuring out what pattern of SNPs is present in  the offspring.</p> <p>OK.</p> <p>So let me actually consider, let me not deal with genotyping for  disease alleles at this point. Let me talk about forensics a bit because  it's kind of interesting. So how do you do this? Let's look through  this slide.</p> <p>You have it as a handout. Here are SNPs. And I've just given you two  chromosomes each with two SNPs. OK? And different people will have  different bases at these particular SNPs, or they'll have different  combinations of these bases.</p> <p>So here's the spot of blood at the crime scene.</p> <p>OK? Our red blood cells do not have nuclei so you cannot get DNA from  those, but there are enough white blood cells that do have nuclei so  you can. And, actually, you know from PCR now that you need very little  to amplify something up by PCR.</p> <p>One cell is sufficient, right? It's pushing the technology, but you can really use one cell.</p> <p>So there are plenty of cells in a spot of blood at a crime scene to  isolate the DNA and to PCR amplify the regions surrounding the SNP.</p> <p>So you're not just dealing with these two nucleotides or the choice  of these two nucleotides at the SNP. You've got a little piece of DNA  that's usually maybe 20 or so bases that includes this choice of single  nucleotide polymorphism.</p> <p>So you amplify the SNP region, OK, a region that's constant, that  includes the nucleotide polymorphism, and you determine the sequence at  the different single nucleotide polymorphism regions. So you might get  someone who, at the red position you an be A or C, at the green you can  be G.</p> <p>OK, let's have an example here. You can get genotypes where at red  you're A or C, green you're G or G, purple GT, and yellow you can be A  or C.</p> <p>And here the example is C and C. So here are the four suspects, numbers one to four.</p> <p>OK. And here are their genotypes. OK. And here is the spot of blood  at the crime scene that actually has this genotype. OK. So let me go  back here. This is the genotype in the blood at the crime scene.</p> <p>OK. So the red sequence on one chromosome is an A, on the other is a C, so you have AC.</p> <p>On the other, the green sequence you have GG, purple you have GT, and  yellow CC. So you're looking to see whether or not any of the suspect  genotypes map up with a spot of blood, right? So we're assuming that a  spot of blood, you know, comes from one of the suspects that was  attacked by the person who was the victim.</p> <p>OK. So you have a victim with scratch. Someone has a spot of blood.</p> <p>And you see whether or not, or you can use semen samples, you can see  whether or not the DNA in the human tissue that is believed to come  from the attacker is matching of any of the suspects' genotypes.</p> <p>So there are a lot of assumptions there, right? You have to have  tissue at the crime scene that you believe to come from the attacker.  And then, once you have that, you can determine its genotype and compare  it to the genotypes of the suspects.</p> <p>And you find, for example, here that, let's see, yeah, so I believe  the suspect number three has the same genotype as the DNA that was in  the spot of blood at the crime scene.</p> <p>And that would be some evidence that this suspect number three was  the person who did it. Now, in actual fact, you do this not just for  four SNPs, you do it for thousands of SNPs.</p> <p>You don't usually do this for 300,000 SNPs because that's expensive and it's a lot of work.</p> <p>And forensics doesn't put that much money into this. However, the  more SNPs you use for genotyping the more sure you are of the suspect's  identity. OK? Because it's really a matter of frequency of whether or  not you're going to get the same combination of these different SNP  bases in different potential suspects.</p> <p>So the greater the spectrum of SNPs you look at, the more sure you are of the suspect's identity.</p> <p>Now, in some cases this has been very, very useful. And there are a  number of people on Death Row who have been exonerated by going back to  DNA recovered from the crime scene sometimes years ago, doing SNP  mapping and showing that they really couldn't have done it because the  genotypes did not match up.</p> <p>Usually these were rape cases and the semen genotype just did not match up with the semen genotype of the person on Death Row.</p> <p>So this is very valuable technology. OK. It was used in the O.J.  Simpson trial, but not as well as it could have been which lead to  equivocation there.</p> <p>OK. So time is fleeting. I'm going to mention a technology to you in  the last couple of minutes, and then we'll come back to it as we go on  through later parts of the course. So I've talked today about DNA  sequencing.</p> <p>I've talked about using polymorphisms to genotype people either, well, for disease alleles I focused on who-done-its.</p> <p>Something else that I want to throw out at you at this point is the  notion of transgenic technology. And I'm going to tell you what  transgenic organisms are as part of completing the Recombinant DNA  Module.</p> <p>And then we'll come back in future modules and talk more about how you make these things.</p> <p>But I want to have this as part of your compendium now. A transgenic  animal or transgenic organism is an organism where you have manipulated  its genome in some way, where you've either inserted extra DNA into its  genome or you've removed DNA from its genome or you've done something to  its genome such that it was not the organism that you started off with.</p> <p>Genetically modified organisms.</p> <p>The food that you eat that is genetically modified has had its genome  tampered with. This type of transgenic technology is very, very useful,  not only for creating genetically modified foods, but it's very, very  useful for creating disease models of animals.</p> <p>And I'll tell you now that there is a mouse model of human familial  hypercholesterolemia that has been created by making a specific  mutation, that T to A mutation in the mouse LDL receptor gene.</p> <p>Another thing that is extremely useful about transgenic animals is that you can get them to make specific proteins.</p> <p>So, for example, there are goats that have had inserted into their  genomes genes that encode for particular medications, for particular  drugs. And you can get these drugs out of the milk of the goats usually  or out of the serum of the goats because they are constitutively  producing them because you've put various genes into their genome.</p> <p>So I'm going to leave it there and we'll talk about how to make transgenics in a future lecture.</p>
</div>
        <div id="vid_related" itemprop="description" class="tabContent hide">
<h2 class="subhead">Free Downloads</h2>
<h3 class="subsubhead">Audio</h3>
<ul>
<li>iTunes U (<a href="http://deimos3.apple.com/WebObjects/Core.woa/Browse/mit.edu-dz.2820703216?i=2085243275">MP3 - 11MB</a>)</li>
<li>Internet Archive (<a href="http://www.archive.org/download/MIT7.013S05/Lecture10.mp3">MP3 - 11MB</a>)</li>
</ul>
<br><h3 class="subsubhead">Subtitle</h3>
<ul><li>English - US (<a href="../../../contents/audio-lectures/15-recombinant-dna-3/7.013-2006-L15.srt">SRT</a>)</li></ul>
<br>
</div>
    
   </div>  


      					 
        <div class="" id="parent-fieldname-bottom_html_area">
            
            
        </div>
    
                    </div>
<!--Course_inner_chip tag close -->
           		</div>
<!--Course_wrapper tag close --> 
            </div>
<!--left tag close -->
            <div id="right">
                <!--Begin Right Portion -->
                    <div>
    
<div id="portletwrapper-6f63772e7269676874746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a646f6e617465" class="portletWrapper kssattr-portlethash-6f63772e7269676874746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a646f6e617465">
<div class="portletStaticText portlet-static-donate"><p class="zero"><a href="http://ocw.mit.edu/donate"><img src="../../../common/images/button_donate-now.png" alt="Donate Now." class="donate"></a></p></div>

</div>




</div>

                	<div>
    



</div>


        <div class="" id="parent-fieldname-rsi_top_html_area">
            
            
        </div>
    

<!-- RSI google ad space-->


<div id="google_ads">    
    <script type="text/javascript" src="http://partner.googleadservices.com/gampad/google_service.js"></script><script type="text/javascript">GS_googleAddAdSenseService("ca-pub-6588555046597237");GS_googleEnableAllServices();</script><script type="text/javascript">GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_A_DL");GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_B_DL");GA_googleAddSlot("ca-pub-6588555046597237", "VIDEO_INDIVIDUAL_SLOT_C_DL");</script><script type="text/javascript">GA_googleFetchAds();</script><script language="javascript" type="text/javascript">
GA_googleAddAttr("TYPE","HOUSE");
GA_googleAddAttr("DEPARTMENT","7");
GA_googleAddAttr("CRS_BEG2","01");
GA_googleAddAttr("CRS_END","3");
GA_googleAddAttr("SESSION","S");
GA_googleAddAttr("YEAR","06");
</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_A_DL");</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_B_DL");</script><script type="text/javascript">GA_googleFillSlot("VIDEO_INDIVIDUAL_SLOT_C_DL");</script>
</div>

<!-- End RSI ads--> 

<div>
    



</div>

            </div>
<!--Right div close -->
            <div class="clear"></div> 
        </div>
<!--grid tag close --> 
      </div>
		
		<div id="bottom">
			<div id="grid">
				
<div id="portletwrapper-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572" class="portletWrapper kssattr-portlethash-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572">
<div class="portletStaticText portlet-static-site-footer"><div id="bottom">
<div id="grid">
<!-- *begin footer* -->
<div id="footer" role="navigation sitemap">
<div id="foot-c1" class="grid_2 alpha">
<h4 class="footer">Courses</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/courses/find-by-topic/">Find by Topic</a></li>
    <li><a href="http://ocw.mit.edu/courses/find-by-number/">Find by Course Number</a></li>
    <li><a href="http://ocw.mit.edu/courses/find-by-department/">Find by Department</a></li>
    <li><a href="http://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/translated-courses/">Translated Courses</a></li>
    <li><a href="http://ocw.mit.edu/resources/">Supplemental Resources</a></li>
</ul>
</div>
<div id="foot-c2" class="grid_2">
<h4 class="footer">About</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/about/">About OpenCourseWare</a></li>
    <li><a href="http://ocw.mit.edu/about/site-statistics/">Site Stats</a></li>
    <li><a href="http://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
    <li><a href="http://ocw.mit.edu/about/ocw-consortium/">OCW Consortium</a></li>
    <li><a href="http://ocw.mit.edu/about/media-coverage/">Media Coverage</a></li>
    <li><a href="http://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
    <li><a href="http://ocw.mit.edu/about/media-coverage/press-releases/">Press Releases</a></li>
    <li><a href="http://ocw.mit.edu/about/next-decade/">OCW's Next Decade</a></li>
</ul>
</div>
<div id="foot-c3" class="grid_2">
<h4 class="footer">Donate</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/donate">Make a Donation</a></li>
    <li><a href="http://ocw.mit.edu/donate/why-donate/">Why Donate?</a></li>
    <li><a href="http://ocw.mit.edu/donate/ocw-course-champions-program/">Become a Course Champion</a></li>
    <li><a href="http://ocw.mit.edu/donate/our-supporters/">Our Supporters</a></li>
    <li><a href="http://ocw.mit.edu/donate/other-ways-to-contribute/">Other Ways to Contribute</a></li>
    <li><a href="http://ocw.mit.edu/donate/shop-ocw/">Shop OCW</a></li>
    <li><a href="http://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
</ul>
</div>
<div id="foot-c4" class="grid_2">
<h4 class="footer">Featured Sites</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/high-school/">Highlights for High School</a></li>
    <li><a href="https://www.edx.org/school/mitx/allcourses">MITx</a></li>
    <li><a href="http://teachingexcellence.mit.edu/">Teaching Excellence at MIT</a></li>
</ul>
<h4 class="footer" style="margin-top: 14px;">Tools</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/help/">Help &amp; FAQs</a></li>
    <li><a href="../../../common/jsp/feedback.htm">Contact Us</a></li>
    <li><a href="../../../common/search/AdvancedSearch.htm">Advanced Search</a></li>
    <li><a href="http://ocw.mit.edu/help/site-map/">Site Map</a></li>
    <li><a href="../../../common/terms/index.htm">Privacy &amp; Terms of Use</a></li>
</ul>
</div>
<div id="foot-c5" class="grid_4 omega" itemprop="publisher" itemscope="" itemtype="http://schema.org/CollegeOrUniversity">
<h4 class="footer">About <span itemprop="name">MIT OpenCourseWare</span>
</h4>
<p class="accent" itemprop="description">MIT OpenCourseWare makes the materials used in the teaching of almost all of MIT's subjects available on the Web, free of charge. With more than 2,000 courses available, OCW is delivering on the promise of open sharing of knowledge.</p>
</div>
<div id="foot-copy" class="grid_8 alpha">
<a href="http://web.mit.edu"><img src="../../../common/images/logo_mit.png" alt="Massachusetts Institute of Technology logo and name." style="width: 195px; height: 44px;"></a>                 <a href="http://www.ocwconsortium.org/"><img src="../../../common/images/logo_ocwc.gif" alt="OpenCourseWare Consortium logo." style="width: 80px; height: 44px;"></a>                 <a rel="license" itemprop="useRightsUrl" href="http://creativecommons.org/licenses/by-nc-sa/3.0/us/deed.en_US"><img src="../../../common/images/cc_by-nc-sa.png" alt="Creative Commons logo with terms BY-NC-SA." style="width: 126px; height: 44px;"></a>
<p class="copyright">© 2001–2013<br>
Massachusetts Institute of Technology</p>
<p style="font-size: 0.9em; margin-bottom: 15px;">Your use of the MIT OpenCourseWare site and materials is subject to our <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/us/deed.en_US" rel="license">Creative Commons License</a> and other <a href="../../../common/terms/index.htm" rel="cc:morePermissions">terms of use</a>.</p>
</div>
<div id="foot-support" class="grid_4 omega"> </div>
</div>
</div>
</div></div>

</div>





                
			</div> <!-- bottom grid end -->
		</div>
<!-- bottom end -->
		
		
   </body>
</html>
