<!DOCTYPE html><html lang="en">
<head>
<meta charset="utf-8">
<meta name="format-detection" content="telephone=no">
<title>Module 1.5: M13.1 | Labs | Laboratory Fundamentals in Biological Engineering | Biological Engineering | MIT OpenCourseWare</title>
<!-- Begin Automatic Metadata Insertion --><meta content="20-109-laboratory-fundamentals-in-biological-engineering-fall-2007" name="WT.cg_n">
<meta content="Module 1.5: M13.1" name="WT.cg_s">
<meta content="" name="Description">
<meta content="Endy, Drew" name="Author">
<meta content="Kuldell, Natalie" name="Author">
<meta content="Lerner, Neal" name="Author">
<meta content="Stachowiak, Agi" name="Author">
<meta content="Belcher, Angela M." name="Author">
<meta content="Banuazizi, Atissa" name="Author">
<meta content="biological engineering,biology,bioengineering,DNA,PCR,RNA,polymerase chain reaction,systems engineering,DNA engineering,protein engineering,bio-material engineering,restriction map,lipofection,screening library,bacterial photography,device characterization,biological parts,openwetware,Biomaterials,Biological Engineering,Genetics,Systems Engineering" name="keywords">
<meta content="20.109 Laboratory Fundamentals in Biological Engineering | Module 1.5: M13.1" name="Search_Display">
<meta content="Biomaterials" itemprop="about">
<meta content="Biological Engineering" itemprop="about">
<meta content="Genetics" itemprop="about">
<meta content="Systems Engineering" itemprop="about">
<!-- End Automatic Metadata Insertion --><link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/grid.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/base.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/menu.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/jquery.bubblepopup.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/courses.css">
<link title="default" rel="stylesheet" type="text/css" href="../../../common/styles/courses_new.css">
<link href="http://ocw.mit.edu/xml/ocwcc.rdf" type="application/rdf+xml" rel="metadata">
<link rel="canonical" href="http://ocw.mit.edu/courses/biological-engineering/20-109-laboratory-fundamentals-in-biological-engineering-fall-2007/labs/mod1_5">
<link rel="apple-touch-icon" href="../../../common/images/apple-touch-icon.png">
<script type="text/javascript" src="../../../common/scripts/common-utils-offline.js"></script><script type="text/javascript" src="../../../common/scripts/ocw-offline.js"></script><script type="text/javascript" src="../../../common/scripts/jquery.bubblepopup.min.js"></script><script type="text/javascript" src="../../../common/scripts/bubble-popup-offline.js"></script>
</head>
<body itemscope itemtype="http://schema.org/WebPage">
        
	

        <div id="top">
			<div id="grid">
				
				
					
<div id="portletwrapper-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572" class="portletWrapper kssattr-portlethash-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572">
<div class="portletStaticText portlet-static-site-header">
<div id="banner" role="banner" class="grid_6 alpha"><a href="http://ocw.mit.edu/"><img src="../../../common/images/ocw_mast.png" alt="MIT OpenCourseWare, Massachusetts Institute of Technology" class="logo"></a></div>
<form action="http://ocw.mit.edu/subscribe/index.htm?utm_source=header" method="get">
    <div id="subscribe" role="form toolbar" class="grid_6 omega">
    <div class="module">
    <table class="social"><tbody><tr>
<td class="text">Subscribe to the<br>
                OCW Newsletter</td>
                <td class="black"><input type="text" onclick="clearEmailBox()" onblur="fillEmailBox()" value="Enter Email" class="greytext emailField" id="email" name="fromemail"></td>
                <td class="black"><input type="image" src="../../../common/images/button_subscribe.png" alt="Subscribe" class="sub_button"></td>
                <td>
<a href="http://facebook.com/mitocw"><img src="../../../common/images/icon_fb.png" alt="Click to visit our Facebook page."></a><a href="http://twitter.com/mitocw"><img src="../../../common/images/icon_tw.png" alt="Click to visit our Twitter feed."></a>
</td>
            </tr></tbody></table>
</div>
    <p class="helplinks"><a href="http://ocw.mit.edu/help">Help</a>   |   <a href="../../../common/jsp/feedback.htm">Contact Us</a></p>
    </div>
    <div class="clear"> </div>
</form>
</div>

</div>





<div id="mega" role="navigation" class="grid_8 alpha">        
	<ul id="menu">
<li id="menu_home">
            <a href="http://ocw.mit.edu/"><img src="../../../common/images/top-nav_home.png" class="home_icon" alt="Click for site home page."></a><!-- Begin Home Item -->
        </li>
<!-- End Home Item -->        
        <li class="selected">
            <a href="#" class="drop">Courses</a><!-- Begin 5 columns Item -->
            <div class="dropdown_5columns-a mega-courses">                    
                <div class="col_1a">
                    <div class="row_1a">
                        <div class="quart">
                            <h2 class="nav">Find courses by:</h2>
                            <ul class="nav-bullet find_by">
<li><a href="http://ocw.mit.edu/courses/find-by-topic/">Topic</a></li>
                                <li><a href="http://ocw.mit.edu/courses/find-by-number/">MIT Course Number</a></li>
                                <li><a href="http://ocw.mit.edu/courses/find-by-department/">Department</a></li>
                            </ul>
</div>
                        <div class="quart">
                            <h2 class="nav">Collections</h2>
                            <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Lectures</a></li>
                                <li><a href="http://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
                                <li><a href="http://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
                                <li><a href="http://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
                                <li><a href="http://ocw.mit.edu/resources">Supplemental Resources</a></li>
                            </ul>
</div>
                        <div class="clear"> </div>
                    </div>
                    <div class="row_1b">
                        <h2 class="nav">Cross-Disciplinary Topic Lists</h2>
                        <div class="quart">
                        <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/energy-courses">Energy</a></li>
                            <li><a href="http://ocw.mit.edu/courses/entrepreneurship">Entrepreneurship</a></li>
                            <li><a href="http://ocw.mit.edu/courses/environment-courses">Environment</a></li>
                        </ul>
</div>    
                        <div class="quart">
                        <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/intro-programming">Introductory Programming</a></li>
                            <li><a href="http://ocw.mit.edu/courses/life-sciences">Life Sciences</a></li>
                            <li><a href="http://ocw.mit.edu/courses/transportation-courses">Transportation</a></li>
                        </ul>
</div>
                        <div class="clear"> </div>
                    </div>
                    <div class="clear"> </div>
                </div>
                <div class="col_1b">
                    <h2 class="nav">Translated Courses</h2>
                    <ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/courses/translated-courses/traditional-chinese">繁體字 / Traditional Chinese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/simplified-chinese">简体字 / Simplified Chinese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/spanish">Español / Spanish</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/portuguese">Português / Portuguese</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/thai">ภาษาเขียน / Thai</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/persian">فارسی / Persian</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/turkish">Türkçe / Turkish</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses/korean">(비디오)한국 / Korean</a></li>
                        <li><a href="http://ocw.mit.edu/courses/translated-courses">More...</a></li>
                    </ul>
</div>
            </div>
        </li>
        <li>
            <a href="" class="drop">About</a>
            <div class="dropdown_1column-a">
                <div class="col_1">
                    <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/">About MIT OpenCourseWare</a></li>
                    </ul>
<ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/site-statistics/">Site Stats</a></li>
                        <li><a href="http://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
                        <li><a href="http://ocw.mit.edu/about/ocw-consortium/">OpenCourseWare Consortium</a></li>
                    </ul>
<ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/about/media-coverage/">Media Coverage</a></li>
                        <li><a href="http://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
                        <li><a href="http://ocw.mit.edu/about/media-coverage/press-releases/">Press Releases</a></li>
                    </ul>
<ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/about/next-decade/">OCW's Next Decade</a></li>
                    </ul>
</div>
            </div>  
        </li>    
        <li>
            <a href="" class="drop">Donate</a>        
            <div class="dropdown_1column-a">
                    <div class="col_1">
                        <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/donate/">Make a Donation</a></li>
                            <li><a href="http://ocw.mit.edu/donate/why-donate/">Why Donate?</a></li>
                            <li><a href="http://ocw.mit.edu/donate/ocw-course-champions-program/">Become a Course Champion</a></li>
                            <li><a href="http://ocw.mit.edu/donate/our-supporters/">Our Supporters</a></li>
                            <li><a href="http://ocw.mit.edu/donate/other-ways-to-contribute/">Other Ways to Contribute</a></li>
                            <li><a href="http://ocw.mit.edu/donate/shop-ocw">Shop OCW</a></li>
                        </ul>
<ul class="nav-bullet">
<li><a href="http://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
                        </ul>
</div>
            </div>            
        </li>        
        <li>
            <a href="" class="drop">Featured Sites</a>        
            <div class="dropdown_1column-a">
                <div class="col_1">
                    <ul class="nav-bullet mega-div-bottom">
<li><a href="http://ocw.mit.edu/high-school/">Highlights for High School</a></li>
                    </ul>
<ul class="nav-bullet mega-div-top">
<li><a href="https://www.edx.org/school/mitx/allcourses">MITx</a></li>
                        <li><a href="http://teachingexcellence.mit.edu/">Teaching Excellence at MIT</a></li>
                    </ul>
</div>
            </div>            
        </li>
    </ul>
</div>
<div id="search" role="search" class="grid_4 omega">
    
    <form method="get" action="../../../common/search/AdvancedSearch.htm">
     	 <table class="search"><tbody><tr>
<td class="black"><input type="text" onblur="fillSearchBox()" onfocus="clearSearchBox()" maxlength="255" value="Search" name="q" class="greytext searchField" id="terms"></td> 			 
                    <td class="black"><input type="image" src="../../../common/images/button_search.png" name="btnG" class="sub_button"></td>			 
                    <td class="text2"><a href="../../../common/search/AdvancedSearch.htm">Advanced<br>Search</a></td>
                </tr></tbody></table>
</form>
</div>
<div class="clear"></div>
<!-- *end header* -->  

				
				
			</div>
<!-- top grid end -->
		</div>
<!-- top end -->
			
		<div id="center">
         <div id="grid">
		     <div id="left-section">
                 <div id="breadcrumb">
                      <p>

    <a href="http://ocw.mit.edu/">Home</a>
    
        »
        
    
    
        
            <a href="http://ocw.mit.edu/courses">Courses</a>
            
                »
                
            
            
         
    
    
        
            <a href="http://ocw.mit.edu/courses/biological-engineering">Biological Engineering</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../../contents/index.htm">Laboratory Fundamentals in Biological Engineering</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../../contents/labs/index.htm">Labs</a>
            
                »
                
            
            
         
    
    
        
            
            
            Module 1.5: M13.1
         
    
</p>

                 </div>
                 <div class="clear"></div>
                 <div id="section_title">
           			<h1 class="title" itemprop="name" property="dct:title">
        <span class="" id="parent-fieldname-title">
            Module 1.5: M13.1
        </span>
    </h1>
           	     </div>
           	     <div class="clear"></div>
           	     <div id="course_wrapper_section">
           	     	 <div id="course_nav">
           			 	



<script language="javascript" type="text/javascript">
function toggleMenu(objID) {
  if (!document.getElementById) return;
  var ob = document.getElementById(objID);
  ob.className = (ob.className == 'selected')?'': 'selected';
}
function toggleClass(id)
{
  var divtoggleClass= document.getElementById(id);
  divtoggleClass.className = (divtoggleClass.className == 'mO')?'mC': 'mO';
  return false;
}
function changeAlt(id)
{
  id.alt = (id.alt == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
  id.title = (id.title == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
}
</script><!--Left Nav Starts --><ul>
<li class="">
			   			<a href="../../../contents/index.htm">
		                  Course Home  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/syllabus/index.htm">
		                  Syllabus  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/lecture-notes/index.htm">
		                  Lecture Notes  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="selected">
			   			<a href="../../../contents/labs/index.htm">
		                  Labs  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/assignments/index.htm">
		                  Assignments  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../../contents/related-resources/index.htm">
		                  Related Resources  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    
		    
         	
	<!--second tal block close-->  
	
</ul>
<!--Left Nav Ends -->
</div>
           		     <div id="course_inner_section">
           		     	 
	       		     	<p class="instruction">Modules: <a href="../../../contents/labs/mod1_1/index.htm">1.1</a> | <a href="../../../contents/labs/mod1_2/index.htm">1.2</a> | <a href="../../../contents/labs/mod1_3/index.htm">1.3</a> | <a href="../../../contents/labs/examine_clones/index.htm">1.4</a> | 1.5 | <a href="../../../contents/labs/mod1_6/index.htm">1.6</a> | <a href="../../../contents/labs/mod1_7/index.htm">1.7</a></p> <h2 class="subhead">Introduction</h2> <p>"Divide and conquer" may be an effective military strategy but its usefulness is not limited to that arena. The reductionist approach has been an important means of understanding complex biological processes. By tweezing apart networks and pathways, the components that contribute to the overall behavior of the system may be understood in great detail. As you've seen, however, there isn't always a straightforward way to reassemble the component level understanding into a predictive and quantitative model for the system. Remember that's what happened with the T7 model that you read about last time, and the author's response to the limited success of the models is what makes that T7 work so novel. Rather than continue to tweeze apart and better understand the natural example, they built a surrogate T7 that was a better template for experimental work, easier to manipulate and analyze, easier to characterize and understand. Now you will help design a surrogate M13.</p> <p><img alt="T7.1 refactoring algorithm." src="../../../contents/labs/surrogate_13.jpg"></p> <p class="instruction">T7.1 refactoring algorithm.</p> <p>To help with your refactoring efforts, a step-by-step example is provided here. It's inspired by the T7 algorithm described in the <a href="http://www.nature.com/msb/journal/v1/n1/full/msb4100025.html">Chan, et al. paper</a>. The example provided here will almost certainly raise some but not all of the issues you will encounter as you try your own refactoring, so read it carefully and then use some or none of the example for your own work.</p> <p>This example addresses the overlap in the genes encoding p9 and p8. In the natural version of M13KO7, the promoter for gene 8 is in an unusual place and the stop codon for gene 9 overlaps the start codon for gene 8. This arrangement makes it nearly impossible to regulate expression of p9 and p8 independently—akin to the gas pedal of your car if it changed both the speed you travel and the volume of the radio.</p> <h2 class="subhead">Step 1: Identify Parts</h2> <p>As you've seen, many bacterial genes have definable "parts," i.e. genetic elements with particular purposes and without which the function of the DNA is impaired. Four parts make up a typical gene expression cassette: a promoter, a ribosome binding site, an open reading frame, and a terminator. In the Van Wezenbeek, et al. <a href="http://www.ncbi.nlm.nih.gov/sites/entrez?Db=pubmed&amp;Cmd=ShowDetailView&amp;TermToSearch=6254849&amp;ordinalpos=8&amp;itool=EntrezSystem2.PEntrez.Pubmed.Pubmed_ResultsPanel.Pubmed_RVDocSum%7C">M13 sequencing paper</a> that was published in 1980, these parts were identified. Tables of promoter, RBS and terminator elements from this paper are reprinted on the discussion page associated with this lab.</p> <p>Another useful resource for refactoring M13 is the <a href="http://partsregistry.org/Main_Page">Registry of Standard Biological Parts</a>. Many of the functional elements of the M13KO7 genome have been entered into that database. A table describing them all is included in the "reagents list" for today's lab.</p> <p>Armed with this information and the <a href="http://www.neb.com/nebecomm/tech_reference/restriction_enzymes/sequences/m13ko7.txt">M13KO7 sequence file</a>, a working copy of the sequence can be copied into a new MSWord document and the relevant portions of the sequence annotated. The following color scheme was used to refactor the genes for p9 and p8, and it would be very helpful if you could follow the pattern when you refactor your portion of the genome. (<strong>Note: In the previous sentence, "very helpful" means that you must follow this labeling scheme for genetic elements, or a superior scheme of your own invention, in which case you must provide a legend for your new scheme. If you don't do this it would be impossible to evaluate your work in a timely fashion, or quickly place DNA synthesis orders.</strong>)</p> <ul>
<li>
<strong>RBS</strong> = blue box</li>     <li>
<strong>promoters</strong> = underlined with -35 and -10 in blue letters, bolding -10, -35 nucleotide</li>     <li>
<strong>transcriptional terminators</strong> = if relevant, boxing in grey</li> </ul>
<p>As you box and color the features, you should note above the region what's what</p> <p><img alt="Step1: identify parts." src="../../../contents/labs/color_coding.jpg"></p> <p class="instruction">Step1: identify parts</p> <p>In reading just a little about promoter elements, for example in this <a href="http://web.archive.org/web/20100325233944/http://www.mun.ca/biochem/courses/3107/Topics/promoter_bacterial.html">refresher on bacterial promoters</a>, it's clear that the gene 8 promoter is much further upstream than most promoters are from the ORFs they direct. So it's worth figuring out if this promoter is still considered the relevant one. The 1980 reference that identified the promoter seems pretty confident:</p> <p><img alt="Promoter for gene 8?" src="../../../contents/labs/reading.jpg"></p> <p class="instruction">Promoter for gene 8.</p> <p class="instruction">From Van Wezenbeek P.M., et al. "Nucleotide sequence of the filamentous bacteriophage M13 genome: comparison with phage fd." <em>Gene</em> 11 (1980): 129-148. Courtesy of Elsevier, Inc., <a href="http://www.sciencedirect.com/">Science Direct</a>. Used with permission.</p> <p>However it's worth looking for more recent data. Some useful sites for the M13 refactoring are <a href="http://www.ncbi.nlm.nih.gov/sites/entrez">the NCBI taxonomy of viruses</a> and the <a href="http://www.ncbi.nlm.nih.gov/ICTVdb/ICTVdB/00.035.0.01.001.htm">International taxonomy of viruses</a>.</p> <h2 class="subhead">Step 2: Characterize Existing Parts</h2> <h3 class="subsubhead">Promoter</h3> <p>After noting the unusual position of the promoter for gene 8, it seems worth taking some time to compare the existing parts to "consensus" or average parts to know how well they compare.</p> <p><img alt="Promoter characterization table." src="../../../contents/labs/consensus.jpg"></p> <p class="instruction">Promoter characterization table.</p> <p>Weights were added as subscripts to each position in the promoter hexamers. If the base does not match the compilation base, then add ? and change to black.</p> <p><img alt="Step 2: characterize of promoter." src="../../../contents/labs/promoter.jpg"></p> <h2 class="subhead">RBS</h2> <p>To estimate the strength of the M13 RBS sequences read: <a href="http://partsregistry.org/Ribosome_Binding_Site">this review site</a> which includes some useful characterization information. Compare the M13 RBS to the characterized ones as well as the consensus sequence to get some sense of the relative strengths.</p> <p><img alt="RBS characterization table." src="../../../contents/labs/rbsvsconsensus.jpg"></p> <p class="instruction">RBS characterization.</p> <h2 class="subhead">Step 3: Identify Flanking Restriction Sites</h2> <p>An M13KO7mut8 scaffold (<a href="../../../contents/labs/m13k07scafold_m8.pdf">PDF</a>) has been made in the Endy lab to facilitate the refactoring work you are undertaking (Bryan Hernandez, a 20.109 student from last term, helped to design these changes; Felix Moser, a technician in the Endy Lab, made the changes and tested the resulting phage). The additional restriction sites were made by de novo DNA synthesis plus a little site directed mutagenesis. The resulting DNA sequence has 8 new unique restriction sites dotting the genome. The changes needed to add these 8 new sites do not change the resulting protein sequences and do not affect known parts like promoters or ribosome binding sites; however, for unknown reasons, the phage appears to grow about 50% less well than wild-type M13.1 (only very early characterization experiments have been done, titering the production of phage from cultures). The following three pictures should help you understand the overall scheme of where these new restriction endonuclease sites are, and how they might help you to make changes:</p> <p>Here is the entire M13KO7 plasmid, showing all the unique restriction endonuclease sites (notice how most of the unique cutters are outside of the M13 region of the plasmid!):</p> <p><img alt="Entire M13K07 plasmid showing single cutters." src="../../../contents/labs/m13k07map.jpg"></p> <p class="instruction">Entire M13KO7 plasmid showing single cutters.</p> <p>Next, here is a "zoom-in" view on a section of the M13 genome from the entire M13KO7 plasmid, from gene II through gene III (again note how few unique restriction sites can be found):</p> <p><img alt="M13K07 map showing single cutters in gene II to gene III region." src="../../../contents/labs/m13_genome.jpg"></p> <p class="instruction">M13KO7 map showing single cutters in gene II to gene III region.</p> <p>Finally, here is the same section for the M13.1 genome, with the new sites added:</p> <p><img alt="M13K07 map showing single cutters in gene II to gene III region." src="../../../contents/labs/m13_1_genome.jpg"></p> <p class="instruction">M13.1 Genome.</p> <p>Now, you can confirm that these unique sites exist using <a href="http://tools.neb.com/NEBcutter2/index.php">NEBcutter</a> from NEB and the M13KO7mut8 scaffold (<a href="../../../contents/labs/m13k07scafold_m8.pdf">PDF</a>) information. For this example the restriction sites that will be useful for refactoring the genes for p9 and p8 were changes to orange font and the enzyme name was noted above the sequence.</p> <p><img alt="Refactoring_Gene." src="../../../contents/labs/refactoring_gene.jpg"></p> <h2 class="subhead">Step 4: Redesign Parts</h2> <p>The refactoring example here addresses both the promoter region of the gene for p8 and the overlap between the gene 9 stop codon and the gene 8 start codon. Unstuffing the overlap requires adding some sequence between the stop and the start. A unique restriction site was added as spacer sequence.</p> <p><img alt="Step 4a: separate stop and start." src="../../../contents/labs/anotation_spacer.jpg"></p> <p class="instruction">Step 4a: separate stop and start.</p> <p>The start codon will not be read without a RBS upstream but a simple cut and paste of the usual p8 RBS will provide a nice pair of direct repeats for the phage to delete. Additionally since the next step will be to move the promoter closer to the gene for p8, presumably strengthening it, it seems wise to weaken the RBS for gene 8. A chimeric RBS was added that mixed the consensus region of the RBS from gene 9 with the flanking sequences from the RBS for gene 8. In a refactoring example when a large portion of sequence is being duplicated, such an ad hoc approach isn't feasible. Instead programs that shuffle codons and consider <a href="http://www.kazusa.or.jp/codon/">codon bias of the host</a> should be used.</p> <p><img alt="Step 4b: add chimeric RBS." src="../../../contents/labs/annotation_rbs.jpg"></p> <p class="instruction">Step 4b: add chimeric RBS.</p> <p>A promoter to direct RNA polymerase to gene 8 separately from the gene 9 was added. The best strength for this promoter is guesswork. Since gene 8 does not normally have a promoter nearby, it seemed wise to add a weak (i.e. far from consensus) promoter so the promoter from the arabinose BAD operon was chosen and inserted between the unique restriction site and the chimeric RBS.</p> <p><img alt="Step 4c: add weak promoter." src="../../../contents/labs/annotation_pbad.jpg"></p> <p class="instruction">Step 4c: add weak promoter.</p> <p>To polish off this refactoring, the spacer sequence between the -35 and the -10 elements would need to be specified. There could be some clever things to do with that spacer, for example add a unique restriction site. And TADA--a new and improve M13KO7! Now you give it a go....</p> <h2 class="subhead">Protocols</h2> <h2 class="subhead">Part 1: M13.1</h2> <p>Refactor a portion of the M13KO7 genome between the unique HpaI site in gene II through the unique BamHI site in gene III, guided by the algorithm described in the Refactoring T7 paper <a href="http://www.nature.com/msb/journal/v1/n1/suppinfo/msb4100025_S1.html">Supplementary Figure 1</a> and the gene 9/gene 8 refactoring example in the introduction to today's lab. For inspiration or comparison, you can look over the list of modifications proposed for the module 1 <a href="../../../contents/assignments/mod1/index.htm">portfolio assessment assignment</a>. If you make new parts for the registry or refine existing ones, add your parts to the <a href="../../../contents/labs/mod1_m13refactor/index.htm">M13 refactoring workpage</a>. Before you leave today you and your partner should hand in one copy of this refactoring effort, but you should also email a copy to yourself to refine your ideas further before submitting it in your portfolio.</p> <h2 class="subhead">Part 2: Oral Presentation Instruction</h2> <p>Over the course of the term, everyone taking 20.109 will be making two presentations to the class. One will be a presentation of a primary journal article related to either <a href="../../../contents/assignments/mod1_papers/index.htm">genome</a> or <a href="../../../contents/assignments/mod2_papers/index.htm">expression</a> engineering. The other presentation will be of a research idea for Module 3. Today our wonderful coach from the Writing program will come to lab to give a talk about giving talks. This presentation should be a welcomed chance to learn what's expected when you speak (in 20.109 and elsewhere) and to gather some great tips for giving polished and interesting seminars. You can follow up this instruction by re-reading the <a href="../../../contents/assignments/guide_oral_prese-htm/index.htm">Guidelines for oral presentations</a>.</p> <h2 class="subhead">Part 3: Sequencing</h2> <p>The invention of automated sequencing machines has made sequence determination a fast and inexpensive endeavor. The method for sequencing DNA is not new but automation of the process is recent, developed in conjunction with the massive genome sequencing efforts of the 1990s. At the heart of sequencing reactions is chemistry worked out by Fred Sanger in the 1970's which uses dideoxynucleotides.</p> <p><img alt="Normal Bases Versus Chain-terminating Bases." src="../../../contents/labs/be109nucleotide.jpg"></p> <p class="instruction">Normal Bases Versus Chain-terminating Bases.</p> <p>Photo removed due to copyright restrictions.</p> <p class="instruction">Sequencing gel</p> <p><img alt="Sequence Trace Data." src="../../../contents/labs/sequence_data.jpg"></p> <p class="instruction">Sequence Trace Data.</p> <p>These chain-terminating bases can be added to a growing chain of DNA but cannot be further extended. Performing four reactions, each with a different chain-terminating base, generates fragments of different lengths ending at G, A, T, or C. The fragments, once separated by size, reflect the DNA's sequence. In the "old days" (all of 10 years ago!) radioactive material was incorporated into the elongating DNA fragments so they could be visualized on X-ray film (image on left). More recently fluorescent dyes, one color linked to each dideoxy-base, have been used instead. The four colored fragments can be passed through capillaries to a computer that can read the output and trace the color intensities detected (image on right). Your sample will be sequenced in this way on an ABI 3730 DNA Analyzer.</p> <p>Sequencing will be one of the last steps to assess the success of your work on the gene for p3. From your experiment last time, you may feel confident that your oligonucleotide pair was successfully inserted into the M13KO7 genome. What remains unclear, however is which direction the insert has gone in. Since the restriction sites are so close together, the orientations can't be distinguished by restriction analysis. To know if the construct you have in hand is in the intended or the reverse direction, it will be necessary to sequence the region.</p> <p>Sequencing reactions require 200-500 ng of plasmid DNA, and 3.2 pmoles of sequencing primer in a final volume of 12 μl. The miniprep'd plasmid should have ~1 ug of nucleic acid/µl but that will be a mixture of RNA and DNA, so we will guess at the amount of plasmid DNA there and use 1 µl of your miniprep'd DNA/sequencing reaction.</p> <p>Prepare an eppendorf tube with enough material for two sequencing reactions, namely</p> <ul>
<li>2 µl of your plasmid DNA candidate(s)</li>     <li>6.4 µl of a 1:100 dilution of the sequencing primer NO198</li>     <li>15.6 µl sterile water</li> </ul>
<p>Pipet this solution up and down to mix and then transfer 12 µl to one of the tubes in the 8-tube strip we will turn in at the <a href="http://web.mit.edu/biopolymers/www/">biopolymers facility</a> in E17 for sequencing. The data will be available for you to examine one week from today. Keep your fingers crossed.</p> <p>DONE!</p> <h2 class="subhead">For Next Time</h2> <ol>
<li>Exchange the writing assignment you handed in today with your lab partner for comments/reactions. Begin by reading the tips for peer review from Neal Lerner (<a href="../../../contents/labs/lerner_rev_tips.pdf">PDF</a>).</li>     <li>Then, as you read your lab partner's writing, be a thoughtful and sensitive reviewer. You will do your partner a disservice if you say the writing is great when it's not. Similarly, you will waste your time adding comments that are too harsh or too obscure to be integrated into a revision. You should begin your review by restating what you think the point of the paragraph is. This will help your partner know if the point got across. Next you should restate what evidence was provided and, if appropriate include editorial comments about their value (e.g. "Natalie, the point about all cells requiring energy is very general and so you might want to say this before you say that the word "cells" rhymes with "bells.") Finally you should comment on the writing itself. If a sentence was confusing, say so and if at all possible, say why (e.g. "Natalie, the verb in your sentence that starts "when life began ages ago" is not found until the 46th word, making the point of the sentence hard to get"). Print out two copies of your review, one to hand in and one for your lab partner. Keep your fingers crossed that your lab partner is as thoughtful a reviewer as you are.</li>     <li>Prepare a figure of the agarose gel you ran last time. The images are posted on the discussion page associated with that lab. This figure will be included in your Module 1 <a href="../../../contents/assignments/mod1/index.htm">portfolio assessment assignment</a> so a good effort now will allow us to return your work with comments and what you submit later will be that much better. If you need information about how to prepare figures, look at our <a href="../../../contents/assignments/guide_lab_report-htm/index.htm">Guidelines for writing a lab report</a>.</li>     <li>Gain some familiarity with protein gels if you haven't ever run one, or if it's been a while. One good link to start with is <a href="http://en.wikipedia.org/wiki/Polyacrylamide_gel">Wikipedia: Polyacrylamide gel</a>.</li> </ol>
<h2 class="subhead">Reagents List</h2> <ul>
<li>Sequencing primers     <ul>
<li>M13KO7seq_2101_fwd         <ul>
<li>NO198 5'- CAA GGC ACT GAC CCC GTT AAA ACT TAT TAC -3'</li>         </ul>
</li>         <li>M13KO7seq_2151_fwd         <ul>
<li>NO199 5'- AAA AGC CAT GTA TGA CGC TTA CTG GAA CGG -3'</li>         </ul>
</li>     </ul>
</li>     <li>Genetic parts for M13KO7 in the <a href="http://partsregistry.org/Main_Page">Registry of Standard Biological Parts</a>
</li> </ul>
<div class="maintabletemplate"><table class="tablewidth75" summary="See table instruction for summary.">
<thead><tr>
<th scope="col">GENETIC ELEMENTS</th>             <th scope="col">PROMOTERS</th>             <th scope="col">RBS</th>             <th scope="col">CODINGS</th>         </tr></thead>
<tbody>
<tr class="row">
<td> </td>             <td colspan="3"><strong>Arbitrary starting point</strong></td>         </tr>
<tr class="alt-row">
<td>gII</td>             <td>BBa_M13102</td>             <td>BBa_M13502</td>             <td>BBa_M13002</td>         </tr>
</tbody>
<tbody>
<tr class="row">
<td>gX</td>             <td>BBa_M13110</td>             <td>BBa_M13510</td>             <td>BBa_M13010</td>         </tr>
<tr class="alt-row">
<td>gV</td>             <td>BBa_M13105</td>             <td>BBa_M13505</td>             <td>BBa_M13005</td>         </tr>
</tbody>
<tbody>
<tr class="row">
<td>gVII</td>             <td> </td>             <td>BBa_M13507</td>             <td>BBa_M13007</td>         </tr>
<tr class="alt-row">
<td>gIX</td>             <td> </td>             <td>BBa_M13509</td>             <td>BBa_M13009</td>         </tr>
</tbody>
<tbody>
<tr class="row">
<td>gVIII</td>             <td>BBa_M13108</td>             <td>BBa_M13508</td>             <td>BBa_M13008</td>         </tr>
<tr class="alt-row">
<td> </td>             <td colspan="3"><strong>Transcriptional terminator</strong></td>         </tr>
</tbody>
<tbody>
<tr class="row">
<td>gIII</td>             <td>BBa_M13103</td>             <td>BBa_M13503</td>             <td>BBa_M13003</td>         </tr>
<tr class="alt-row">
<td>gVI</td>             <td>BBa_M13106</td>             <td>BBa_M13506</td>             <td>BBa_M13006</td>         </tr>
</tbody>
<tbody>
<tr class="row">
<td>gI</td>             <td>BBa_M13101</td>             <td>BBa_M13501</td>             <td>BBa_M13001</td>         </tr>
<tr class="alt-row">
<td>gXI</td>             <td> </td>             <td>BBa_M13511</td>             <td>BBa_M13011</td>         </tr>
</tbody>
<tbody>
<tr class="row">
<td>gIV</td>             <td>BBa_M13104</td>             <td>BBa_M13504</td>             <td>BBa_M13004</td>         </tr>
<tr class="alt-row">
<td> </td>             <td colspan="3"><strong>M13KO7 ori/KanR/p15a ori</strong></td>         </tr>
</tbody>
</table></div>
	                    
	        			
	                    
	                    
	                    	
	                       
	                    
            		 </div>
<!--Course_inner_section tag close -->
           		     <div class="clear"></div>
 				 </div>
<!--Course_wrapper tag close -->             
 			 </div>
<!--left tag close -->
 			                
         </div>
<!--grid tag close --> 
      </div>
		
		<div id="bottom">
			<div id="grid">
				
<div id="portletwrapper-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572" class="portletWrapper kssattr-portlethash-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572">
<div class="portletStaticText portlet-static-site-footer"><div id="bottom">
<div id="grid">
<!-- *begin footer* -->
<div id="footer" role="navigation sitemap">
<div id="foot-c1" class="grid_2 alpha">
<h4 class="footer">Courses</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/courses/find-by-topic/">Find by Topic</a></li>
    <li><a href="http://ocw.mit.edu/courses/find-by-number/">Find by Course Number</a></li>
    <li><a href="http://ocw.mit.edu/courses/find-by-department/">Find by Department</a></li>
    <li><a href="http://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
    <li><a href="http://ocw.mit.edu/courses/translated-courses/">Translated Courses</a></li>
    <li><a href="http://ocw.mit.edu/resources/">Supplemental Resources</a></li>
</ul>
</div>
<div id="foot-c2" class="grid_2">
<h4 class="footer">About</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/about/">About OpenCourseWare</a></li>
    <li><a href="http://ocw.mit.edu/about/site-statistics/">Site Stats</a></li>
    <li><a href="http://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
    <li><a href="http://ocw.mit.edu/about/ocw-consortium/">OCW Consortium</a></li>
    <li><a href="http://ocw.mit.edu/about/media-coverage/">Media Coverage</a></li>
    <li><a href="http://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
    <li><a href="http://ocw.mit.edu/about/media-coverage/press-releases/">Press Releases</a></li>
    <li><a href="http://ocw.mit.edu/about/next-decade/">OCW's Next Decade</a></li>
</ul>
</div>
<div id="foot-c3" class="grid_2">
<h4 class="footer">Donate</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/donate">Make a Donation</a></li>
    <li><a href="http://ocw.mit.edu/donate/why-donate/">Why Donate?</a></li>
    <li><a href="http://ocw.mit.edu/donate/ocw-course-champions-program/">Become a Course Champion</a></li>
    <li><a href="http://ocw.mit.edu/donate/our-supporters/">Our Supporters</a></li>
    <li><a href="http://ocw.mit.edu/donate/other-ways-to-contribute/">Other Ways to Contribute</a></li>
    <li><a href="http://ocw.mit.edu/donate/shop-ocw/">Shop OCW</a></li>
    <li><a href="http://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
</ul>
</div>
<div id="foot-c4" class="grid_2">
<h4 class="footer">Featured Sites</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/high-school/">Highlights for High School</a></li>
    <li><a href="https://www.edx.org/school/mitx/allcourses">MITx</a></li>
    <li><a href="http://teachingexcellence.mit.edu/">Teaching Excellence at MIT</a></li>
</ul>
<h4 class="footer" style="margin-top: 14px;">Tools</h4>
<ul class="foot-bullet">
<li><a href="http://ocw.mit.edu/help/">Help &amp; FAQs</a></li>
    <li><a href="../../../common/jsp/feedback.htm">Contact Us</a></li>
    <li><a href="../../../common/search/AdvancedSearch.htm">Advanced Search</a></li>
    <li><a href="http://ocw.mit.edu/help/site-map/">Site Map</a></li>
    <li><a href="../../../common/terms/index.htm">Privacy &amp; Terms of Use</a></li>
</ul>
</div>
<div id="foot-c5" class="grid_4 omega" itemprop="publisher" itemscope="" itemtype="http://schema.org/CollegeOrUniversity">
<h4 class="footer">About <span itemprop="name">MIT OpenCourseWare</span>
</h4>
<p class="accent" itemprop="description">MIT OpenCourseWare makes the materials used in the teaching of almost all of MIT's subjects available on the Web, free of charge. With more than 2,000 courses available, OCW is delivering on the promise of open sharing of knowledge.</p>
</div>
<div id="foot-copy" class="grid_8 alpha">
<a href="http://web.mit.edu"><img src="../../../common/images/logo_mit.png" alt="Massachusetts Institute of Technology logo and name." style="width: 195px; height: 44px;"></a>                 <a href="http://www.ocwconsortium.org/"><img src="../../../common/images/logo_ocwc.gif" alt="OpenCourseWare Consortium logo." style="width: 80px; height: 44px;"></a>                 <a rel="license" itemprop="useRightsUrl" href="http://creativecommons.org/licenses/by-nc-sa/3.0/us/deed.en_US"><img src="../../../common/images/cc_by-nc-sa.png" alt="Creative Commons logo with terms BY-NC-SA." style="width: 126px; height: 44px;"></a>
<p class="copyright">© 2001–2013<br>
Massachusetts Institute of Technology</p>
<p style="font-size: 0.9em; margin-bottom: 15px;">Your use of the MIT OpenCourseWare site and materials is subject to our <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/us/deed.en_US" rel="license">Creative Commons License</a> and other <a href="../../../common/terms/index.htm" rel="cc:morePermissions">terms of use</a>.</p>
</div>
<div id="foot-support" class="grid_4 omega"> </div>
</div>
</div>
</div></div>

</div>





                
			</div> <!-- bottom grid end -->
		</div>
<!-- bottom end -->
		
		
   </body>
</html>
